<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">

<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1"/>
  <meta name="generator" content="distill" />

  <style type="text/css">
  /* Hide doc at startup (prevent jankiness while JS renders/transforms) */
  body {
    visibility: hidden;
  }
  </style>

 <!--radix_placeholder_import_source-->
 <!--/radix_placeholder_import_source-->

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ad0000; } /* Alert */
code span.an { color: #5e5e5e; } /* Annotation */
code span.at { } /* Attribute */
code span.bn { color: #ad0000; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007ba5; } /* ControlFlow */
code span.ch { color: #20794d; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #5e5e5e; } /* Comment */
code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
code span.dt { color: #ad0000; } /* DataType */
code span.dv { color: #ad0000; } /* DecVal */
code span.er { color: #ad0000; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #ad0000; } /* Float */
code span.fu { color: #4758ab; } /* Function */
code span.im { } /* Import */
code span.in { color: #5e5e5e; } /* Information */
code span.kw { color: #007ba5; } /* Keyword */
code span.op { color: #5e5e5e; } /* Operator */
code span.ot { color: #007ba5; } /* Other */
code span.pp { color: #ad0000; } /* Preprocessor */
code span.sc { color: #5e5e5e; } /* SpecialChar */
code span.ss { color: #20794d; } /* SpecialString */
code span.st { color: #20794d; } /* String */
code span.va { color: #111111; } /* Variable */
code span.vs { color: #20794d; } /* VerbatimString */
code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
</style>

  <!--radix_placeholder_meta_tags-->
<title>ffli.dev: Comparing HGNC Symbol Mappings by 3 Different Databases</title>

<meta property="description" itemprop="description" content="Mapping from Entrez gene IDs to HGNC gene symbols for genes in GSE152641 using biomaRt, BridgeDbR, and org.Hs.eg.db."/>

<link rel="canonical" href="https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/"/>
<link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>

<!--  https://schema.org/Article -->
<meta property="article:published" itemprop="datePublished" content="2021-07-11"/>
<meta property="article:created" itemprop="dateCreated" content="2021-07-11"/>
<meta name="article:author" content="Feifei Li"/>

<!--  https://developers.facebook.com/docs/sharing/webmasters#markup -->
<meta property="og:title" content="ffli.dev: Comparing HGNC Symbol Mappings by 3 Different Databases"/>
<meta property="og:type" content="article"/>
<meta property="og:description" content="Mapping from Entrez gene IDs to HGNC gene symbols for genes in GSE152641 using biomaRt, BridgeDbR, and org.Hs.eg.db."/>
<meta property="og:url" content="https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/"/>
<meta property="og:image" content="https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/biomart.png"/>
<meta property="og:image:width" content="250"/>
<meta property="og:image:height" content="99"/>
<meta property="og:locale" content="en_US"/>
<meta property="og:site_name" content="ffli.dev"/>

<!--  https://dev.twitter.com/cards/types/summary -->
<meta property="twitter:card" content="summary_large_image"/>
<meta property="twitter:title" content="ffli.dev: Comparing HGNC Symbol Mappings by 3 Different Databases"/>
<meta property="twitter:description" content="Mapping from Entrez gene IDs to HGNC gene symbols for genes in GSE152641 using biomaRt, BridgeDbR, and org.Hs.eg.db."/>
<meta property="twitter:url" content="https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/"/>
<meta property="twitter:image" content="https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/biomart.png"/>
<meta property="twitter:image:width" content="250"/>
<meta property="twitter:image:height" content="99"/>

<!--  https://scholar.google.com/intl/en/scholar/inclusion.html#indexing -->
<meta name="citation_title" content="ffli.dev: Comparing HGNC Symbol Mappings by 3 Different Databases"/>
<meta name="citation_fulltext_html_url" content="https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/"/>
<meta name="citation_fulltext_world_readable" content=""/>
<meta name="citation_online_date" content="2021/07/11"/>
<meta name="citation_publication_date" content="2021/07/11"/>
<meta name="citation_author" content="Feifei Li"/>
<!--/radix_placeholder_meta_tags-->
  
  <meta name="citation_reference" content="citation_title=Org. Hs. Eg. Db: Genome wide annotation for human;citation_publication_date=2019;citation_volume=3;citation_author=Marc Carlson;citation_author=S Falcon;citation_author=H Pages;citation_author=N Li"/>
  <meta name="citation_reference" content="citation_title=BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis;citation_publication_date=2005;citation_volume=21;citation_doi=10.1093/bioinformatics/bti525;citation_issn=1367-4803;citation_author=Steffen Durinck;citation_author=Yves Moreau;citation_author=Arek Kasprzyk;citation_author=Sean Davis;citation_author=Bart De Moor;citation_author=Alvis Brazma;citation_author=Wolfgang Huber"/>
  <meta name="citation_reference" content="citation_title=BridgeDbR;citation_publication_date=2017;citation_publisher=Bioconductor;citation_doi=10.18129/B9.BIOC.BRIDGEDBR;citation_author=BridgeDbR"/>
  <meta name="citation_reference" content="citation_title=The BridgeDb framework: Standardized access to gene, protein and metabolite identifier mapping services;citation_publication_date=2010;citation_volume=11;citation_doi=10.1186/1471-2105-11-5;citation_author=Martijn Van Iersel;citation_author=Alexander Pico;citation_author=Thomas Kelder;citation_author=Jianjiong Gao;citation_author=Isaac Ho;citation_author=Kristina Hanspers;citation_author=Bruce Conklin;citation_author=Chris Evelo"/>
  <meta name="citation_reference" content="citation_title=AnnotationDbi;citation_publication_date=2017;citation_publisher=Bioconductor;citation_doi=10.18129/B9.BIOC.ANNOTATIONDBI;citation_author=Marc Carlson HervÃ© PagÃ¨s"/>
  <meta name="citation_reference" content="citation_title=Correction: Corrigendum: An apoptosis-enhancing drug overcomes platinum resistance in a tumour-initiating subpopulation of ovarian cancer;citation_publication_date=2016;citation_volume=7;citation_doi=10.1038/ncomms10703;citation_issn=2041-1723;citation_author=D. M. Janzen;citation_author=E. Tiourin;citation_author=J. A. Salehi;citation_author=D. Y. Paik;citation_author=J. Lu;citation_author=M. Pellegrini;citation_author=S. Memarzadeh"/>
  <meta name="citation_reference" content="citation_title=RETRACTED ARTICLE: An apoptosis-enhancing drug overcomes platinum resistance in a tumour-initiating subpopulation of ovarian cancer;citation_publication_date=2015;citation_publisher=Nature Publishing Group;citation_volume=6;citation_author=DM Janzen;citation_author=E Tiourin;citation_author=JA Salehi;citation_author=DY Paik;citation_author=J Lu;citation_author=M Pellegrini;citation_author=S Memarzadeh"/>
  <!--radix_placeholder_rmarkdown_metadata-->

<script type="text/json" id="radix-rmarkdown-metadata">
{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["title","description","author","date","bibliography","output","creative_commons","categories","preview","citation_url","canonical_url"]}},"value":[{"type":"character","attributes":{},"value":["Comparing HGNC Symbol Mappings by 3 Different Databases"]},{"type":"character","attributes":{},"value":["Mapping from Entrez gene IDs to HGNC gene symbols for genes in GSE152641 using biomaRt, BridgeDbR, and org.Hs.eg.db."]},{"type":"list","attributes":{},"value":[{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["name","url"]}},"value":[{"type":"character","attributes":{},"value":["Feifei Li"]},{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":[]}},"value":[]}]}]},{"type":"character","attributes":{},"value":["2021-07-11"]},{"type":"character","attributes":{},"value":["biblio.bib"]},{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["distill::distill_article"]}},"value":[{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["self_contained","toc"]}},"value":[{"type":"logical","attributes":{},"value":[false]},{"type":"logical","attributes":{},"value":[true]}]}]},{"type":"character","attributes":{},"value":["CC BY"]},{"type":"character","attributes":{},"value":["R","bioconductor","bioinformatics"]},{"type":"character","attributes":{},"value":["biomart.png"]},{"type":"character","attributes":{},"value":["https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/"]},{"type":"character","attributes":{},"value":["https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/"]}]}
</script>
<!--/radix_placeholder_rmarkdown_metadata-->
  
  <script type="text/json" id="radix-resource-manifest">
  {"type":"character","attributes":{},"value":["biblio.bib","biomart.png","comparing-hgnc-symbol-mappings-by-3-different-databases_files/anchor-4.2.2/anchor.min.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/bowser-1.9.3/bowser.min.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/distill-2.2.21/template.v2.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/header-attrs-2.9/header-attrs.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/jquery-1.11.3/jquery.min.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/kePrint-0.0.1/kePrint.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/lightable-0.0.1/lightable.css","comparing-hgnc-symbol-mappings-by-3-different-databases_files/popper-2.6.0/popper.min.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/tippy-6.2.7/tippy-bundle.umd.min.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/tippy-6.2.7/tippy-light-border.css","comparing-hgnc-symbol-mappings-by-3-different-databases_files/tippy-6.2.7/tippy.css","comparing-hgnc-symbol-mappings-by-3-different-databases_files/tippy-6.2.7/tippy.umd.min.js","comparing-hgnc-symbol-mappings-by-3-different-databases_files/webcomponents-2.0.0/webcomponents.js","data/Hs_Derby_Ensembl_91.bridge","data/normalised_counts.rds","derby.log"]}
  </script>
  <!--radix_placeholder_navigation_in_header-->
<meta name="distill:offset" content="../.."/>

<script type="application/javascript">

  window.headroom_prevent_pin = false;

  window.document.addEventListener("DOMContentLoaded", function (event) {

    // initialize headroom for banner
    var header = $('header').get(0);
    var headerHeight = header.offsetHeight;
    var headroom = new Headroom(header, {
      tolerance: 5,
      onPin : function() {
        if (window.headroom_prevent_pin) {
          window.headroom_prevent_pin = false;
          headroom.unpin();
        }
      }
    });
    headroom.init();
    if(window.location.hash)
      headroom.unpin();
    $(header).addClass('headroom--transition');

    // offset scroll location for banner on hash change
    // (see: https://github.com/WickyNilliams/headroom.js/issues/38)
    window.addEventListener("hashchange", function(event) {
      window.scrollTo(0, window.pageYOffset - (headerHeight + 25));
    });

    // responsive menu
    $('.distill-site-header').each(function(i, val) {
      var topnav = $(this);
      var toggle = topnav.find('.nav-toggle');
      toggle.on('click', function() {
        topnav.toggleClass('responsive');
      });
    });

    // nav dropdowns
    $('.nav-dropbtn').click(function(e) {
      $(this).next('.nav-dropdown-content').toggleClass('nav-dropdown-active');
      $(this).parent().siblings('.nav-dropdown')
         .children('.nav-dropdown-content').removeClass('nav-dropdown-active');
    });
    $("body").click(function(e){
      $('.nav-dropdown-content').removeClass('nav-dropdown-active');
    });
    $(".nav-dropdown").click(function(e){
      e.stopPropagation();
    });
  });
</script>

<style type="text/css">

/* Theme (user-documented overrideables for nav appearance) */

.distill-site-nav {
  color: rgba(255, 255, 255, 0.8);
  background-color: #0F2E3D;
  font-size: 15px;
  font-weight: 300;
}

.distill-site-nav a {
  color: inherit;
  text-decoration: none;
}

.distill-site-nav a:hover {
  color: white;
}

@media print {
  .distill-site-nav {
    display: none;
  }
}

.distill-site-header {

}

.distill-site-footer {

}


/* Site Header */

.distill-site-header {
  width: 100%;
  box-sizing: border-box;
  z-index: 3;
}

.distill-site-header .nav-left {
  display: inline-block;
  margin-left: 8px;
}

@media screen and (max-width: 768px) {
  .distill-site-header .nav-left {
    margin-left: 0;
  }
}


.distill-site-header .nav-right {
  float: right;
  margin-right: 8px;
}

.distill-site-header a,
.distill-site-header .title {
  display: inline-block;
  text-align: center;
  padding: 14px 10px 14px 10px;
}

.distill-site-header .title {
  font-size: 18px;
  min-width: 150px;
}

.distill-site-header .logo {
  padding: 0;
}

.distill-site-header .logo img {
  display: none;
  max-height: 20px;
  width: auto;
  margin-bottom: -4px;
}

.distill-site-header .nav-image img {
  max-height: 18px;
  width: auto;
  display: inline-block;
  margin-bottom: -3px;
}



@media screen and (min-width: 1000px) {
  .distill-site-header .logo img {
    display: inline-block;
  }
  .distill-site-header .nav-left {
    margin-left: 20px;
  }
  .distill-site-header .nav-right {
    margin-right: 20px;
  }
  .distill-site-header .title {
    padding-left: 12px;
  }
}


.distill-site-header .nav-toggle {
  display: none;
}

.nav-dropdown {
  display: inline-block;
  position: relative;
}

.nav-dropdown .nav-dropbtn {
  border: none;
  outline: none;
  color: rgba(255, 255, 255, 0.8);
  padding: 16px 10px;
  background-color: transparent;
  font-family: inherit;
  font-size: inherit;
  font-weight: inherit;
  margin: 0;
  margin-top: 1px;
  z-index: 2;
}

.nav-dropdown-content {
  display: none;
  position: absolute;
  background-color: white;
  min-width: 200px;
  border: 1px solid rgba(0,0,0,0.15);
  border-radius: 4px;
  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.1);
  z-index: 1;
  margin-top: 2px;
  white-space: nowrap;
  padding-top: 4px;
  padding-bottom: 4px;
}

.nav-dropdown-content hr {
  margin-top: 4px;
  margin-bottom: 4px;
  border: none;
  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
}

.nav-dropdown-active {
  display: block;
}

.nav-dropdown-content a, .nav-dropdown-content .nav-dropdown-header {
  color: black;
  padding: 6px 24px;
  text-decoration: none;
  display: block;
  text-align: left;
}

.nav-dropdown-content .nav-dropdown-header {
  display: block;
  padding: 5px 24px;
  padding-bottom: 0;
  text-transform: uppercase;
  font-size: 14px;
  color: #999999;
  white-space: nowrap;
}

.nav-dropdown:hover .nav-dropbtn {
  color: white;
}

.nav-dropdown-content a:hover {
  background-color: #ddd;
  color: black;
}

.nav-right .nav-dropdown-content {
  margin-left: -45%;
  right: 0;
}

@media screen and (max-width: 768px) {
  .distill-site-header a, .distill-site-header .nav-dropdown  {display: none;}
  .distill-site-header a.nav-toggle {
    float: right;
    display: block;
  }
  .distill-site-header .title {
    margin-left: 0;
  }
  .distill-site-header .nav-right {
    margin-right: 0;
  }
  .distill-site-header {
    overflow: hidden;
  }
  .nav-right .nav-dropdown-content {
    margin-left: 0;
  }
}


@media screen and (max-width: 768px) {
  .distill-site-header.responsive {position: relative; min-height: 500px; }
  .distill-site-header.responsive a.nav-toggle {
    position: absolute;
    right: 0;
    top: 0;
  }
  .distill-site-header.responsive a,
  .distill-site-header.responsive .nav-dropdown {
    display: block;
    text-align: left;
  }
  .distill-site-header.responsive .nav-left,
  .distill-site-header.responsive .nav-right {
    width: 100%;
  }
  .distill-site-header.responsive .nav-dropdown {float: none;}
  .distill-site-header.responsive .nav-dropdown-content {position: relative;}
  .distill-site-header.responsive .nav-dropdown .nav-dropbtn {
    display: block;
    width: 100%;
    text-align: left;
  }
}

/* Site Footer */

.distill-site-footer {
  width: 100%;
  overflow: hidden;
  box-sizing: border-box;
  z-index: 3;
  margin-top: 30px;
  padding-top: 30px;
  padding-bottom: 30px;
  text-align: center;
}

/* Headroom */

d-title {
  padding-top: 6rem;
}

@media print {
  d-title {
    padding-top: 4rem;
  }
}

.headroom {
  z-index: 1000;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
}

.headroom--transition {
  transition: all .4s ease-in-out;
}

.headroom--unpinned {
  top: -100px;
}

.headroom--pinned {
  top: 0;
}

/* adjust viewport for navbar height */
/* helps vertically center bootstrap (non-distill) content */
.min-vh-100 {
  min-height: calc(100vh - 100px) !important;
}

</style>

<script src="../../site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="../../site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet"/>
<link href="../../site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet"/>
<script src="../../site_libs/headroom-0.9.4/headroom.min.js"></script>
<script src="../../site_libs/autocomplete-0.37.1/autocomplete.min.js"></script>
<script src="../../site_libs/fuse-6.4.1/fuse.min.js"></script>

<script type="application/javascript">

function getMeta(metaName) {
  var metas = document.getElementsByTagName('meta');
  for (let i = 0; i < metas.length; i++) {
    if (metas[i].getAttribute('name') === metaName) {
      return metas[i].getAttribute('content');
    }
  }
  return '';
}

function offsetURL(url) {
  var offset = getMeta('distill:offset');
  return offset ? offset + '/' + url : url;
}

function createFuseIndex() {

  // create fuse index
  var options = {
    keys: [
      { name: 'title', weight: 20 },
      { name: 'categories', weight: 15 },
      { name: 'description', weight: 10 },
      { name: 'contents', weight: 5 },
    ],
    ignoreLocation: true,
    threshold: 0
  };
  var fuse = new window.Fuse([], options);

  // fetch the main search.json
  return fetch(offsetURL('search.json'))
    .then(function(response) {
      if (response.status == 200) {
        return response.json().then(function(json) {
          // index main articles
          json.articles.forEach(function(article) {
            fuse.add(article);
          });
          // download collections and index their articles
          return Promise.all(json.collections.map(function(collection) {
            return fetch(offsetURL(collection)).then(function(response) {
              if (response.status === 200) {
                return response.json().then(function(articles) {
                  articles.forEach(function(article) {
                    fuse.add(article);
                  });
                })
              } else {
                return Promise.reject(
                  new Error('Unexpected status from search index request: ' +
                            response.status)
                );
              }
            });
          })).then(function() {
            return fuse;
          });
        });

      } else {
        return Promise.reject(
          new Error('Unexpected status from search index request: ' +
                      response.status)
        );
      }
    });
}

window.document.addEventListener("DOMContentLoaded", function (event) {

  // get search element (bail if we don't have one)
  var searchEl = window.document.getElementById('distill-search');
  if (!searchEl)
    return;

  createFuseIndex()
    .then(function(fuse) {

      // make search box visible
      searchEl.classList.remove('hidden');

      // initialize autocomplete
      var options = {
        autoselect: true,
        hint: false,
        minLength: 2,
      };
      window.autocomplete(searchEl, options, [{
        source: function(query, callback) {
          const searchOptions = {
            isCaseSensitive: false,
            shouldSort: true,
            minMatchCharLength: 2,
            limit: 10,
          };
          var results = fuse.search(query, searchOptions);
          callback(results
            .map(function(result) { return result.item; })
          );
        },
        templates: {
          suggestion: function(suggestion) {
            var img = suggestion.preview && Object.keys(suggestion.preview).length > 0
              ? `<img src="${offsetURL(suggestion.preview)}"</img>`
              : '';
            var html = `
              <div class="search-item">
                <h3>${suggestion.title}</h3>
                <div class="search-item-description">
                  ${suggestion.description || ''}
                </div>
                <div class="search-item-preview">
                  ${img}
                </div>
              </div>
            `;
            return html;
          }
        }
      }]).on('autocomplete:selected', function(event, suggestion) {
        window.location.href = offsetURL(suggestion.path);
      });
      // remove inline display style on autocompleter (we want to
      // manage responsive display via css)
      $('.algolia-autocomplete').css("display", "");
    })
    .catch(function(error) {
      console.log(error);
    });

});

</script>

<style type="text/css">

.nav-search {
  font-size: x-small;
}

/* Algolioa Autocomplete */

.algolia-autocomplete {
  display: inline-block;
  margin-left: 10px;
  vertical-align: sub;
  background-color: white;
  color: black;
  padding: 6px;
  padding-top: 8px;
  padding-bottom: 0;
  border-radius: 6px;
  border: 1px #0F2E3D solid;
  width: 180px;
}


@media screen and (max-width: 768px) {
  .distill-site-nav .algolia-autocomplete {
    display: none;
    visibility: hidden;
  }
  .distill-site-nav.responsive .algolia-autocomplete {
    display: inline-block;
    visibility: visible;
  }
  .distill-site-nav.responsive .algolia-autocomplete .aa-dropdown-menu {
    margin-left: 0;
    width: 400px;
    max-height: 400px;
  }
}

.algolia-autocomplete .aa-input, .algolia-autocomplete .aa-hint {
  width: 90%;
  outline: none;
  border: none;
}

.algolia-autocomplete .aa-hint {
  color: #999;
}
.algolia-autocomplete .aa-dropdown-menu {
  width: 550px;
  max-height: 70vh;
  overflow-x: visible;
  overflow-y: scroll;
  padding: 5px;
  margin-top: 3px;
  margin-left: -150px;
  background-color: #fff;
  border-radius: 5px;
  border: 1px solid #999;
  border-top: none;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion {
  cursor: pointer;
  padding: 5px 4px;
  border-bottom: 1px solid #eee;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion:last-of-type {
  border-bottom: none;
  margin-bottom: 2px;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion .search-item {
  overflow: hidden;
  font-size: 0.8em;
  line-height: 1.4em;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion .search-item h3 {
  font-size: 1rem;
  margin-block-start: 0;
  margin-block-end: 5px;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion .search-item-description {
  display: inline-block;
  overflow: hidden;
  height: 2.8em;
  width: 80%;
  margin-right: 4%;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion .search-item-preview {
  display: inline-block;
  width: 15%;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion .search-item-preview img {
  height: 3em;
  width: auto;
  display: none;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion .search-item-preview img[src] {
  display: initial;
}

.algolia-autocomplete .aa-dropdown-menu .aa-suggestion.aa-cursor {
  background-color: #eee;
}
.algolia-autocomplete .aa-dropdown-menu .aa-suggestion em {
  font-weight: bold;
  font-style: normal;
}

</style>


<!--/radix_placeholder_navigation_in_header-->
  <!--radix_placeholder_distill-->

<style type="text/css">

body {
  background-color: white;
}

.pandoc-table {
  width: 100%;
}

.pandoc-table>caption {
  margin-bottom: 10px;
}

.pandoc-table th:not([align]) {
  text-align: left;
}

.pagedtable-footer {
  font-size: 15px;
}

d-byline .byline {
  grid-template-columns: 2fr 2fr;
}

d-byline .byline h3 {
  margin-block-start: 1.5em;
}

d-byline .byline .authors-affiliations h3 {
  margin-block-start: 0.5em;
}

.authors-affiliations .orcid-id {
  width: 16px;
  height:16px;
  margin-left: 4px;
  margin-right: 4px;
  vertical-align: middle;
  padding-bottom: 2px;
}

d-title .dt-tags {
  margin-top: 1em;
  grid-column: text;
}

.dt-tags .dt-tag {
  text-decoration: none;
  display: inline-block;
  color: rgba(0,0,0,0.6);
  padding: 0em 0.4em;
  margin-right: 0.5em;
  margin-bottom: 0.4em;
  font-size: 70%;
  border: 1px solid rgba(0,0,0,0.2);
  border-radius: 3px;
  text-transform: uppercase;
  font-weight: 500;
}

d-article table.gt_table td,
d-article table.gt_table th {
  border-bottom: none;
  font-size: 100%;
}

.html-widget {
  margin-bottom: 2.0em;
}

.l-screen-inset {
  padding-right: 16px;
}

.l-screen .caption {
  margin-left: 10px;
}

.shaded {
  background: rgb(247, 247, 247);
  padding-top: 20px;
  padding-bottom: 20px;
  border-top: 1px solid rgba(0, 0, 0, 0.1);
  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
}

.shaded .html-widget {
  margin-bottom: 0;
  border: 1px solid rgba(0, 0, 0, 0.1);
}

.shaded .shaded-content {
  background: white;
}

.text-output {
  margin-top: 0;
  line-height: 1.5em;
}

.hidden {
  display: none !important;
}

hr.section-separator {
  border: none;
  border-top: 1px solid rgba(0, 0, 0, 0.1);
  margin: 0px;
}


d-byline {
  border-top: none;
}

d-article {
  padding-top: 2.5rem;
  padding-bottom: 30px;
  border-top: none;
}

d-appendix {
  padding-top: 30px;
}

d-article>p>img {
  width: 100%;
}

d-article h2 {
  margin: 1rem 0 1.5rem 0;
}

d-article h3 {
  margin-top: 1.5rem;
}

d-article iframe {
  border: 1px solid rgba(0, 0, 0, 0.1);
  margin-bottom: 2.0em;
  width: 100%;
}

/* Tweak code blocks */

d-article div.sourceCode code,
d-article pre code {
  font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
}

d-article pre,
d-article div.sourceCode,
d-article div.sourceCode pre {
  overflow: auto;
}

d-article div.sourceCode {
  background-color: white;
}

d-article div.sourceCode pre {
  padding-left: 10px;
  font-size: 12px;
  border-left: 2px solid rgba(0,0,0,0.1);
}

d-article pre {
  font-size: 12px;
  color: black;
  background: none;
  margin-top: 0;
  text-align: left;
  white-space: pre;
  word-spacing: normal;
  word-break: normal;
  word-wrap: normal;
  line-height: 1.5;

  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;

  -webkit-hyphens: none;
  -moz-hyphens: none;
  -ms-hyphens: none;
  hyphens: none;
}

d-article pre a {
  border-bottom: none;
}

d-article pre a:hover {
  border-bottom: none;
  text-decoration: underline;
}

d-article details {
  grid-column: text;
  margin-bottom: 0.8em;
}

@media(min-width: 768px) {

d-article pre,
d-article div.sourceCode,
d-article div.sourceCode pre {
  overflow: visible !important;
}

d-article div.sourceCode pre {
  padding-left: 18px;
  font-size: 14px;
}

/* tweak for Pandoc numbered line within distill */
d-article pre.numberSource code > span {
    left: -2em;
}

d-article pre {
  font-size: 14px;
}

}

figure img.external {
  background: white;
  border: 1px solid rgba(0, 0, 0, 0.1);
  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.1);
  padding: 18px;
  box-sizing: border-box;
}

/* CSS for d-contents */

.d-contents {
  grid-column: text;
  color: rgba(0,0,0,0.8);
  font-size: 0.9em;
  padding-bottom: 1em;
  margin-bottom: 1em;
  padding-bottom: 0.5em;
  margin-bottom: 1em;
  padding-left: 0.25em;
  justify-self: start;
}

@media(min-width: 1000px) {
  .d-contents.d-contents-float {
    height: 0;
    grid-column-start: 1;
    grid-column-end: 4;
    justify-self: center;
    padding-right: 3em;
    padding-left: 2em;
  }
}

.d-contents nav h3 {
  font-size: 18px;
  margin-top: 0;
  margin-bottom: 1em;
}

.d-contents li {
  list-style-type: none
}

.d-contents nav > ul {
  padding-left: 0;
}

.d-contents ul {
  padding-left: 1em
}

.d-contents nav ul li {
  margin-top: 0.6em;
  margin-bottom: 0.2em;
}

.d-contents nav a {
  font-size: 13px;
  border-bottom: none;
  text-decoration: none
  color: rgba(0, 0, 0, 0.8);
}

.d-contents nav a:hover {
  text-decoration: underline solid rgba(0, 0, 0, 0.6)
}

.d-contents nav > ul > li > a {
  font-weight: 600;
}

.d-contents nav > ul > li > ul {
  font-weight: inherit;
}

.d-contents nav > ul > li > ul > li {
  margin-top: 0.2em;
}


.d-contents nav ul {
  margin-top: 0;
  margin-bottom: 0.25em;
}

.d-article-with-toc h2:nth-child(2) {
  margin-top: 0;
}


/* Figure */

.figure {
  position: relative;
  margin-bottom: 2.5em;
  margin-top: 1.5em;
}

.figure .caption {
  color: rgba(0, 0, 0, 0.6);
  font-size: 12px;
  line-height: 1.5em;
}

.figure img.external {
  background: white;
  border: 1px solid rgba(0, 0, 0, 0.1);
  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.1);
  padding: 18px;
  box-sizing: border-box;
}

.figure .caption a {
  color: rgba(0, 0, 0, 0.6);
}

.figure .caption b,
.figure .caption strong, {
  font-weight: 600;
  color: rgba(0, 0, 0, 1.0);
}

/* Citations */

d-article .citation {
  color: inherit;
  cursor: inherit;
}

div.hanging-indent{
  margin-left: 1em; text-indent: -1em;
}

/* Citation hover box */

.tippy-box[data-theme~=light-border] {
  background-color: rgba(250, 250, 250, 0.95);
}

.tippy-content > p {
  margin-bottom: 0;
  padding: 2px;
}


/* Tweak 1000px media break to show more text */

@media(min-width: 1000px) {
  .base-grid,
  distill-header,
  d-title,
  d-abstract,
  d-article,
  d-appendix,
  distill-appendix,
  d-byline,
  d-footnote-list,
  d-citation-list,
  distill-footer {
    grid-template-columns: [screen-start] 1fr [page-start kicker-start] 80px [middle-start] 50px [text-start kicker-end] 65px 65px 65px 65px 65px 65px 65px 65px [text-end gutter-start] 65px [middle-end] 65px [page-end gutter-end] 1fr [screen-end];
    grid-column-gap: 16px;
  }

  .grid {
    grid-column-gap: 16px;
  }

  d-article {
    font-size: 1.06rem;
    line-height: 1.7em;
  }
  figure .caption, .figure .caption, figure figcaption {
    font-size: 13px;
  }
}

@media(min-width: 1180px) {
  .base-grid,
  distill-header,
  d-title,
  d-abstract,
  d-article,
  d-appendix,
  distill-appendix,
  d-byline,
  d-footnote-list,
  d-citation-list,
  distill-footer {
    grid-template-columns: [screen-start] 1fr [page-start kicker-start] 60px [middle-start] 60px [text-start kicker-end] 60px 60px 60px 60px 60px 60px 60px 60px [text-end gutter-start] 60px [middle-end] 60px [page-end gutter-end] 1fr [screen-end];
    grid-column-gap: 32px;
  }

  .grid {
    grid-column-gap: 32px;
  }
}


/* Get the citation styles for the appendix (not auto-injected on render since
   we do our own rendering of the citation appendix) */

d-appendix .citation-appendix,
.d-appendix .citation-appendix {
  font-size: 11px;
  line-height: 15px;
  border-left: 1px solid rgba(0, 0, 0, 0.1);
  padding-left: 18px;
  border: 1px solid rgba(0,0,0,0.1);
  background: rgba(0, 0, 0, 0.02);
  padding: 10px 18px;
  border-radius: 3px;
  color: rgba(150, 150, 150, 1);
  overflow: hidden;
  margin-top: -12px;
  white-space: pre-wrap;
  word-wrap: break-word;
}

/* Include appendix styles here so they can be overridden */

d-appendix {
  contain: layout style;
  font-size: 0.8em;
  line-height: 1.7em;
  margin-top: 60px;
  margin-bottom: 0;
  border-top: 1px solid rgba(0, 0, 0, 0.1);
  color: rgba(0,0,0,0.5);
  padding-top: 60px;
  padding-bottom: 48px;
}

d-appendix h3 {
  grid-column: page-start / text-start;
  font-size: 15px;
  font-weight: 500;
  margin-top: 1em;
  margin-bottom: 0;
  color: rgba(0,0,0,0.65);
}

d-appendix h3 + * {
  margin-top: 1em;
}

d-appendix ol {
  padding: 0 0 0 15px;
}

@media (min-width: 768px) {
  d-appendix ol {
    padding: 0 0 0 30px;
    margin-left: -30px;
  }
}

d-appendix li {
  margin-bottom: 1em;
}

d-appendix a {
  color: rgba(0, 0, 0, 0.6);
}

d-appendix > * {
  grid-column: text;
}

d-appendix > d-footnote-list,
d-appendix > d-citation-list,
d-appendix > distill-appendix {
  grid-column: screen;
}

/* Include footnote styles here so they can be overridden */

d-footnote-list {
  contain: layout style;
}

d-footnote-list > * {
  grid-column: text;
}

d-footnote-list a.footnote-backlink {
  color: rgba(0,0,0,0.3);
  padding-left: 0.5em;
}



/* Anchor.js */

.anchorjs-link {
  /*transition: all .25s linear; */
  text-decoration: none;
  border-bottom: none;
}
*:hover > .anchorjs-link {
  margin-left: -1.125em !important;
  text-decoration: none;
  border-bottom: none;
}

/* Social footer */

.social_footer {
  margin-top: 30px;
  margin-bottom: 0;
  color: rgba(0,0,0,0.67);
}

.disqus-comments {
  margin-right: 30px;
}

.disqus-comment-count {
  border-bottom: 1px solid rgba(0, 0, 0, 0.4);
  cursor: pointer;
}

#disqus_thread {
  margin-top: 30px;
}

.article-sharing a {
  border-bottom: none;
  margin-right: 8px;
}

.article-sharing a:hover {
  border-bottom: none;
}

.sidebar-section.subscribe {
  font-size: 12px;
  line-height: 1.6em;
}

.subscribe p {
  margin-bottom: 0.5em;
}


.article-footer .subscribe {
  font-size: 15px;
  margin-top: 45px;
}


.sidebar-section.custom {
  font-size: 12px;
  line-height: 1.6em;
}

.custom p {
  margin-bottom: 0.5em;
}

/* Styles for listing layout (hide title) */
.layout-listing d-title, .layout-listing .d-title {
  display: none;
}

/* Styles for posts lists (not auto-injected) */


.posts-with-sidebar {
  padding-left: 45px;
  padding-right: 45px;
}

.posts-list .description h2,
.posts-list .description p {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
}

.posts-list .description h2 {
  font-weight: 700;
  border-bottom: none;
  padding-bottom: 0;
}

.posts-list h2.post-tag {
  border-bottom: 1px solid rgba(0, 0, 0, 0.2);
  padding-bottom: 12px;
}
.posts-list {
  margin-top: 60px;
  margin-bottom: 24px;
}

.posts-list .post-preview {
  text-decoration: none;
  overflow: hidden;
  display: block;
  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
  padding: 24px 0;
}

.post-preview-last {
  border-bottom: none !important;
}

.posts-list .posts-list-caption {
  grid-column: screen;
  font-weight: 400;
}

.posts-list .post-preview h2 {
  margin: 0 0 6px 0;
  line-height: 1.2em;
  font-style: normal;
  font-size: 24px;
}

.posts-list .post-preview p {
  margin: 0 0 12px 0;
  line-height: 1.4em;
  font-size: 16px;
}

.posts-list .post-preview .thumbnail {
  box-sizing: border-box;
  margin-bottom: 24px;
  position: relative;
  max-width: 500px;
}
.posts-list .post-preview img {
  width: 100%;
  display: block;
}

.posts-list .metadata {
  font-size: 12px;
  line-height: 1.4em;
  margin-bottom: 18px;
}

.posts-list .metadata > * {
  display: inline-block;
}

.posts-list .metadata .publishedDate {
  margin-right: 2em;
}

.posts-list .metadata .dt-authors {
  display: block;
  margin-top: 0.3em;
  margin-right: 2em;
}

.posts-list .dt-tags {
  display: block;
  line-height: 1em;
}

.posts-list .dt-tags .dt-tag {
  display: inline-block;
  color: rgba(0,0,0,0.6);
  padding: 0.3em 0.4em;
  margin-right: 0.2em;
  margin-bottom: 0.4em;
  font-size: 60%;
  border: 1px solid rgba(0,0,0,0.2);
  border-radius: 3px;
  text-transform: uppercase;
  font-weight: 500;
}

.posts-list img {
  opacity: 1;
}

.posts-list img[data-src] {
  opacity: 0;
}

.posts-more {
  clear: both;
}


.posts-sidebar {
  font-size: 16px;
}

.posts-sidebar h3 {
  font-size: 16px;
  margin-top: 0;
  margin-bottom: 0.5em;
  font-weight: 400;
  text-transform: uppercase;
}

.sidebar-section {
  margin-bottom: 30px;
}

.categories ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
}

.categories li {
  color: rgba(0, 0, 0, 0.8);
  margin-bottom: 0;
}

.categories li>a {
  border-bottom: none;
}

.categories li>a:hover {
  border-bottom: 1px solid rgba(0, 0, 0, 0.4);
}

.categories .active {
  font-weight: 600;
}

.categories .category-count {
  color: rgba(0, 0, 0, 0.4);
}


@media(min-width: 768px) {
  .posts-list .post-preview h2 {
    font-size: 26px;
  }
  .posts-list .post-preview .thumbnail {
    float: right;
    width: 30%;
    margin-bottom: 0;
  }
  .posts-list .post-preview .description {
    float: left;
    width: 45%;
  }
  .posts-list .post-preview .metadata {
    float: left;
    width: 20%;
    margin-top: 8px;
  }
  .posts-list .post-preview p {
    margin: 0 0 12px 0;
    line-height: 1.5em;
    font-size: 16px;
  }
  .posts-with-sidebar .posts-list {
    float: left;
    width: 75%;
  }
  .posts-with-sidebar .posts-sidebar {
    float: right;
    width: 20%;
    margin-top: 60px;
    padding-top: 24px;
    padding-bottom: 24px;
  }
}


/* Improve display for browsers without grid (IE/Edge <= 15) */

.downlevel {
  line-height: 1.6em;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
  margin: 0;
}

.downlevel .d-title {
  padding-top: 6rem;
  padding-bottom: 1.5rem;
}

.downlevel .d-title h1 {
  font-size: 50px;
  font-weight: 700;
  line-height: 1.1em;
  margin: 0 0 0.5rem;
}

.downlevel .d-title p {
  font-weight: 300;
  font-size: 1.2rem;
  line-height: 1.55em;
  margin-top: 0;
}

.downlevel .d-byline {
  padding-top: 0.8em;
  padding-bottom: 0.8em;
  font-size: 0.8rem;
  line-height: 1.8em;
}

.downlevel .section-separator {
  border: none;
  border-top: 1px solid rgba(0, 0, 0, 0.1);
}

.downlevel .d-article {
  font-size: 1.06rem;
  line-height: 1.7em;
  padding-top: 1rem;
  padding-bottom: 2rem;
}


.downlevel .d-appendix {
  padding-left: 0;
  padding-right: 0;
  max-width: none;
  font-size: 0.8em;
  line-height: 1.7em;
  margin-bottom: 0;
  color: rgba(0,0,0,0.5);
  padding-top: 40px;
  padding-bottom: 48px;
}

.downlevel .footnotes ol {
  padding-left: 13px;
}

.downlevel .base-grid,
.downlevel .distill-header,
.downlevel .d-title,
.downlevel .d-abstract,
.downlevel .d-article,
.downlevel .d-appendix,
.downlevel .distill-appendix,
.downlevel .d-byline,
.downlevel .d-footnote-list,
.downlevel .d-citation-list,
.downlevel .distill-footer,
.downlevel .appendix-bottom,
.downlevel .posts-container {
  padding-left: 40px;
  padding-right: 40px;
}

@media(min-width: 768px) {
  .downlevel .base-grid,
  .downlevel .distill-header,
  .downlevel .d-title,
  .downlevel .d-abstract,
  .downlevel .d-article,
  .downlevel .d-appendix,
  .downlevel .distill-appendix,
  .downlevel .d-byline,
  .downlevel .d-footnote-list,
  .downlevel .d-citation-list,
  .downlevel .distill-footer,
  .downlevel .appendix-bottom,
  .downlevel .posts-container {
  padding-left: 150px;
  padding-right: 150px;
  max-width: 900px;
}
}

.downlevel pre code {
  display: block;
  border-left: 2px solid rgba(0, 0, 0, .1);
  padding: 0 0 0 20px;
  font-size: 14px;
}

.downlevel code, .downlevel pre {
  color: black;
  background: none;
  font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
  text-align: left;
  white-space: pre;
  word-spacing: normal;
  word-break: normal;
  word-wrap: normal;
  line-height: 1.5;

  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;

  -webkit-hyphens: none;
  -moz-hyphens: none;
  -ms-hyphens: none;
  hyphens: none;
}

.downlevel .posts-list .post-preview {
  color: inherit;
}



</style>

<script type="application/javascript">

function is_downlevel_browser() {
  if (bowser.isUnsupportedBrowser({ msie: "12", msedge: "16"},
                                 window.navigator.userAgent)) {
    return true;
  } else {
    return window.load_distill_framework === undefined;
  }
}

// show body when load is complete
function on_load_complete() {

  // add anchors
  if (window.anchors) {
    window.anchors.options.placement = 'left';
    window.anchors.add('d-article > h2, d-article > h3, d-article > h4, d-article > h5');
  }


  // set body to visible
  document.body.style.visibility = 'visible';

  // force redraw for leaflet widgets
  if (window.HTMLWidgets) {
    var maps = window.HTMLWidgets.findAll(".leaflet");
    $.each(maps, function(i, el) {
      var map = this.getMap();
      map.invalidateSize();
      map.eachLayer(function(layer) {
        if (layer instanceof L.TileLayer)
          layer.redraw();
      });
    });
  }

  // trigger 'shown' so htmlwidgets resize
  $('d-article').trigger('shown');
}

function init_distill() {

  init_common();

  // create front matter
  var front_matter = $('<d-front-matter></d-front-matter>');
  $('#distill-front-matter').wrap(front_matter);

  // create d-title
  $('.d-title').changeElementType('d-title');

  // separator
  var separator = '<hr class="section-separator" style="clear: both"/>';
  // prepend separator above appendix
  $('.d-byline').before(separator);
  $('.d-article').before(separator);

  // create d-byline
  var byline = $('<d-byline></d-byline>');
  $('.d-byline').replaceWith(byline);

  // create d-article
  var article = $('<d-article></d-article>');
  $('.d-article').wrap(article).children().unwrap();

  // move posts container into article
  $('.posts-container').appendTo($('d-article'));

  // create d-appendix
  $('.d-appendix').changeElementType('d-appendix');

  // flag indicating that we have appendix items
  var appendix = $('.appendix-bottom').children('h3').length > 0;

  // replace footnotes with <d-footnote>
  $('.footnote-ref').each(function(i, val) {
    appendix = true;
    var href = $(this).attr('href');
    var id = href.replace('#', '');
    var fn = $('#' + id);
    var fn_p = $('#' + id + '>p');
    fn_p.find('.footnote-back').remove();
    var text = fn_p.html();
    var dtfn = $('<d-footnote></d-footnote>');
    dtfn.html(text);
    $(this).replaceWith(dtfn);
  });
  // remove footnotes
  $('.footnotes').remove();

  // move refs into #references-listing
  $('#references-listing').replaceWith($('#refs'));

  $('h1.appendix, h2.appendix').each(function(i, val) {
    $(this).changeElementType('h3');
  });
  $('h3.appendix').each(function(i, val) {
    var id = $(this).attr('id');
    $('.d-contents a[href="#' + id + '"]').parent().remove();
    appendix = true;
    $(this).nextUntil($('h1, h2, h3')).addBack().appendTo($('d-appendix'));
  });

  // show d-appendix if we have appendix content
  $("d-appendix").css('display', appendix ? 'grid' : 'none');

  // localize layout chunks to just output
  $('.layout-chunk').each(function(i, val) {

    // capture layout
    var layout = $(this).attr('data-layout');

    // apply layout to markdown level block elements
    var elements = $(this).children().not('details, div.sourceCode, pre, script');
    elements.each(function(i, el) {
      var layout_div = $('<div class="' + layout + '"></div>');
      if (layout_div.hasClass('shaded')) {
        var shaded_content = $('<div class="shaded-content"></div>');
        $(this).wrap(shaded_content);
        $(this).parent().wrap(layout_div);
      } else {
        $(this).wrap(layout_div);
      }
    });


    // unwrap the layout-chunk div
    $(this).children().unwrap();
  });

  // remove code block used to force  highlighting css
  $('.distill-force-highlighting-css').parent().remove();

  // remove empty line numbers inserted by pandoc when using a
  // custom syntax highlighting theme, except when numbering line
  // in code chunk
  $('pre:not(.numberLines) code.sourceCode a:empty').remove();

  // load distill framework
  load_distill_framework();

  // wait for window.distillRunlevel == 4 to do post processing
  function distill_post_process() {

    if (!window.distillRunlevel || window.distillRunlevel < 4)
      return;

    // hide author/affiliations entirely if we have no authors
    var front_matter = JSON.parse($("#distill-front-matter").html());
    var have_authors = front_matter.authors && front_matter.authors.length > 0;
    if (!have_authors)
      $('d-byline').addClass('hidden');

    // article with toc class
    $('.d-contents').parent().addClass('d-article-with-toc');

    // strip links that point to #
    $('.authors-affiliations').find('a[href="#"]').removeAttr('href');

    // add orcid ids
    $('.authors-affiliations').find('.author').each(function(i, el) {
      var orcid_id = front_matter.authors[i].orcidID;
      var author_name = front_matter.authors[i].author
      if (orcid_id) {
        var a = $('<a></a>');
        a.attr('href', 'https://orcid.org/' + orcid_id);
        var img = $('<img></img>');
        img.addClass('orcid-id');
        img.attr('alt', author_name ? 'ORCID ID for ' + author_name : 'ORCID ID');
        img.attr('src','data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg==');
        a.append(img);
        $(this).append(a);
      }
    });

    // hide elements of author/affiliations grid that have no value
    function hide_byline_column(caption) {
      $('d-byline').find('h3:contains("' + caption + '")').parent().css('visibility', 'hidden');
    }

    // affiliations
    var have_affiliations = false;
    for (var i = 0; i<front_matter.authors.length; ++i) {
      var author = front_matter.authors[i];
      if (author.affiliation !== "&nbsp;") {
        have_affiliations = true;
        break;
      }
    }
    if (!have_affiliations)
      $('d-byline').find('h3:contains("Affiliations")').css('visibility', 'hidden');

    // published date
    if (!front_matter.publishedDate)
      hide_byline_column("Published");

    // document object identifier
    var doi = $('d-byline').find('h3:contains("DOI")');
    var doi_p = doi.next().empty();
    if (!front_matter.doi) {
      // if we have a citation and valid citationText then link to that
      if ($('#citation').length > 0 && front_matter.citationText) {
        doi.html('Citation');
        $('<a href="#citation"></a>')
          .text(front_matter.citationText)
          .appendTo(doi_p);
      } else {
        hide_byline_column("DOI");
      }
    } else {
      $('<a></a>')
         .attr('href', "https://doi.org/" + front_matter.doi)
         .html(front_matter.doi)
         .appendTo(doi_p);
    }

     // change plural form of authors/affiliations
    if (front_matter.authors.length === 1) {
      var grid = $('.authors-affiliations');
      grid.children('h3:contains("Authors")').text('Author');
      grid.children('h3:contains("Affiliations")').text('Affiliation');
    }

    // remove d-appendix and d-footnote-list local styles
    $('d-appendix > style:first-child').remove();
    $('d-footnote-list > style:first-child').remove();

    // move appendix-bottom entries to the bottom
    $('.appendix-bottom').appendTo('d-appendix').children().unwrap();
    $('.appendix-bottom').remove();

    // hoverable references
    $('span.citation[data-cites]').each(function() {
      const citeChild = $(this).children()[0]
      // Do not process if @xyz has been used without escaping and without bibliography activated
      // https://github.com/rstudio/distill/issues/466
      if (citeChild === undefined) return true

      if (citeChild.nodeName == "D-FOOTNOTE") {
        var fn = citeChild
        $(this).html(fn.shadowRoot.querySelector("sup"))
        $(this).id = fn.id
        fn.remove()
      }
      var refs = $(this).attr('data-cites').split(" ");
      var refHtml = refs.map(function(ref) {
        // Could use CSS.escape too here, we insure backward compatibility in navigator
        return "<p>" + $('div[id="ref-' + ref + '"]').html() + "</p>";
      }).join("\n");
      window.tippy(this, {
        allowHTML: true,
        content: refHtml,
        maxWidth: 500,
        interactive: true,
        interactiveBorder: 10,
        theme: 'light-border',
        placement: 'bottom-start'
      });
    });

    // fix footnotes in tables (#411)
    // replacing broken distill.pub feature
    $('table d-footnote').each(function() {
      // we replace internal showAtNode methode which is triggered when hovering a footnote
      this.hoverBox.showAtNode = function(node) {
        // ported from https://github.com/distillpub/template/pull/105/files
        calcOffset = function(elem) {
            let x = elem.offsetLeft;
            let y = elem.offsetTop;
            // Traverse upwards until an `absolute` element is found or `elem`
            // becomes null.
            while (elem = elem.offsetParent && elem.style.position != 'absolute') {
                x += elem.offsetLeft;
                y += elem.offsetTop;
            }

            return { left: x, top: y };
        }
        // https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetTop
        const bbox = node.getBoundingClientRect();
        const offset = calcOffset(node);
        this.show([offset.left + bbox.width, offset.top + bbox.height]);
      }
    })

    // clear polling timer
    clearInterval(tid);

    // show body now that everything is ready
    on_load_complete();
  }

  var tid = setInterval(distill_post_process, 50);
  distill_post_process();

}

function init_downlevel() {

  init_common();

   // insert hr after d-title
  $('.d-title').after($('<hr class="section-separator"/>'));

  // check if we have authors
  var front_matter = JSON.parse($("#distill-front-matter").html());
  var have_authors = front_matter.authors && front_matter.authors.length > 0;

  // manage byline/border
  if (!have_authors)
    $('.d-byline').remove();
  $('.d-byline').after($('<hr class="section-separator"/>'));
  $('.d-byline a').remove();

  // remove toc
  $('.d-contents').remove();

  // move appendix elements
  $('h1.appendix, h2.appendix').each(function(i, val) {
    $(this).changeElementType('h3');
  });
  $('h3.appendix').each(function(i, val) {
    $(this).nextUntil($('h1, h2, h3')).addBack().appendTo($('.d-appendix'));
  });


  // inject headers into references and footnotes
  var refs_header = $('<h3></h3>');
  refs_header.text('References');
  $('#refs').prepend(refs_header);

  var footnotes_header = $('<h3></h3');
  footnotes_header.text('Footnotes');
  $('.footnotes').children('hr').first().replaceWith(footnotes_header);

  // move appendix-bottom entries to the bottom
  $('.appendix-bottom').appendTo('.d-appendix').children().unwrap();
  $('.appendix-bottom').remove();

  // remove appendix if it's empty
  if ($('.d-appendix').children().length === 0)
    $('.d-appendix').remove();

  // prepend separator above appendix
  $('.d-appendix').before($('<hr class="section-separator" style="clear: both"/>'));

  // trim code
  $('pre>code').each(function(i, val) {
    $(this).html($.trim($(this).html()));
  });

  // move posts-container right before article
  $('.posts-container').insertBefore($('.d-article'));

  $('body').addClass('downlevel');

  on_load_complete();
}


function init_common() {

  // jquery plugin to change element types
  (function($) {
    $.fn.changeElementType = function(newType) {
      var attrs = {};

      $.each(this[0].attributes, function(idx, attr) {
        attrs[attr.nodeName] = attr.nodeValue;
      });

      this.replaceWith(function() {
        return $("<" + newType + "/>", attrs).append($(this).contents());
      });
    };
  })(jQuery);

  // prevent underline for linked images
  $('a > img').parent().css({'border-bottom' : 'none'});

  // mark non-body figures created by knitr chunks as 100% width
  $('.layout-chunk').each(function(i, val) {
    var figures = $(this).find('img, .html-widget');
    // ignore leaflet img layers (#106)
    figures = figures.filter(':not(img[class*="leaflet"])')
    if ($(this).attr('data-layout') !== "l-body") {
      figures.css('width', '100%');
    } else {
      figures.css('max-width', '100%');
      figures.filter("[width]").each(function(i, val) {
        var fig = $(this);
        fig.css('width', fig.attr('width') + 'px');
      });

    }
  });

  // auto-append index.html to post-preview links in file: protocol
  // and in rstudio ide preview
  $('.post-preview').each(function(i, val) {
    if (window.location.protocol === "file:")
      $(this).attr('href', $(this).attr('href') + "index.html");
  });

  // get rid of index.html references in header
  if (window.location.protocol !== "file:") {
    $('.distill-site-header a[href]').each(function(i,val) {
      $(this).attr('href', $(this).attr('href').replace(/^index[.]html/, "./"));
    });
  }

  // add class to pandoc style tables
  $('tr.header').parent('thead').parent('table').addClass('pandoc-table');
  $('.kable-table').children('table').addClass('pandoc-table');

  // add figcaption style to table captions
  $('caption').parent('table').addClass("figcaption");

  // initialize posts list
  if (window.init_posts_list)
    window.init_posts_list();

  // implmement disqus comment link
  $('.disqus-comment-count').click(function() {
    window.headroom_prevent_pin = true;
    $('#disqus_thread').toggleClass('hidden');
    if (!$('#disqus_thread').hasClass('hidden')) {
      var offset = $(this).offset();
      $(window).resize();
      $('html, body').animate({
        scrollTop: offset.top - 35
      });
    }
  });
}

document.addEventListener('DOMContentLoaded', function() {
  if (is_downlevel_browser())
    init_downlevel();
  else
    window.addEventListener('WebComponentsReady', init_distill);
});

</script>

<style type="text/css">
/* base variables */

/* Edit the CSS properties in this file to create a custom
   Distill theme. Only edit values in the right column
   for each row; values shown are the CSS defaults.
   To return any property to the default,
   you may set its value to: unset
   All rows must end with a semi-colon.                      */

/* Optional: embed custom fonts here with `@import`          */
/* This must remain at the top of this file.                 */



html {
  /*-- Main font sizes --*/
  --title-size:      50px;
  --body-size:       1.06rem;
  --code-size:       14px;
  --aside-size:      12px;
  --fig-cap-size:    13px;
  /*-- Main font colors --*/
  --title-color:     #000000;
  --header-color:    rgba(0, 0, 0, 0.8);
  --body-color:      rgba(0, 0, 0, 0.8);
  --aside-color:     rgba(0, 0, 0, 0.6);
  --fig-cap-color:   rgba(0, 0, 0, 0.6);
  /*-- Specify custom fonts ~~~ must be imported above   --*/
  --heading-font:    sans-serif;
  --mono-font:       monospace;
  --body-font:       sans-serif;
  --navbar-font:     sans-serif;  /* websites + blogs only */
}

/*-- ARTICLE METADATA --*/
d-byline {
  --heading-size:    0.6rem;
  --heading-color:   rgba(0, 0, 0, 0.5);
  --body-size:       0.8rem;
  --body-color:      rgba(0, 0, 0, 0.8);
}

/*-- ARTICLE TABLE OF CONTENTS --*/
.d-contents {
  --heading-size:    18px;
  --contents-size:   13px;
}

/*-- ARTICLE APPENDIX --*/
d-appendix {
  --heading-size:    15px;
  --heading-color:   rgba(0, 0, 0, 0.65);
  --text-size:       0.8em;
  --text-color:      rgba(0, 0, 0, 0.5);
}

/*-- WEBSITE HEADER + FOOTER --*/
/* These properties only apply to Distill sites and blogs  */

.distill-site-header {
  --title-size:       18px;
  --text-color:       rgba(255, 255, 255, 0.8);
  --text-size:        15px;
  --hover-color:      white;
  --bkgd-color:       #0F2E3D;
}

.distill-site-footer {
  --text-color:       rgba(255, 255, 255, 0.8);
  --text-size:        15px;
  --hover-color:      white;
  --bkgd-color:       #0F2E3D;
}

/*-- Additional custom styles --*/
/* Add any additional CSS rules below                      */
</style>
<style type="text/css">
/* base variables */

/* This theme is taken from Thomas's The Mockup Blog
   (https://github.com/jthomasmock/radix_themockup.git)
   I modified the theme based on my need.
   */

/* Optional: embed custom fonts here with `@import`          */
/* This must remain at the top of this file.                 */
@import url('https://fonts.googleapis.com/css2?family=Lato');
@import url('https://fonts.googleapis.com/css2?family=Chivo');
@import url('https://fonts.googleapis.com/css2?family=Fira+Mono');

html {
    /*-- Main font sizes --*/
    --title-size:         50px;
    --body-size:          1.06rem;
    --code-size:          14px;
    --aside-size:         12px;
    --fig-cap-size:       13px;
    /*-- Main font colors --*/
    --title-color:        #000000;
    --header-color:       rgba(0, 0, 0, 0.8);
    --body-color:         rgba(0, 0, 0, 0.8);
    --aside-color:        rgba(0, 0, 0, 0.6);
    --fig-cap-color:      rgba(0, 0, 0, 0.6);
    /*-- Specify custom fonts ~~~ must be imported above   --*/
    --heading-font:       'Fira Mono', sans-serif;
    --mono-font:          'Fira Mono', monospace;
    --body-font:          'Lato', sans-serif;;
    --navbar-font:        "Fira Mono", serif;
    /* -- Darkmode -- */
    --darkmode-bg:        #121212;
    --darkmode-text:      #FFFFFF;
    --darkmode-contrast:  #BB86FC;
    --darkmode-error:     #CF6679;
    --darkmode-code-bg:   #212121;
    /* Lightmode */
    --lightmode-error:    #B00020;
    --lightmode-contrast: #ff6700;
}

/*-- ARTICLE METADATA --*/
.d-byline {
    --heading-size:    0.6rem;
    --heading-color:   rgba(0, 0, 0, 0.5);
    --body-size:       0.8rem;
    --body-color:      rgba(0, 0, 0, 0.8);
}

/* -- ARTICLE PREVIEW IMAGE SIZE -- */
.posts-list .post-preview .thumbnail img {
    width: 50%;
    display: block;
}

/*-- ARTICLE TABLE OF CONTENTS --*/
.d-contents {
    --heading-size:    18px;
    --contents-size:   13px;
}

.d-contents a {
    color:        var(--body-color);
    border-color: var(--body-color);
}

.d-contents li > *:hover {
    color:        var(--lightmode-contrast);
    border-color: var(--lightmode-contrast);
}

/* -- MAIN CATEGORY TAGS -- */
.posts-list .dt-tags .dt-tag:hover {
    color:        var(--lightmode-contrast);
    border-color: var(--lightmode-contrast);
}

/* -- ARTICLE PREVIEW */
.posts-list .post-preview .description h2:hover {
    color: var(--lightmode-contrast);
}

.posts-list .post-preview .description p:hover {
    color: var(--lightmode-contrast);
}

/*-- ARTICLE CATEGORY TAGS --*/
.dt-tag {
    color:        var(--body-color);
    border-color: var(--body-color);
}

.dt-tag:hover {
    color:        var(--lightmode-contrast);
    border-color: var(--lightmode-contrast);
}

/* -- ARTICLE CLICKABLE -- */
d-article a:hover {
    color: var(--lightmode-contrast);
    border-color: var(--lightmode-contrast);
}

/*-- ARTICLE APPENDIX --*/
.d-appendix {
    --heading-size:    15px;
    --heading-color:   rgba(0, 0, 0, 0.65);
    --text-size:       0.8em;
    --text-color:      rgba(0, 0, 0, 0.5);
}

/* -- HOME CATEGORIES -- */

.categories li>a:hover {
    color:         var(--lightmode-contrast);
    border-bottom: none;
}

/*-- WEBSITE HEADER + FOOTER --*/
/* These properties only apply to Distill sites and blogs  */

.distill-site-header {
    --title-size:       18px;
    --text-color:       #000;
    --text-size:        15px;
    --hover-color:      var(--lightmode-contrast);
    --bkgd-color:       white;
}

.distill-site-footer {
    --text-color:       #000;
    --text-size:        15px;
    --hover-color:      var(--lightmode-contrast);
    --bkgd-color:       white;
}

d-appendix p a:hover {
    color:        var(--lightmode-contrast);
    border-color: var(--lightmode-contrast);
}

d-appendix .citation-appendix, .d-appendix .citation-appendix {
    background: rgba(255, 250, 250, 1);
}

.social_footer {
    color: var(--body-color);
}

.social_footer a:hover {
    color: var(--lightmode-contrast);
}

/* -- DARKMODE CONFIG -- */
/* Buhr (2021, March 14). jmbuhr.de: Making a Darkmode for your distill site.
Retrieved from https://jmbuhr.de/posts/2021-03-14-making-a-darkmode-for-your-distill-site/ */
.darkmode {
    --title-color:     var(--darkmode-contrast);
    --header-color:    var(--darkmode-text);
    --body-color:      var(--darkmode-text);
    --aside-color:     var(--darkmode-text);
    --fig-cap-color:   var(--darkmode-text);
    --hover-color:     var(--darkmode-contrast);
    background-color:  var(--darkmode-bg);
    color:             var(--darkmode-text);
}

.darkmode d-article,
.darkmode d-appendix,
.darkmode body {
    background-color: var(--darkmode-bg);
    color:            var(--darkmode-text);
}

.darkmode d-title {
    color: var(--darkmode-text);
}

.darkmode d-article blockquote {
    color: var(--darkmode-text);
}

/*-- ARTICLE METADATA --*/
.darkmode d-byline {
    --heading-color:  var(--darkmode-text);
    --body-color:     var(--darkmode-text);
}

/*-- ARTICLE TABLE OF CONTENTS --*/
.darkmode .d-contents li > * {
    color:        var(--darkmode-text);
    border-color: var(--darkmode-text);
}

.darkmode .d-contents li > *:hover {
    color:        var(--darkmode-contrast);
    border-color: var(--darkmode-contrast);
}

/* -- ARTICLE CLICKABLE -- */
.darkmode d-article a:hover {
    color:        var(--darkmode-contrast);
    border-color: var(--darkmode-contrast);
}

/*-- ARTICLE APPENDIX --*/
.darkmode d-appendix {
    border-color:      var(--darkmode-text);
    --heading-color:   var(--darkmode-text);
    --text-color:      var(--darkmode-text);
}

.darkmode d-appendix a {
    color: var(--darkmode-text);
}

.darkmode d-appendix a:hover {
    color:        var(--darkmode-contrast);
    border-color: var(--darkmode-contrast);
}

.darkmode .social_footer {
    color: var(--darkmode-text);
}

/*-- WEBSITE HEADER + FOOTER --*/
.darkmode .distill-site-header {
  --text-color:  var(--darkmode-text);
  --bkgd-color:  var(--darkmode-bg);
  --hover-color: var(--darkmode-contrast);
}

.darkmode .distill-site-footer {
  --text-color:  var(--darkmode-text);
  --bkgd-color:  var(--darkmode-bg);
  --hover-color: var(--darkmode-contrast);
}

.darkmode d-appendix p a:hover {
    color:        var(--darkmode-contrast);
    border-color: var(--darkmode-contrast);
}

.darkmode d-appendix .citation-appendix, .d-appendix .citation-appendix {
    background: var(--darkmode-code-bg);
}

/*-- Categorical tags on main page --*/

.darkmode .categories li > * {
  color: var(--darkmode-text);
}

.darkmode .categories li>a:hover {
    color: var(--darkmode-contrast);
}

.darkmode .categories .active {
    color: var(--darkmode-contrast);
}

/*-- Categorical tags in blogs --*/
.darkmode .posts-list .dt-tags .dt-tag:hover {
    color:        var(--darkmode-contrast);
    border-color: var(--darkmode-contrast);
}

.darkmode .dt-tags .dt-tag {
    color:        var(--darkmode-text);
    border-color: var(--darkmode-text);
}

.darkmode .dt-tag:hover {
    color:        var(--darkmode-contrast);
    border-color: var(--darkmode-contrast);
}

/* -- POST PREVIEW -- */
.darkmode .posts-list .post-preview {
    border-color: var(--darkmode-bg);
}

.darkmode .posts-list .post-preview .description h2:hover {
    color: var(--darkmode-contrast);
}

.darkmode .posts-list .post-preview .description p:hover {
    color: var(--darkmode-contrast);
}

/* -- CODE CHUNK -- */
.darkmode d-article div.sourceCode,
.darkmode code {
    background-color: var(--darkmode-code-bg);
}

.darkmode d-article pre {
    color: var(--darkmode-text);
}

.darkmode code span {
    color: var(--darkmode-text);
}</style>
<style type="text/css">
/* base style */

/* FONT FAMILIES */

:root {
  --heading-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
  --mono-default: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
  --body-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
}

body,
.posts-list .post-preview p,
.posts-list .description p {
  font-family: var(--body-font), var(--body-default);
}

h1, h2, h3, h4, h5, h6,
.posts-list .post-preview h2,
.posts-list .description h2 {
  font-family: var(--heading-font), var(--heading-default);
}

d-article div.sourceCode code,
d-article pre code {
  font-family: var(--mono-font), var(--mono-default);
}


/*-- TITLE --*/
d-title h1,
.posts-list > h1 {
  color: var(--title-color, black);
}

d-title h1 {
  font-size: var(--title-size, 50px);
}

/*-- HEADERS --*/
d-article h1,
d-article h2,
d-article h3,
d-article h4,
d-article h5,
d-article h6 {
  color: var(--header-color, rgba(0, 0, 0, 0.8));
}

/*-- BODY --*/
d-article > p,  /* only text inside of <p> tags */
d-article > ul, /* lists */
d-article > ol {
  color: var(--body-color, rgba(0, 0, 0, 0.8));
  font-size: var(--body-size, 1.06rem);
}


/*-- CODE --*/
d-article div.sourceCode code,
d-article pre code {
  font-size: var(--code-size, 14px);
}

/*-- ASIDE --*/
d-article aside {
  font-size: var(--aside-size, 12px);
  color: var(--aside-color, rgba(0, 0, 0, 0.6));
}

/*-- FIGURE CAPTIONS --*/
figure .caption,
figure figcaption,
.figure .caption {
  font-size: var(--fig-cap-size, 13px);
  color: var(--fig-cap-color, rgba(0, 0, 0, 0.6));
}

/*-- METADATA --*/
d-byline h3 {
  font-size: var(--heading-size, 0.6rem);
  color: var(--heading-color, rgba(0, 0, 0, 0.5));
}

d-byline {
  font-size: var(--body-size, 0.8rem);
  color: var(--body-color, rgba(0, 0, 0, 0.8));
}

d-byline a,
d-article d-byline a {
  color: var(--body-color, rgba(0, 0, 0, 0.8));
}

/*-- TABLE OF CONTENTS --*/
.d-contents nav h3 {
  font-size: var(--heading-size, 18px);
}

.d-contents nav a {
  font-size: var(--contents-size, 13px);
}

/*-- APPENDIX --*/
d-appendix h3 {
  font-size: var(--heading-size, 15px);
  color: var(--heading-color, rgba(0, 0, 0, 0.65));
}

d-appendix {
  font-size: var(--text-size, 0.8em);
  color: var(--text-color, rgba(0, 0, 0, 0.5));
}

d-appendix d-footnote-list a.footnote-backlink {
  color: var(--text-color, rgba(0, 0, 0, 0.5));
}

/*-- WEBSITE HEADER + FOOTER --*/
.distill-site-header .title {
  font-size: var(--title-size, 18px);
  font-family: var(--navbar-font), var(--heading-default);
}

.distill-site-header a,
.nav-dropdown .nav-dropbtn {
  font-family: var(--navbar-font), var(--heading-default);
}

.nav-dropdown .nav-dropbtn {
  color: var(--text-color, rgba(255, 255, 255, 0.8));
  font-size: var(--text-size, 15px);
}

.distill-site-header a:hover,
.nav-dropdown:hover .nav-dropbtn {
  color: var(--hover-color, white);
}

.distill-site-header {
  font-size: var(--text-size, 15px);
  color: var(--text-color, rgba(255, 255, 255, 0.8));
  background-color: var(--bkgd-color, #0F2E3D);
}

.distill-site-footer {
  font-size: var(--text-size, 15px);
  color: var(--text-color, rgba(255, 255, 255, 0.8));
  background-color: var(--bkgd-color, #0F2E3D);
}

.distill-site-footer a:hover {
  color: var(--hover-color, white);
}</style>
<!--/radix_placeholder_distill-->
  <script src="../../site_libs/header-attrs-2.9/header-attrs.js"></script>
  <script src="../../site_libs/kePrint-0.0.1/kePrint.js"></script>
  <link href="../../site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
  <script src="../../site_libs/jquery-1.11.3/jquery.min.js"></script>
  <script src="../../site_libs/popper-2.6.0/popper.min.js"></script>
  <link href="../../site_libs/tippy-6.2.7/tippy.css" rel="stylesheet" />
  <link href="../../site_libs/tippy-6.2.7/tippy-light-border.css" rel="stylesheet" />
  <script src="../../site_libs/tippy-6.2.7/tippy.umd.min.js"></script>
  <script src="../../site_libs/anchor-4.2.2/anchor.min.js"></script>
  <script src="../../site_libs/bowser-1.9.3/bowser.min.js"></script>
  <script src="../../site_libs/webcomponents-2.0.0/webcomponents.js"></script>
  <script src="../../site_libs/distill-2.2.21/template.v2.js"></script>
  <!--radix_placeholder_site_in_header-->
<!--/radix_placeholder_site_in_header-->


</head>

<body>

<!--radix_placeholder_front_matter-->

<script id="distill-front-matter" type="text/json">
{"title":"Comparing HGNC Symbol Mappings by 3 Different Databases","description":"Mapping from Entrez gene IDs to HGNC gene symbols for genes in GSE152641 using biomaRt, BridgeDbR, and org.Hs.eg.db.","authors":[{"author":"Feifei Li","authorURL":{},"affiliation":"&nbsp;","affiliationURL":"#","orcidID":""}],"publishedDate":"2021-07-11T00:00:00.000+08:00","citationText":"Li, 2021"}
</script>

<!--/radix_placeholder_front_matter-->
<!--radix_placeholder_navigation_before_body-->
<header class="header header--fixed" role="banner">
<nav class="distill-site-nav distill-site-header">
<div class="nav-left">
<a href="../../index.html" class="title">ffli.dev</a>
<input id="distill-search" class="nav-search hidden" type="text" placeholder="Search..."/>
</div>
<div class="nav-right">
<a href="../../index.html">Home</a>
<a href="../../about.html">About</a>
<a href="https://github.com/ff98li">
<i class="fab fa-github-alt" aria-hidden="true"></i>
</a>
<a href="../../cv.pdf">
<i class="fas fa-graduation-cap" aria-hidden="true"></i>
</a>
<a>
<i class="fas fa-lightbulb" aria-hidden="true"></i>
</a>
<a href="javascript:void(0);" class="nav-toggle">&#9776;</a>
</div>
</nav>
</header>
<!--/radix_placeholder_navigation_before_body-->
<!--radix_placeholder_site_before_body-->
<!--/radix_placeholder_site_before_body-->

<div class="d-title">
<h1>Comparing HGNC Symbol Mappings by 3 Different Databases</h1>
<!--radix_placeholder_categories-->
<div class="dt-tags">
  <a href="../../index.html#category:R" class="dt-tag">R</a>
  <a href="../../index.html#category:bioconductor" class="dt-tag">bioconductor</a>
  <a href="../../index.html#category:bioinformatics" class="dt-tag">bioinformatics</a>
</div>
<!--/radix_placeholder_categories-->
<p><p>Mapping from Entrez gene IDs to HGNC gene symbols for genes in GSE152641 using biomaRt, BridgeDbR, and org.Hs.eg.db.</p></p>
</div>

<div class="d-byline">
  Feifei Li true 
  
<br/>2021-07-11
</div>

<div class="d-article">
<div class="d-contents d-contents-float">
<nav class="l-text toc figcaption" id="TOC">
<h3>Contents</h3>
<ul>
<li><a href="#biomart">biomaRt</a>
<ul>
<li><a href="#mart-database">Mart Database</a></li>
<li><a href="#biomart-query">biomaRt Query</a></li>
<li><a href="#result">Result</a></li>
</ul></li>
<li><a href="#bridgedbr">BridgeDbR</a>
<ul>
<li><a href="#database">Database</a></li>
<li><a href="#mapping-identifiers">Mapping identifiers</a></li>
<li><a href="#result-1">Result</a></li>
</ul></li>
<li><a href="#orgdb">OrgDb</a>
<ul>
<li><a href="#org.hs.eg.db">org.Hs.eg.db</a></li>
<li><a href="#mapping">Mapping</a></li>
<li><a href="#result-2">Result</a></li>
</ul></li>
<li><a href="#manual-clean-up">Manual Clean-up</a>
<ul>
<li><a href="#enterz-gene-ids-missing-hgnc-symbols">Enterz gene IDs missing HGNC symbols</a></li>
<li><a href="#replcaing-entrez-id-in-the-count-matrix">Replcaing Entrez ID in the Count Matrix</a></li>
</ul></li>
<li><a href="#conclusion">Conclusion</a></li>
<li><a href="#to-do">To Do</a></li>
</ul>
</nav>
</div>
<p>This post is a continuation of <a href="https://www.ffli.dev/posts/2021-07-05-geo-datasets-experiment-selection-and-initial-processing/">the initial processing of GSE152641 RNA-seq data</a>. The aim is to map the identifiers of genes in GSE152641 from Entrez gene IDs to the corresponding HGNC gene names using three different databases, compare the results of mapping by these databases, and choose the mapping that has the best coverage. Note that we are not mapping HGNC symbols for all 20460 genes, but for the remaining 14426 genes after filtering genes with low counts in <a href="https://www.ffli.dev/posts/2021-07-05-geo-datasets-experiment-selection-and-initial-processing/#tmm-normalisation">the normalisation procedure</a>.<br />
</p>
<p>To load the normalised count matrix and get the Entrez gene IDs:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/readRDS.html'>readRDS</a></span><span class='op'>(</span>file <span class='op'>=</span> <span class='st'>"./data/normalised_counts.rds"</span><span class='op'>)</span>
<span class='va'>entrezID</span>          <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<h2 id="biomart">biomaRt</h2>
<p><a href="10.18129/B9.bioc.biomaRt"><code>biomaRt</code></a> is perhaps the most popular choice for mapping gene identifiers in R. It provides an R interface that executes direct SQL queries to <a href="http://www.biomart.org/notice.html">the BioMart databases</a>.<span class="citation" data-cites="biomart1">(<a href="#ref-biomart1" role="doc-biblioref">Durinck et al. 2005</a>)</span></p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'>## install biomaRt via bioconductor</span>
<span class='kw'>if</span> <span class='op'>(</span><span class='op'>!</span><span class='fu'><a href='https://rdrr.io/r/base/ns-load.html'>requireNamespace</a></span><span class='op'>(</span><span class='st'>"biomaRt"</span>, quietly <span class='op'>=</span> <span class='cn'>TRUE</span><span class='op'>)</span><span class='op'>)</span>
    <span class='fu'>BiocManager</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BiocManager/man/install.html'>install</a></span><span class='op'>(</span><span class='st'>"biomaRt"</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<h3 id="mart-database">Mart Database</h3>
<p>To see mart databases that can be accessed through the biomaRt interface:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listMarts.html'>listMarts</a></span><span class='op'>(</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
biomart
</th>
<th style="text-align:left;">
version
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ENSEMBL_MART_ENSEMBL
</td>
<td style="text-align:left;">
Ensembl Genes 104
</td>
</tr>
<tr>
<td style="text-align:left;">
ENSEMBL_MART_MOUSE
</td>
<td style="text-align:left;">
Mouse strains 104
</td>
</tr>
<tr>
<td style="text-align:left;">
ENSEMBL_MART_SNP
</td>
<td style="text-align:left;">
Ensembl Variation 104
</td>
</tr>
<tr>
<td style="text-align:left;">
ENSEMBL_MART_FUNCGEN
</td>
<td style="text-align:left;">
Ensembl Regulation 104
</td>
</tr>
</tbody>
</table>
</div>
<p>Note that by default it only lists mart databases hosted by Ensembl, as the <code>host</code> argument by default is set to <code>www.ensembl.org</code>. To see mart databases available on other hosts, users need to supply the <code>host</code> argument. For example, the Phytozome mart database hosted by JGI:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listMarts.html'>listMarts</a></span><span class='op'>(</span>host <span class='op'>=</span> <span class='st'>"https://phytozome.jgi.doe.gov"</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
biomart
</th>
<th style="text-align:left;">
version
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
phytozome_diversity__mart
</td>
<td style="text-align:left;">
V12 Genome Diversity
</td>
</tr>
<tr>
<td style="text-align:left;">
phytozome_mart_archive
</td>
<td style="text-align:left;">
Genome Archive
</td>
</tr>
</tbody>
</table>
</div>
<p>For Ensembl’s mart databases, according to the <a href="https://m.ensembl.org/info/data/biomart/how_to_use_biomart.html">tutorial page</a> for Ensembl’s web-based BioMart query tool,</p>
<ul>
<li>Ensembl Genes
<ul>
<li>contains the Ensembl gene set</li>
<li>allows users to retrieve:
<ul>
<li>Ensembl genes, transcripts, proteins as well as external references, microarrays, protein domains, structure, sequences, variants (only variants mapped to Ensembl Transcripts) and homology data</li>
</ul></li>
</ul></li>
<li>Ensembl Variation
<ul>
<li>allows users to retrieve germline and somatic variants as well as germline and somatic structural variants</li>
<li>contains variants’ phenotypes, citations, synomyms, consequences and flanking sequences</li>
<li>retrieve Ensembl genes, transcripts, regulatory and motif features mapped to variants</li>
</ul></li>
<li>Ensembl Regulation
<ul>
<li>allows users to retrieve regulatory features, evidence and segments, miRNA target regions, binding motifs and other regulatory regions</li>
</ul></li>
<li>Mouse strains
<ul>
<li>No ideas what this mart is. Looked it up on Google but nothing turned up.</li>
</ul></li>
</ul>
<p>Note that the VEGA mart mentioned in the tutorial has been archived and no longer available in biomaRt after <a href="http://vega.archive.ensembl.org/info/website/news.html?id=68&amp;submit=Go#news_1222">release 67</a>. For those who would like to access the archived VEGA mart database:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/useEnsembl.html'>useEnsembl</a></span><span class='op'>(</span>biomart <span class='op'>=</span> <span class='st'>"vega"</span>, version <span class='op'>=</span> <span class='fl'>67</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>Just FYI.<br />
</p>
<p>For mapping gene identifiers, we only need Ensembl Genes:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>ensembl</span> <span class='op'>&lt;-</span> <span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/useMart.html'>useMart</a></span><span class='op'>(</span><span class='st'>"ENSEMBL_MART_ENSEMBL"</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>To see datasets available in the chosen mart:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listDatasets.html'>listDatasets</a></span><span class='op'>(</span><span class='va'>ensembl</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
dataset
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
version
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
abrachyrhynchus_gene_ensembl
</td>
<td style="text-align:left;">
Pink-footed goose genes (ASM259213v1)
</td>
<td style="text-align:left;">
ASM259213v1
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_gene_ensembl
</td>
<td style="text-align:left;">
Eastern happy genes (fAstCal1.2)
</td>
<td style="text-align:left;">
fAstCal1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_gene_ensembl
</td>
<td style="text-align:left;">
Green anole genes (AnoCar2.0v2)
</td>
<td style="text-align:left;">
AnoCar2.0v2
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_gene_ensembl
</td>
<td style="text-align:left;">
Golden eagle genes (bAquChr1.2)
</td>
<td style="text-align:left;">
bAquChr1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_gene_ensembl
</td>
<td style="text-align:left;">
Midas cichlid genes (Midas_v5)
</td>
<td style="text-align:left;">
Midas_v5
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_gene_ensembl
</td>
<td style="text-align:left;">
Giant panda genes (ASM200744v2)
</td>
<td style="text-align:left;">
ASM200744v2
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_gene_ensembl
</td>
<td style="text-align:left;">
Mexican tetra genes (Astyanax_mexicanus-2.0)
</td>
<td style="text-align:left;">
Astyanax_mexicanus-2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_gene_ensembl
</td>
<td style="text-align:left;">
Ma’s night monkey genes (Anan_2.0)
</td>
<td style="text-align:left;">
Anan_2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_gene_ensembl
</td>
<td style="text-align:left;">
Clown anemonefish genes (AmpOce1.0)
</td>
<td style="text-align:left;">
AmpOce1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_gene_ensembl
</td>
<td style="text-align:left;">
Orange clownfish genes (Nemo_v1)
</td>
<td style="text-align:left;">
Nemo_v1
</td>
</tr>
<tr>
<td style="text-align:left;">
aplatyrhynchos_gene_ensembl
</td>
<td style="text-align:left;">
Mallard genes (ASM874695v1)
</td>
<td style="text-align:left;">
ASM874695v1
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_gene_ensembl
</td>
<td style="text-align:left;">
Spiny chromis genes (ASM210954v1)
</td>
<td style="text-align:left;">
ASM210954v1
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_gene_ensembl
</td>
<td style="text-align:left;">
Duck genes (CAU_duck1.0)
</td>
<td style="text-align:left;">
CAU_duck1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_gene_ensembl
</td>
<td style="text-align:left;">
Climbing perch genes (fAnaTes1.2)
</td>
<td style="text-align:left;">
fAnaTes1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_gene_ensembl
</td>
<td style="text-align:left;">
American bison genes (Bison_UMD1.0)
</td>
<td style="text-align:left;">
Bison_UMD1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_gene_ensembl
</td>
<td style="text-align:left;">
Domestic yak genes (LU_Bosgru_v3.0)
</td>
<td style="text-align:left;">
LU_Bosgru_v3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_gene_ensembl
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus genes (UOA_Brahman_1)
</td>
<td style="text-align:left;">
UOA_Brahman_1
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_gene_ensembl
</td>
<td style="text-align:left;">
Blue whale genes (mBalMus1.v2)
</td>
<td style="text-align:left;">
mBalMus1.v2
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_gene_ensembl
</td>
<td style="text-align:left;">
Wild yak genes (BosGru_v2.0)
</td>
<td style="text-align:left;">
BosGru_v2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_gene_ensembl
</td>
<td style="text-align:left;">
Siamese fighting fish genes (fBetSpl5.2)
</td>
<td style="text-align:left;">
fBetSpl5.2
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_gene_ensembl
</td>
<td style="text-align:left;">
Cow genes (ARS-UCD1.2)
</td>
<td style="text-align:left;">
ARS-UCD1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_gene_ensembl
</td>
<td style="text-align:left;">
Abingdon island giant tortoise genes (ASM359739v1)
</td>
<td style="text-align:left;">
ASM359739v1
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_gene_ensembl
</td>
<td style="text-align:left;">
Sooty mangabey genes (Caty_1.0)
</td>
<td style="text-align:left;">
Caty_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_gene_ensembl
</td>
<td style="text-align:left;">
Goldfish genes (ASM336829v1)
</td>
<td style="text-align:left;">
ASM336829v1
</td>
</tr>
<tr>
<td style="text-align:left;">
ccapucinus_gene_ensembl
</td>
<td style="text-align:left;">
Capuchin genes (Cebus_imitator-1.0)
</td>
<td style="text-align:left;">
Cebus_imitator-1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_gene_ensembl
</td>
<td style="text-align:left;">
Common carp genes (common_carp_genome)
</td>
<td style="text-align:left;">
common_carp_genome
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_gene_ensembl
</td>
<td style="text-align:left;">
Arabian camel genes (CamDro2)
</td>
<td style="text-align:left;">
CamDro2
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_gene_ensembl
</td>
<td style="text-align:left;">
Caenorhabditis elegans genes (WBcel235)
</td>
<td style="text-align:left;">
WBcel235
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_gene_ensembl
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS genes (CHOK1GS_HDv1)
</td>
<td style="text-align:left;">
CHOK1GS_HDv1
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_gene_ensembl
</td>
<td style="text-align:left;">
Channel bull blenny genes (fCotGob3.1)
</td>
<td style="text-align:left;">
fCotGob3.1
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_gene_ensembl
</td>
<td style="text-align:left;">
Atlantic herring genes (Ch_v2.0.2)
</td>
<td style="text-align:left;">
Ch_v2.0.2
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_gene_ensembl
</td>
<td style="text-align:left;">
Goat genes (ARS1)
</td>
<td style="text-align:left;">
ARS1
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_gene_ensembl
</td>
<td style="text-align:left;">
Sloth genes (choHof1)
</td>
<td style="text-align:left;">
choHof1
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_gene_ensembl
</td>
<td style="text-align:left;">
Yarkand deer genes (CEY_v1)
</td>
<td style="text-align:left;">
CEY_v1
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_gene_ensembl
</td>
<td style="text-align:left;">
C.intestinalis genes (KH)
</td>
<td style="text-align:left;">
KH
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_gene_ensembl
</td>
<td style="text-align:left;">
Marmoset genes (ASM275486v1)
</td>
<td style="text-align:left;">
ASM275486v1
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_gene_ensembl
</td>
<td style="text-align:left;">
Japanese quail genes (Coturnix_japonica_2.0)
</td>
<td style="text-align:left;">
Coturnix_japonica_2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_gene_ensembl
</td>
<td style="text-align:left;">
Long-tailed chinchilla genes (ChiLan1.0)
</td>
<td style="text-align:left;">
ChiLan1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_gene_ensembl
</td>
<td style="text-align:left;">
Dingo genes (ASM325472v1)
</td>
<td style="text-align:left;">
ASM325472v1
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_gene_ensembl
</td>
<td style="text-align:left;">
Dog genes (CanFam3.1)
</td>
<td style="text-align:left;">
CanFam3.1
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_gene_ensembl
</td>
<td style="text-align:left;">
Lumpfish genes (fCycLum1.pri)
</td>
<td style="text-align:left;">
fCycLum1.pri
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_gene_ensembl
</td>
<td style="text-align:left;">
Elephant shark genes (Callorhinchus_milii-6.1.3)
</td>
<td style="text-align:left;">
Callorhinchus_milii-6.1.3
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_gene_ensembl
</td>
<td style="text-align:left;">
Painted turtle genes (Chrysemys_picta_bellii-3.0.3)
</td>
<td style="text-align:left;">
Chrysemys_picta_bellii-3.0.3
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_gene_ensembl
</td>
<td style="text-align:left;">
Guinea Pig genes (Cavpor3.0)
</td>
<td style="text-align:left;">
Cavpor3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_gene_ensembl
</td>
<td style="text-align:left;">
Australian saltwater crocodile genes (CroPor_comp1)
</td>
<td style="text-align:left;">
CroPor_comp1
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_gene_ensembl
</td>
<td style="text-align:left;">
Vervet-AGM genes (ChlSab1.1)
</td>
<td style="text-align:left;">
ChlSab1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_gene_ensembl
</td>
<td style="text-align:left;">
C.savignyi genes (CSAV 2.0)
</td>
<td style="text-align:left;">
CSAV 2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_gene_ensembl
</td>
<td style="text-align:left;">
Tongue sole genes (Cse_v1.0)
</td>
<td style="text-align:left;">
Cse_v1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_gene_ensembl
</td>
<td style="text-align:left;">
Tarsier genes (Tarsius_syrichta-2.0.1)
</td>
<td style="text-align:left;">
Tarsius_syrichta-2.0.1
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_gene_ensembl
</td>
<td style="text-align:left;">
Sheepshead minnow genes (C_variegatus-1.0)
</td>
<td style="text-align:left;">
C_variegatus-1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_gene_ensembl
</td>
<td style="text-align:left;">
Chacoan peccary genes (CatWag_v2_BIUU_UCD)
</td>
<td style="text-align:left;">
CatWag_v2_BIUU_UCD
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_gene_ensembl
</td>
<td style="text-align:left;">
Denticle herring genes (fDenClu1.1)
</td>
<td style="text-align:left;">
fDenClu1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_gene_ensembl
</td>
<td style="text-align:left;">
European seabass genes (seabass_V1.0)
</td>
<td style="text-align:left;">
seabass_V1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_gene_ensembl
</td>
<td style="text-align:left;">
Beluga whale genes (ASM228892v3)
</td>
<td style="text-align:left;">
ASM228892v3
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_gene_ensembl
</td>
<td style="text-align:left;">
Drosophila melanogaster genes (BDGP6.32)
</td>
<td style="text-align:left;">
BDGP6.32
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_gene_ensembl
</td>
<td style="text-align:left;">
Armadillo genes (Dasnov3.0)
</td>
<td style="text-align:left;">
Dasnov3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_gene_ensembl
</td>
<td style="text-align:left;">
Kangaroo rat genes (Dord_2.0)
</td>
<td style="text-align:left;">
Dord_2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_gene_ensembl
</td>
<td style="text-align:left;">
Zebrafish genes (GRCz11)
</td>
<td style="text-align:left;">
GRCz11
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_gene_ensembl
</td>
<td style="text-align:left;">
Donkey genes (ASM303372v1)
</td>
<td style="text-align:left;">
ASM303372v1
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_gene_ensembl
</td>
<td style="text-align:left;">
Hagfish genes (Eburgeri_3.2)
</td>
<td style="text-align:left;">
Eburgeri_3.2
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_gene_ensembl
</td>
<td style="text-align:left;">
Horse genes (EquCab3.0)
</td>
<td style="text-align:left;">
EquCab3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_gene_ensembl
</td>
<td style="text-align:left;">
Reedfish genes (fErpCal1.1)
</td>
<td style="text-align:left;">
fErpCal1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_gene_ensembl
</td>
<td style="text-align:left;">
Electric eel genes (Ee_SOAP_WITH_SSPACE)
</td>
<td style="text-align:left;">
Ee_SOAP_WITH_SSPACE
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_gene_ensembl
</td>
<td style="text-align:left;">
Hedgehog genes (eriEur1)
</td>
<td style="text-align:left;">
eriEur1
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_gene_ensembl
</td>
<td style="text-align:left;">
Northern pike genes (Eluc_v4)
</td>
<td style="text-align:left;">
Eluc_v4
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_gene_ensembl
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec genes (TENREC)
</td>
<td style="text-align:left;">
TENREC
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_gene_ensembl
</td>
<td style="text-align:left;">
Collared flycatcher genes (FicAlb1.5)
</td>
<td style="text-align:left;">
FicAlb1.5
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_gene_ensembl
</td>
<td style="text-align:left;">
Cat genes (Felis_catus_9.0)
</td>
<td style="text-align:left;">
Felis_catus_9.0
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_gene_ensembl
</td>
<td style="text-align:left;">
Mummichog genes (Fundulus_heteroclitus-3.0.2)
</td>
<td style="text-align:left;">
Fundulus_heteroclitus-3.0.2
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_gene_ensembl
</td>
<td style="text-align:left;">
Stickleback genes (BROAD S1)
</td>
<td style="text-align:left;">
BROAD S1
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_gene_ensembl
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise genes (rGopEvg1_v1.p)
</td>
<td style="text-align:left;">
rGopEvg1_v1.p
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_gene_ensembl
</td>
<td style="text-align:left;">
Medium ground-finch genes (GeoFor_1.0)
</td>
<td style="text-align:left;">
GeoFor_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_gene_ensembl
</td>
<td style="text-align:left;">
Chicken genes (GRCg6a)
</td>
<td style="text-align:left;">
GRCg6a
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_gene_ensembl
</td>
<td style="text-align:left;">
Gorilla genes (gorGor4)
</td>
<td style="text-align:left;">
gorGor4
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_gene_ensembl
</td>
<td style="text-align:left;">
Atlantic cod genes (gadMor3.0)
</td>
<td style="text-align:left;">
gadMor3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_gene_ensembl
</td>
<td style="text-align:left;">
Burton’s mouthbrooder genes (AstBur1.0)
</td>
<td style="text-align:left;">
AstBur1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_gene_ensembl
</td>
<td style="text-align:left;">
Tiger tail seahorse genes (H_comes_QL1_v1)
</td>
<td style="text-align:left;">
H_comes_QL1_v1
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_gene_ensembl
</td>
<td style="text-align:left;">
Naked mole-rat female genes (HetGla_female_1.0)
</td>
<td style="text-align:left;">
HetGla_female_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_gene_ensembl
</td>
<td style="text-align:left;">
Huchen genes (ASM331708v1)
</td>
<td style="text-align:left;">
ASM331708v1
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_gene_ensembl
</td>
<td style="text-align:left;">
Human genes (GRCh38.p13)
</td>
<td style="text-align:left;">
GRCh38.p13
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_gene_ensembl
</td>
<td style="text-align:left;">
Channel catfish genes (IpCoco_1.2)
</td>
<td style="text-align:left;">
IpCoco_1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_gene_ensembl
</td>
<td style="text-align:left;">
Squirrel genes (SpeTri2.0)
</td>
<td style="text-align:left;">
SpeTri2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_gene_ensembl
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa genes (JacJac1.0)
</td>
<td style="text-align:left;">
JacJac1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_gene_ensembl
</td>
<td style="text-align:left;">
Mangrove rivulus genes (ASM164957v1)
</td>
<td style="text-align:left;">
ASM164957v1
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_gene_ensembl
</td>
<td style="text-align:left;">
Elephant genes (Loxafr3.0)
</td>
<td style="text-align:left;">
Loxafr3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_gene_ensembl
</td>
<td style="text-align:left;">
Ballan wrasse genes (BallGen_V1)
</td>
<td style="text-align:left;">
BallGen_V1
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_gene_ensembl
</td>
<td style="text-align:left;">
Barramundi perch genes (ASB_HGAPassembly_v1)
</td>
<td style="text-align:left;">
ASB_HGAPassembly_v1
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_gene_ensembl
</td>
<td style="text-align:left;">
Coelacanth genes (LatCha1)
</td>
<td style="text-align:left;">
LatCha1
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_gene_ensembl
</td>
<td style="text-align:left;">
Large yellow croaker genes (L_crocea_2.0)
</td>
<td style="text-align:left;">
L_crocea_2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_gene_ensembl
</td>
<td style="text-align:left;">
Blue-ringed sea krait genes (latLat_1.0)
</td>
<td style="text-align:left;">
latLat_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_gene_ensembl
</td>
<td style="text-align:left;">
Leishan spiny toad genes (ASM966780v1)
</td>
<td style="text-align:left;">
ASM966780v1
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_gene_ensembl
</td>
<td style="text-align:left;">
Spotted gar genes (LepOcu1)
</td>
<td style="text-align:left;">
LepOcu1
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_gene_ensembl
</td>
<td style="text-align:left;">
Zig-zag eel genes (fMasArm1.2)
</td>
<td style="text-align:left;">
fMasArm1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_gene_ensembl
</td>
<td style="text-align:left;">
Golden Hamster genes (MesAur1.0)
</td>
<td style="text-align:left;">
MesAur1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_gene_ensembl
</td>
<td style="text-align:left;">
Ryukyu mouse genes (CAROLI_EIJ_v1.1)
</td>
<td style="text-align:left;">
CAROLI_EIJ_v1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_gene_ensembl
</td>
<td style="text-align:left;">
Opossum genes (ASM229v1)
</td>
<td style="text-align:left;">
ASM229v1
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_gene_ensembl
</td>
<td style="text-align:left;">
Crab-eating macaque genes (Macaca_fascicularis_6.0)
</td>
<td style="text-align:left;">
Macaca_fascicularis_6.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_gene_ensembl
</td>
<td style="text-align:left;">
Turkey genes (Turkey_5.1)
</td>
<td style="text-align:left;">
Turkey_5.1
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_gene_ensembl
</td>
<td style="text-align:left;">
Drill genes (Mleu.le_1.0)
</td>
<td style="text-align:left;">
Mleu.le_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_gene_ensembl
</td>
<td style="text-align:left;">
Microbat genes (Myoluc2.0)
</td>
<td style="text-align:left;">
Myoluc2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_gene_ensembl
</td>
<td style="text-align:left;">
Alpine marmot genes (marMar2.1)
</td>
<td style="text-align:left;">
marMar2.1
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_gene_ensembl
</td>
<td style="text-align:left;">
Narwhal genes (NGI_Narwhal_1)
</td>
<td style="text-align:left;">
NGI_Narwhal_1
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_gene_ensembl
</td>
<td style="text-align:left;">
Siberian musk deer genes (MosMos_v2_BIUU_UCD)
</td>
<td style="text-align:left;">
MosMos_v2_BIUU_UCD
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_gene_ensembl
</td>
<td style="text-align:left;">
Macaque genes (Mmul_10)
</td>
<td style="text-align:left;">
Mmul_10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_gene_ensembl
</td>
<td style="text-align:left;">
Pinecone soldierfish genes (fMyrMur1.1)
</td>
<td style="text-align:left;">
fMyrMur1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_gene_ensembl
</td>
<td style="text-align:left;">
Mouse Lemur genes (Mmur_3.0)
</td>
<td style="text-align:left;">
Mmur_3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_gene_ensembl
</td>
<td style="text-align:left;">
Mouse genes (GRCm39)
</td>
<td style="text-align:left;">
GRCm39
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_gene_ensembl
</td>
<td style="text-align:left;">
Pig-tailed macaque genes (Mnem_1.0)
</td>
<td style="text-align:left;">
Mnem_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_gene_ensembl
</td>
<td style="text-align:left;">
Prairie vole genes (MicOch1.0)
</td>
<td style="text-align:left;">
MicOch1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_gene_ensembl
</td>
<td style="text-align:left;">
Shrew mouse genes (PAHARI_EIJ_v1.1)
</td>
<td style="text-align:left;">
PAHARI_EIJ_v1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_gene_ensembl
</td>
<td style="text-align:left;">
Ferret genes (MusPutFur1.0)
</td>
<td style="text-align:left;">
MusPutFur1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_gene_ensembl
</td>
<td style="text-align:left;">
Steppe mouse genes (MUSP714)
</td>
<td style="text-align:left;">
MUSP714
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_gene_ensembl
</td>
<td style="text-align:left;">
Algerian mouse genes (SPRET_EiJ_v1)
</td>
<td style="text-align:left;">
SPRET_EiJ_v1
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_gene_ensembl
</td>
<td style="text-align:left;">
Zebra mbuna genes (M_zebra_UMD2a)
</td>
<td style="text-align:left;">
M_zebra_UMD2a
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_gene_ensembl
</td>
<td style="text-align:left;">
Lyretail cichlid genes (NeoBri1.0)
</td>
<td style="text-align:left;">
NeoBri1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_gene_ensembl
</td>
<td style="text-align:left;">
Wallaby genes (Meug_1.0)
</td>
<td style="text-align:left;">
Meug_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_gene_ensembl
</td>
<td style="text-align:left;">
Turquoise killifish genes (Nfu_20140520)
</td>
<td style="text-align:left;">
Nfu_20140520
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_gene_ensembl
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat genes (S.galili_v1.0)
</td>
<td style="text-align:left;">
S.galili_v1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_gene_ensembl
</td>
<td style="text-align:left;">
Gibbon genes (Nleu_3.0)
</td>
<td style="text-align:left;">
Nleu_3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_gene_ensembl
</td>
<td style="text-align:left;">
Indian cobra genes (Nana_v5)
</td>
<td style="text-align:left;">
Nana_v5
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_gene_ensembl
</td>
<td style="text-align:left;">
Mainland tiger snake genes (TS10Xv2-PRI)
</td>
<td style="text-align:left;">
TS10Xv2-PRI
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_gene_ensembl
</td>
<td style="text-align:left;">
American mink genes (NNQGG.v01)
</td>
<td style="text-align:left;">
NNQGG.v01
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_gene_ensembl
</td>
<td style="text-align:left;">
Platypus genes (mOrnAna1.p.v1)
</td>
<td style="text-align:left;">
mOrnAna1.p.v1
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_gene_ensembl
</td>
<td style="text-align:left;">
Sheep genes (Oar_rambouillet_v1.0)
</td>
<td style="text-align:left;">
Oar_rambouillet_v1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
oaries_gene_ensembl
</td>
<td style="text-align:left;">
Sheep (texel) genes (Oar_v3.1)
</td>
<td style="text-align:left;">
Oar_v3.1
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_gene_ensembl
</td>
<td style="text-align:left;">
Rabbit genes (OryCun2.0)
</td>
<td style="text-align:left;">
OryCun2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_gene_ensembl
</td>
<td style="text-align:left;">
Degu genes (OctDeg1.0)
</td>
<td style="text-align:left;">
OctDeg1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_gene_ensembl
</td>
<td style="text-align:left;">
Bushbaby genes (OtoGar3)
</td>
<td style="text-align:left;">
OtoGar3
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_gene_ensembl
</td>
<td style="text-align:left;">
Javanese ricefish genes (OJAV_1.1)
</td>
<td style="text-align:left;">
OJAV_1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_gene_ensembl
</td>
<td style="text-align:left;">
Coho salmon genes (Okis_V2)
</td>
<td style="text-align:left;">
Okis_V2
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_gene_ensembl
</td>
<td style="text-align:left;">
Japanese medaka HdrR genes (ASM223467v1)
</td>
<td style="text-align:left;">
ASM223467v1
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_gene_ensembl
</td>
<td style="text-align:left;">
Indian medaka genes (Om_v0.7.RACA)
</td>
<td style="text-align:left;">
Om_v0.7.RACA
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_gene_ensembl
</td>
<td style="text-align:left;">
Rainbow trout genes (Omyk_1.0)
</td>
<td style="text-align:left;">
Omyk_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_gene_ensembl
</td>
<td style="text-align:left;">
Nile tilapia genes (O_niloticus_UMD_NMBU)
</td>
<td style="text-align:left;">
O_niloticus_UMD_NMBU
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_gene_ensembl
</td>
<td style="text-align:left;">
Pika genes (OchPri2.0-Ens)
</td>
<td style="text-align:left;">
OchPri2.0-Ens
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_gene_ensembl
</td>
<td style="text-align:left;">
Chinese medaka genes (ASM858656v1)
</td>
<td style="text-align:left;">
ASM858656v1
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_gene_ensembl
</td>
<td style="text-align:left;">
Chinook salmon genes (Otsh_v1.0)
</td>
<td style="text-align:left;">
Otsh_v1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_gene_ensembl
</td>
<td style="text-align:left;">
Orangutan genes (PPYG2)
</td>
<td style="text-align:left;">
PPYG2
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_gene_ensembl
</td>
<td style="text-align:left;">
Olive baboon genes (Panu_3.0)
</td>
<td style="text-align:left;">
Panu_3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_gene_ensembl
</td>
<td style="text-align:left;">
Hyrax genes (proCap1)
</td>
<td style="text-align:left;">
proCap1
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_gene_ensembl
</td>
<td style="text-align:left;">
Sperm whale genes (ASM283717v2)
</td>
<td style="text-align:left;">
ASM283717v2
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_gene_ensembl
</td>
<td style="text-align:left;">
Koala genes (phaCin_unsw_v4.1)
</td>
<td style="text-align:left;">
phaCin_unsw_v4.1
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_gene_ensembl
</td>
<td style="text-align:left;">
Coquerel’s sifaka genes (Pcoq_1.0)
</td>
<td style="text-align:left;">
Pcoq_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_gene_ensembl
</td>
<td style="text-align:left;">
Amazon molly genes (Poecilia_formosa-5.1.2)
</td>
<td style="text-align:left;">
Poecilia_formosa-5.1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_gene_ensembl
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae genes (PKINGS_0.1)
</td>
<td style="text-align:left;">
PKINGS_0.1
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_gene_ensembl
</td>
<td style="text-align:left;">
Sailfin molly genes (P_latipinna-1.0)
</td>
<td style="text-align:left;">
P_latipinna-1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_gene_ensembl
</td>
<td style="text-align:left;">
Lion genes (PanLeo1.0)
</td>
<td style="text-align:left;">
PanLeo1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_gene_ensembl
</td>
<td style="text-align:left;">
Great Tit genes (Parus_major1.1)
</td>
<td style="text-align:left;">
Parus_major1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_gene_ensembl
</td>
<td style="text-align:left;">
Lamprey genes (Pmarinus_7.0)
</td>
<td style="text-align:left;">
Pmarinus_7.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_gene_ensembl
</td>
<td style="text-align:left;">
Northern American deer mouse genes (HU_Pman_2.1)
</td>
<td style="text-align:left;">
HU_Pman_2.1
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_gene_ensembl
</td>
<td style="text-align:left;">
Common wall lizard genes (PodMur_1.0)
</td>
<td style="text-align:left;">
PodMur_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_gene_ensembl
</td>
<td style="text-align:left;">
Red-bellied piranha genes (Pygocentrus_nattereri-1.0.2)
</td>
<td style="text-align:left;">
Pygocentrus_nattereri-1.0.2
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_gene_ensembl
</td>
<td style="text-align:left;">
Makobe Island cichlid genes (PunNye1.0)
</td>
<td style="text-align:left;">
PunNye1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_gene_ensembl
</td>
<td style="text-align:left;">
Bonobo genes (panpan1.1)
</td>
<td style="text-align:left;">
panpan1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_gene_ensembl
</td>
<td style="text-align:left;">
Leopard genes (PanPar1.0)
</td>
<td style="text-align:left;">
PanPar1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_gene_ensembl
</td>
<td style="text-align:left;">
Guppy genes (Guppy_female_1.0_MT)
</td>
<td style="text-align:left;">
Guppy_female_1.0_MT
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_gene_ensembl
</td>
<td style="text-align:left;">
Greater bamboo lemur genes (Prosim_1.0)
</td>
<td style="text-align:left;">
Prosim_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_gene_ensembl
</td>
<td style="text-align:left;">
Chinese softshell turtle genes (PelSin_1.0)
</td>
<td style="text-align:left;">
PelSin_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_gene_ensembl
</td>
<td style="text-align:left;">
Vaquita genes (mPhoSin1.pri)
</td>
<td style="text-align:left;">
mPhoSin1.pri
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_gene_ensembl
</td>
<td style="text-align:left;">
Tiger genes (PanTig1.0)
</td>
<td style="text-align:left;">
PanTig1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_gene_ensembl
</td>
<td style="text-align:left;">
Eastern brown snake genes (EBS10Xv2-PRI)
</td>
<td style="text-align:left;">
EBS10Xv2-PRI
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_gene_ensembl
</td>
<td style="text-align:left;">
Chimpanzee genes (Pan_tro_3.0)
</td>
<td style="text-align:left;">
Pan_tro_3.0
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_gene_ensembl
</td>
<td style="text-align:left;">
Megabat genes (pteVam1)
</td>
<td style="text-align:left;">
pteVam1
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_gene_ensembl
</td>
<td style="text-align:left;">
Black snub-nosed monkey genes (ASM169854v1)
</td>
<td style="text-align:left;">
ASM169854v1
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_gene_ensembl
</td>
<td style="text-align:left;">
Greater horseshoe bat genes (mRhiFer1_v1.p)
</td>
<td style="text-align:left;">
mRhiFer1_v1.p
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_gene_ensembl
</td>
<td style="text-align:left;">
Rat genes (Rnor_6.0)
</td>
<td style="text-align:left;">
Rnor_6.0
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_gene_ensembl
</td>
<td style="text-align:left;">
Golden snub-nosed monkey genes (Rrox_v1)
</td>
<td style="text-align:left;">
Rrox_v1
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_gene_ensembl
</td>
<td style="text-align:left;">
Shrew genes (sorAra1)
</td>
<td style="text-align:left;">
sorAra1
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_gene_ensembl
</td>
<td style="text-align:left;">
Gilthead seabream genes (fSpaAur1.1)
</td>
<td style="text-align:left;">
fSpaAur1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_gene_ensembl
</td>
<td style="text-align:left;">
Bolivian squirrel monkey genes (SaiBol1.0)
</td>
<td style="text-align:left;">
SaiBol1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_gene_ensembl
</td>
<td style="text-align:left;">
Common canary genes (SCA1)
</td>
<td style="text-align:left;">
SCA1
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_gene_ensembl
</td>
<td style="text-align:left;">
African ostrich genes (ASM69896v1)
</td>
<td style="text-align:left;">
ASM69896v1
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_gene_ensembl
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae genes (R64-1-1)
</td>
<td style="text-align:left;">
R64-1-1
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_gene_ensembl
</td>
<td style="text-align:left;">
Greater amberjack genes (Sdu_1.0)
</td>
<td style="text-align:left;">
Sdu_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_gene_ensembl
</td>
<td style="text-align:left;">
Asian bonytongue genes (fSclFor1.1)
</td>
<td style="text-align:left;">
fSclFor1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_gene_ensembl
</td>
<td style="text-align:left;">
Golden-line barbel genes (SAMN03320097.WGS_v1.1)
</td>
<td style="text-align:left;">
SAMN03320097.WGS_v1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_gene_ensembl
</td>
<td style="text-align:left;">
Kakapo genes (bStrHab1_v1.p)
</td>
<td style="text-align:left;">
bStrHab1_v1.p
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_gene_ensembl
</td>
<td style="text-align:left;">
Tasmanian devil genes (mSarHar1.11)
</td>
<td style="text-align:left;">
mSarHar1.11
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_gene_ensembl
</td>
<td style="text-align:left;">
Yellowtail amberjack genes (Sedor1)
</td>
<td style="text-align:left;">
Sedor1
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_gene_ensembl
</td>
<td style="text-align:left;">
Pike-perch genes (SLUC_FBN_1)
</td>
<td style="text-align:left;">
SLUC_FBN_1
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_gene_ensembl
</td>
<td style="text-align:left;">
Turbot genes (ASM1334776v1)
</td>
<td style="text-align:left;">
ASM1334776v1
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_gene_ensembl
</td>
<td style="text-align:left;">
Argentine black and white tegu genes (HLtupMer3)
</td>
<td style="text-align:left;">
HLtupMer3
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_gene_ensembl
</td>
<td style="text-align:left;">
Bicolor damselfish genes (Stegastes_partitus-1.0.2)
</td>
<td style="text-align:left;">
Stegastes_partitus-1.0.2
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_gene_ensembl
</td>
<td style="text-align:left;">
Tuatara genes (ASM311381v1)
</td>
<td style="text-align:left;">
ASM311381v1
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_gene_ensembl
</td>
<td style="text-align:left;">
Atlantic salmon genes (ICSASG_v2)
</td>
<td style="text-align:left;">
ICSASG_v2
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_gene_ensembl
</td>
<td style="text-align:left;">
Pig genes (Sscrofa11.1)
</td>
<td style="text-align:left;">
Sscrofa11.1
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_gene_ensembl
</td>
<td style="text-align:left;">
Brown trout genes (fSalTru1.1)
</td>
<td style="text-align:left;">
fSalTru1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_gene_ensembl
</td>
<td style="text-align:left;">
Eurasian red squirrel genes (mSciVul1.1)
</td>
<td style="text-align:left;">
mSciVul1.1
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_gene_ensembl
</td>
<td style="text-align:left;">
Tree Shrew genes (tupBel1)
</td>
<td style="text-align:left;">
tupBel1
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_gene_ensembl
</td>
<td style="text-align:left;">
Three-toed box turtle genes (T_m_triunguis-2.0)
</td>
<td style="text-align:left;">
T_m_triunguis-2.0
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_gene_ensembl
</td>
<td style="text-align:left;">
Zebra finch genes (bTaeGut1_v1.p)
</td>
<td style="text-align:left;">
bTaeGut1_v1.p
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_gene_ensembl
</td>
<td style="text-align:left;">
Tetraodon genes (TETRAODON 8.0)
</td>
<td style="text-align:left;">
TETRAODON 8.0
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_gene_ensembl
</td>
<td style="text-align:left;">
Fugu genes (fTakRub1.2)
</td>
<td style="text-align:left;">
fTakRub1.2
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_gene_ensembl
</td>
<td style="text-align:left;">
Dolphin genes (turTru1)
</td>
<td style="text-align:left;">
turTru1
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_gene_ensembl
</td>
<td style="text-align:left;">
American black bear genes (ASM334442v1)
</td>
<td style="text-align:left;">
ASM334442v1
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_gene_ensembl
</td>
<td style="text-align:left;">
Polar bear genes (UrsMar_1.0)
</td>
<td style="text-align:left;">
UrsMar_1.0
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_gene_ensembl
</td>
<td style="text-align:left;">
Arctic ground squirrel genes (ASM342692v1)
</td>
<td style="text-align:left;">
ASM342692v1
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_gene_ensembl
</td>
<td style="text-align:left;">
Alpaca genes (vicPac1)
</td>
<td style="text-align:left;">
vicPac1
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_gene_ensembl
</td>
<td style="text-align:left;">
Common wombat genes (bare-nosed_wombat_genome_assembly)
</td>
<td style="text-align:left;">
bare-nosed_wombat_genome_assembly
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_gene_ensembl
</td>
<td style="text-align:left;">
Red fox genes (VulVul2.2)
</td>
<td style="text-align:left;">
VulVul2.2
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_gene_ensembl
</td>
<td style="text-align:left;">
Platyfish genes (X_maculatus-5.0-male)
</td>
<td style="text-align:left;">
X_maculatus-5.0-male
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_gene_ensembl
</td>
<td style="text-align:left;">
Tropical clawed frog genes (Xenopus_tropicalis_v9.1)
</td>
<td style="text-align:left;">
Xenopus_tropicalis_v9.1
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Since we are mapping human genes, we filter datasets with the keyword “sapiens”:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>datasets</span> <span class='op'>&lt;-</span> <span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listDatasets.html'>listDatasets</a></span><span class='op'>(</span><span class='va'>ensembl</span><span class='op'>)</span>
<span class='va'>datasets</span> <span class='op'>&lt;-</span> <span class='va'>datasets</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/grep.html'>grep</a></span><span class='op'>(</span><span class='va'>datasets</span><span class='op'>$</span><span class='va'>dataset</span>, pattern <span class='op'>=</span> <span class='st'>"sapiens"</span><span class='op'>)</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<p>Or alternatively, use the built-in biomaRt wrapper to filter datasets:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>datasets</span> <span class='op'>&lt;-</span> <span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listDatasets.html'>searchDatasets</a></span><span class='op'>(</span>mart <span class='op'>=</span> <span class='va'>ensembl</span>, pattern <span class='op'>=</span> <span class='st'>"sapiens"</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:left;">
dataset
</th>
<th style="text-align:left;">
description
</th>
<th style="text-align:left;">
version
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
hsapiens_gene_ensembl
</td>
<td style="text-align:left;">
Human genes (GRCh38.p13)
</td>
<td style="text-align:left;">
GRCh38.p13
</td>
</tr>
</tbody>
</table>
</div>
<p>And we found the dataset we need to use for mapping. To use this dataset:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'># Choose the dataset for Homo sapiens</span>
<span class='va'>ensembl</span> <span class='op'>&lt;-</span> <span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/useDataset.html'>useDataset</a></span><span class='op'>(</span>mart    <span class='op'>=</span> <span class='va'>ensembl</span>,
                               dataset <span class='op'>=</span> <span class='st'>"hsapiens_gene_ensembl"</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<h3 id="biomart-query">biomaRt Query</h3>
<p>A biomaRt query consists of <strong>filters</strong>, <strong>values</strong>, and <strong>attributes</strong>, which define restrictions on the query. Specifically, values are the actual information to query about, which in our case is the Entrez gene IDs in GSE152641, while filters define the scope of query, e.g. Entrez gene ID in this case. For filters available in the chosen dataset of an Ensembl mart:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listFilters.html'>listFilters</a></span><span class='op'>(</span>mart <span class='op'>=</span> <span class='va'>ensembl</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:300px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
</tr>
<tr>
<td style="text-align:left;">
start
</td>
<td style="text-align:left;">
Start
</td>
</tr>
<tr>
<td style="text-align:left;">
end
</td>
<td style="text-align:left;">
End
</td>
</tr>
<tr>
<td style="text-align:left;">
band_start
</td>
<td style="text-align:left;">
Band Start
</td>
</tr>
<tr>
<td style="text-align:left;">
band_end
</td>
<td style="text-align:left;">
Band End
</td>
</tr>
<tr>
<td style="text-align:left;">
marker_start
</td>
<td style="text-align:left;">
Marker Start
</td>
</tr>
<tr>
<td style="text-align:left;">
marker_end
</td>
<td style="text-align:left;">
Marker End
</td>
</tr>
<tr>
<td style="text-align:left;">
encode_region
</td>
<td style="text-align:left;">
Encode region
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosomal_region
</td>
<td style="text-align:left;">
e.g. 1:100:10000:-1, 1:100000:200000:1
</td>
</tr>
<tr>
<td style="text-align:left;">
with_biogrid
</td>
<td style="text-align:left;">
With BioGRID Interaction data, The General Repository for Interaction Datasets ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ccds
</td>
<td style="text-align:left;">
With CCDS ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_chembl
</td>
<td style="text-align:left;">
With ChEMBL ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dbass3
</td>
<td style="text-align:left;">
With DataBase of Aberrant 3’ Splice Sites ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dbass5
</td>
<td style="text-align:left;">
With DataBase of Aberrant 5’ Splice Sites ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ens_hs_transcript
</td>
<td style="text-align:left;">
With Ensembl Human Transcript ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ens_hs_translation
</td>
<td style="text-align:left;">
With Ensembl Human Translation ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_entrezgene_trans_name
</td>
<td style="text-align:left;">
With EntrezGene transcript name ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_embl
</td>
<td style="text-align:left;">
With European Nucleotide Archive ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_arrayexpress
</td>
<td style="text-align:left;">
With Expression Atlas ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_genedb
</td>
<td style="text-align:left;">
With GeneDB ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_go
</td>
<td style="text-align:left;">
With GO ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_goslim_goa
</td>
<td style="text-align:left;">
With GOSlim GOA ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hgnc
</td>
<td style="text-align:left;">
With HGNC Symbol ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hpa
</td>
<td style="text-align:left;">
With Human Protein Atlas ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_protein_id
</td>
<td style="text-align:left;">
With INSDC protein ID ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_kegg_enzyme
</td>
<td style="text-align:left;">
With KEGG Pathway and Enzyme ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ens_lrg_gene
</td>
<td style="text-align:left;">
With LRG display in Ensembl gene ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ens_lrg_transcript
</td>
<td style="text-align:left;">
With LRG display in Ensembl transcript ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_merops
</td>
<td style="text-align:left;">
With MEROPS - the Peptidase Database ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mim_gene
</td>
<td style="text-align:left;">
With MIM gene ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mim_morbid
</td>
<td style="text-align:left;">
With MIM morbid ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mirbase
</td>
<td style="text-align:left;">
With miRBase ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mirbase_trans_name
</td>
<td style="text-align:left;">
With miRBase transcript name ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_entrezgene
</td>
<td style="text-align:left;">
With NCBI gene (formerly Entrezgene) ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pdb
</td>
<td style="text-align:left;">
With PDB ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_reactome
</td>
<td style="text-align:left;">
With Reactome ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_reactome_gene
</td>
<td style="text-align:left;">
With Reactome gene ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_reactome_transcript
</td>
<td style="text-align:left;">
With Reactome transcript ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_refseq_mrna
</td>
<td style="text-align:left;">
With RefSeq mRNA ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_refseq_mrna_predicted
</td>
<td style="text-align:left;">
With RefSeq mRNA predicted ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_refseq_ncrna
</td>
<td style="text-align:left;">
With RefSeq ncRNA ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_refseq_ncrna_predicted
</td>
<td style="text-align:left;">
With RefSeq ncRNA predicted ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_refseq_peptide
</td>
<td style="text-align:left;">
With RefSeq peptide ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_refseq_peptide_predicted
</td>
<td style="text-align:left;">
With RefSeq peptide predicted ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rfam
</td>
<td style="text-align:left;">
With RFAM ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rfam_trans_name
</td>
<td style="text-align:left;">
With RFAM transcript name ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rnacentral
</td>
<td style="text-align:left;">
With RNAcentral ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hgnc_trans_name
</td>
<td style="text-align:left;">
With Transcript name ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ucsc
</td>
<td style="text-align:left;">
With UCSC Stable ID ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uniparc
</td>
<td style="text-align:left;">
With UniParc ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uniprot_gn
</td>
<td style="text-align:left;">
With UniProtKB Gene Name ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uniprot_isoform
</td>
<td style="text-align:left;">
With UniProtKB isoform ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uniprotswissprot
</td>
<td style="text-align:left;">
With UniProtKB/Swiss-Prot ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uniprotsptrembl
</td>
<td style="text-align:left;">
With UniProtKB/TrEMBL ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_wikigene
</td>
<td style="text-align:left;">
With WikiGene ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID(s) [e.g. ENSG00000000003]
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID(s) with version [e.g. ENSG00000000003.15]
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID(s) [e.g. ENST00000000233]
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID(s) with version [e.g. ENST00000000233.10]
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID(s) [e.g. ENSP00000000233]
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID(s) with version [e.g. ENSP00000000233.5]
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_exon_id
</td>
<td style="text-align:left;">
Exon ID(s) [e.g. ENSE00000000003]
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene Name(s) [e.g. MT-TF]
</td>
</tr>
<tr>
<td style="text-align:left;">
external_transcript_name
</td>
<td style="text-align:left;">
Transcript Name(s) [e.g. MT-TF-201]
</td>
</tr>
<tr>
<td style="text-align:left;">
external_synonym
</td>
<td style="text-align:left;">
Gene Synonym(s) [e.g. 0808y08y]
</td>
</tr>
<tr>
<td style="text-align:left;">
biogrid
</td>
<td style="text-align:left;">
BioGRID Interaction data, The General Repository for Interaction Datasets ID(s) [e.g. 106523]
</td>
</tr>
<tr>
<td style="text-align:left;">
ccds
</td>
<td style="text-align:left;">
CCDS ID(s) [e.g. CCDS10]
</td>
</tr>
<tr>
<td style="text-align:left;">
chembl
</td>
<td style="text-align:left;">
ChEMBL ID(s) [e.g. CHEMBL1075092]
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass3_name
</td>
<td style="text-align:left;">
DataBase of Aberrant 3’ Splice Sites name(s) [e.g. ABCR ]
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass3_id
</td>
<td style="text-align:left;">
DataBase of Aberrant 3’ Splice Sites ID(s) [e.g. 1]
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass5_name
</td>
<td style="text-align:left;">
DataBase of Aberrant 5’ Splice Sites name(s) [e.g. BRCA1]
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass5_id
</td>
<td style="text-align:left;">
DataBase of Aberrant 5’ Splice Sites ID(s) [e.g. 382]
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_hs_transcript
</td>
<td style="text-align:left;">
Ensembl Human Transcript ID(s) [e.g. ENST00000200639]
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_hs_translation
</td>
<td style="text-align:left;">
Ensembl Human Translation ID(s) [e.g. ENSP00000200639]
</td>
</tr>
<tr>
<td style="text-align:left;">
entrezgene_trans_name
</td>
<td style="text-align:left;">
EntrezGene transcript name ID(s) [e.g. AA06-201]
</td>
</tr>
<tr>
<td style="text-align:left;">
embl
</td>
<td style="text-align:left;">
European Nucleotide Archive ID(s) [e.g. A06800]
</td>
</tr>
<tr>
<td style="text-align:left;">
arrayexpress
</td>
<td style="text-align:left;">
Expression Atlas ID(s) [e.g. ENSG00000000003]
</td>
</tr>
<tr>
<td style="text-align:left;">
genedb
</td>
<td style="text-align:left;">
GeneDB ID(s) [e.g. Smp_009580.1:pep]
</td>
</tr>
<tr>
<td style="text-align:left;">
go
</td>
<td style="text-align:left;">
GO ID(s) [e.g. <a href="GO:0000002" class="uri">GO:0000002</a>]
</td>
</tr>
<tr>
<td style="text-align:left;">
goslim_goa
</td>
<td style="text-align:left;">
GOSlim GOA ID(s) [e.g. <a href="GO:0000003" class="uri">GO:0000003</a>]
</td>
</tr>
<tr>
<td style="text-align:left;">
hgnc_id
</td>
<td style="text-align:left;">
HGNC ID(s) [e.g. HGNC:100]
</td>
</tr>
<tr>
<td style="text-align:left;">
hgnc_symbol
</td>
<td style="text-align:left;">
HGNC symbol(s) [e.g. A1BG]
</td>
</tr>
<tr>
<td style="text-align:left;">
hpa_accession
</td>
<td style="text-align:left;">
Human Protein Atlas accession(s) [e.g. CAB000001]
</td>
</tr>
<tr>
<td style="text-align:left;">
hpa_id
</td>
<td style="text-align:left;">
Human Protein Atlas ID(s) [e.g. 1]
</td>
</tr>
<tr>
<td style="text-align:left;">
protein_id
</td>
<td style="text-align:left;">
INSDC protein ID(s) [e.g. AAA02489]
</td>
</tr>
<tr>
<td style="text-align:left;">
kegg_enzyme
</td>
<td style="text-align:left;">
KEGG Pathway and Enzyme ID(s) [e.g. 00010]
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_lrg_gene
</td>
<td style="text-align:left;">
LRG display in Ensembl gene ID(s) [e.g. LRG_1]
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_lrg_transcript
</td>
<td style="text-align:left;">
LRG display in Ensembl transcript ID(s) [e.g. LRG_100t1]
</td>
</tr>
<tr>
<td style="text-align:left;">
merops
</td>
<td style="text-align:left;">
MEROPS - the Peptidase Database ID(s) [e.g. A01.001]
</td>
</tr>
<tr>
<td style="text-align:left;">
mim_gene_accession
</td>
<td style="text-align:left;">
MIM gene accession(s) [e.g. 100640]
</td>
</tr>
<tr>
<td style="text-align:left;">
mim_morbid_accession
</td>
<td style="text-align:left;">
MIM morbid accession(s) [e.g. 100100]
</td>
</tr>
<tr>
<td style="text-align:left;">
mirbase_accession
</td>
<td style="text-align:left;">
miRBase accession(s) [e.g. MI0000060]
</td>
</tr>
<tr>
<td style="text-align:left;">
mirbase_id
</td>
<td style="text-align:left;">
miRBase ID(s) [e.g. hsa-let-7a-1]
</td>
</tr>
<tr>
<td style="text-align:left;">
mirbase_trans_name
</td>
<td style="text-align:left;">
miRBase transcript name ID(s) [e.g. hsa-mir-1253.1-201]
</td>
</tr>
<tr>
<td style="text-align:left;">
entrezgene_accession
</td>
<td style="text-align:left;">
NCBI gene (formerly Entrezgene) accession(s) [e.g. A1BG]
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
entrezgene_id
</td>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
NCBI gene (formerly Entrezgene) ID(s) [e.g. 1]
</td>
</tr>
<tr>
<td style="text-align:left;">
pdb
</td>
<td style="text-align:left;">
PDB ID(s) [e.g. 10GS]
</td>
</tr>
<tr>
<td style="text-align:left;">
reactome
</td>
<td style="text-align:left;">
Reactome ID(s) [e.g. R-HSA-1059683]
</td>
</tr>
<tr>
<td style="text-align:left;">
reactome_gene
</td>
<td style="text-align:left;">
Reactome gene ID(s) [e.g. R-HSA-1059683]
</td>
</tr>
<tr>
<td style="text-align:left;">
reactome_transcript
</td>
<td style="text-align:left;">
Reactome transcript ID(s) [e.g. R-HSA-1059683]
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_mrna
</td>
<td style="text-align:left;">
RefSeq mRNA ID(s) [e.g. NM_000014]
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_mrna_predicted
</td>
<td style="text-align:left;">
RefSeq mRNA predicted ID(s) [e.g. XM_003118719]
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_ncrna
</td>
<td style="text-align:left;">
RefSeq ncRNA ID(s) [e.g. NR_000005]
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_ncrna_predicted
</td>
<td style="text-align:left;">
RefSeq ncRNA predicted ID(s) [e.g. XR_001736900]
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_peptide
</td>
<td style="text-align:left;">
RefSeq peptide ID(s) [e.g. NP_000005]
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_peptide_predicted
</td>
<td style="text-align:left;">
RefSeq peptide predicted ID(s) [e.g. XP_003118767]
</td>
</tr>
<tr>
<td style="text-align:left;">
rfam
</td>
<td style="text-align:left;">
RFAM ID(s) [e.g. RF00001]
</td>
</tr>
<tr>
<td style="text-align:left;">
rfam_trans_name
</td>
<td style="text-align:left;">
RFAM transcript name ID(s) [e.g. 5S_rRNA.1-201]
</td>
</tr>
<tr>
<td style="text-align:left;">
rnacentral
</td>
<td style="text-align:left;">
RNAcentral ID(s) [e.g. URS0000000055]
</td>
</tr>
<tr>
<td style="text-align:left;">
hgnc_trans_name
</td>
<td style="text-align:left;">
Transcript name ID(s) [e.g. A1BG-201]
</td>
</tr>
<tr>
<td style="text-align:left;">
ucsc
</td>
<td style="text-align:left;">
UCSC Stable ID(s) [e.g. uc001aak.4]
</td>
</tr>
<tr>
<td style="text-align:left;">
uniparc
</td>
<td style="text-align:left;">
UniParc ID(s) [e.g. UPI000000003C]
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprot_gn_symbol
</td>
<td style="text-align:left;">
UniProtKB Gene Name symbol(s) [e.g. 5HT1A]
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprot_gn_id
</td>
<td style="text-align:left;">
UniProtKB Gene Name ID(s) [e.g. A0A023T6R1]
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprot_isoform
</td>
<td style="text-align:left;">
UniProtKB isoform ID(s) [e.g. A0A096LP49-1]
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprotswissprot
</td>
<td style="text-align:left;">
UniProtKB/Swiss-Prot ID(s) [e.g. A0A024RBG1]
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprotsptrembl
</td>
<td style="text-align:left;">
UniProtKB/TrEMBL ID(s) [e.g. A0A023T6R1]
</td>
</tr>
<tr>
<td style="text-align:left;">
wikigene_name
</td>
<td style="text-align:left;">
WikiGene name(s) [e.g. A1BG]
</td>
</tr>
<tr>
<td style="text-align:left;">
wikigene_id
</td>
<td style="text-align:left;">
WikiGene ID(s) [e.g. 1]
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hc_g110
</td>
<td style="text-align:left;">
With AFFY HC G110 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_focus
</td>
<td style="text-align:left;">
With AFFY HG Focus probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u133a
</td>
<td style="text-align:left;">
With AFFY HG U133A probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u133a_2
</td>
<td style="text-align:left;">
With AFFY HG U133A 2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u133b
</td>
<td style="text-align:left;">
With AFFY HG U133B probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u133_plus_2
</td>
<td style="text-align:left;">
With AFFY HG U133 Plus 2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u95a
</td>
<td style="text-align:left;">
With AFFY HG U95A probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u95av2
</td>
<td style="text-align:left;">
With AFFY HG U95Av2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u95b
</td>
<td style="text-align:left;">
With AFFY HG U95B probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u95c
</td>
<td style="text-align:left;">
With AFFY HG U95C probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u95d
</td>
<td style="text-align:left;">
With AFFY HG U95D probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hg_u95e
</td>
<td style="text-align:left;">
With AFFY HG U95E probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hta_2_0
</td>
<td style="text-align:left;">
With AFFY HTA 2 0 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_huex_1_0_st_v2
</td>
<td style="text-align:left;">
With AFFY HuEx 1 0 st v2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hugenefl
</td>
<td style="text-align:left;">
With AFFY HuGeneFL probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hugene_1_0_st_v1
</td>
<td style="text-align:left;">
With AFFY HuGene 1 0 st v1 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_hugene_2_0_st_v1
</td>
<td style="text-align:left;">
With AFFY HuGene 2 0 st v1 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_primeview
</td>
<td style="text-align:left;">
With AFFY PrimeView probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_affy_u133_x3p
</td>
<td style="text-align:left;">
With AFFY U133 X3P probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_cgh_44b
</td>
<td style="text-align:left;">
With AGILENT CGH 44b probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_gpl26966
</td>
<td style="text-align:left;">
With AGILENT GPL26966 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_gpl6848
</td>
<td style="text-align:left;">
With AGILENT GPL6848 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_sureprint_g3_ge_8x60k
</td>
<td style="text-align:left;">
With AGILENT SurePrint G3 GE 8x60k probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_sureprint_g3_ge_8x60k_v2
</td>
<td style="text-align:left;">
With AGILENT SurePrint G3 GE 8x60k v2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_wholegenome
</td>
<td style="text-align:left;">
With AGILENT WholeGenome probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_wholegenome_4x44k_v1
</td>
<td style="text-align:left;">
With AGILENT WholeGenome 4x44k v1 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_agilent_wholegenome_4x44k_v2
</td>
<td style="text-align:left;">
With AGILENT WholeGenome 4x44k v2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_codelink_codelink
</td>
<td style="text-align:left;">
With CODELINK CODELINK probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_illumina_humanht_12_v3
</td>
<td style="text-align:left;">
With ILLUMINA HumanHT 12 V3 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_illumina_humanht_12_v4
</td>
<td style="text-align:left;">
With ILLUMINA HumanHT 12 V4 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_illumina_humanref_8_v3
</td>
<td style="text-align:left;">
With ILLUMINA HumanRef 8 V3 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_illumina_humanwg_6_v1
</td>
<td style="text-align:left;">
With ILLUMINA HumanWG 6 V1 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_illumina_humanwg_6_v2
</td>
<td style="text-align:left;">
With ILLUMINA HumanWG 6 V2 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_illumina_humanwg_6_v3
</td>
<td style="text-align:left;">
With ILLUMINA HumanWG 6 V3 probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_phalanx_onearray
</td>
<td style="text-align:left;">
With PHALANX OneArray probe ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hc_g110
</td>
<td style="text-align:left;">
AFFY HC G110 probe ID(s) [e.g. 737_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_focus
</td>
<td style="text-align:left;">
AFFY HG Focus probe ID(s) [e.g. 220771_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133a
</td>
<td style="text-align:left;">
AFFY HG U133A probe ID(s) [e.g. 211600_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133a_2
</td>
<td style="text-align:left;">
AFFY HG U133A 2 probe ID(s) [e.g. 211600_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133b
</td>
<td style="text-align:left;">
AFFY HG U133B probe ID(s) [e.g. 224372_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133_plus_2
</td>
<td style="text-align:left;">
AFFY HG U133 Plus 2 probe ID(s) [e.g. 1553551_s_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95a
</td>
<td style="text-align:left;">
AFFY HG U95A probe ID(s) [e.g. 34279_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95av2
</td>
<td style="text-align:left;">
AFFY HG U95Av2 probe ID(s) [e.g. 34279_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95b
</td>
<td style="text-align:left;">
AFFY HG U95B probe ID(s) [e.g. 49072_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95c
</td>
<td style="text-align:left;">
AFFY HG U95C probe ID(s) [e.g. 64410_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95d
</td>
<td style="text-align:left;">
AFFY HG U95D probe ID(s) [e.g. 76451_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95e
</td>
<td style="text-align:left;">
AFFY HG U95E probe ID(s) [e.g. 88289_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hta_2_0
</td>
<td style="text-align:left;">
AFFY HTA 2 0 probe ID(s) [e.g. TC0M000002.hg]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_huex_1_0_st_v2
</td>
<td style="text-align:left;">
AFFY HuEx 1 0 st v2 probe ID(s) [e.g. 2864632]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hugenefl
</td>
<td style="text-align:left;">
AFFY HuGeneFL probe ID(s) [e.g. Z70759_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hugene_1_0_st_v1
</td>
<td style="text-align:left;">
AFFY HuGene 1 0 st v1 probe ID(s) [e.g. 7946565]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hugene_2_0_st_v1
</td>
<td style="text-align:left;">
AFFY HuGene 2 0 st v1 probe ID(s) [e.g. 17100641]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_primeview
</td>
<td style="text-align:left;">
AFFY PrimeView probe ID(s) [e.g. 11761516_x_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_u133_x3p
</td>
<td style="text-align:left;">
AFFY U133 X3P probe ID(s) [e.g. 1553551_3p_s_at]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_cgh_44b
</td>
<td style="text-align:left;">
AGILENT CGH 44b probe ID(s) [e.g. A_14_P129616]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_gpl26966
</td>
<td style="text-align:left;">
AGILENT GPL26966 probe ID(s) [e.g. HMNXSV003012672]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_gpl6848
</td>
<td style="text-align:left;">
AGILENT GPL6848 probe ID(s) [e.g. A_24_P179339]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_sureprint_g3_ge_8x60k
</td>
<td style="text-align:left;">
AGILENT SurePrint G3 GE 8x60k probe ID(s) [e.g. A_24_P182122]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_sureprint_g3_ge_8x60k_v2
</td>
<td style="text-align:left;">
AGILENT SurePrint G3 GE 8x60k v2 probe ID(s) [e.g. A_24_P182122]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_wholegenome
</td>
<td style="text-align:left;">
AGILENT WholeGenome probe ID(s) [e.g. A_24_P42453]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_wholegenome_4x44k_v1
</td>
<td style="text-align:left;">
AGILENT WholeGenome 4x44k v1 probe ID(s) [e.g. A_24_P179339]
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_wholegenome_4x44k_v2
</td>
<td style="text-align:left;">
AGILENT WholeGenome 4x44k v2 probe ID(s) [e.g. A_24_P182122]
</td>
</tr>
<tr>
<td style="text-align:left;">
codelink_codelink
</td>
<td style="text-align:left;">
CODELINK CODELINK probe ID(s) [e.g. GE84207]
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanht_12_v3
</td>
<td style="text-align:left;">
ILLUMINA HumanHT 12 V3 probe ID(s) [e.g. ILMN_1779625]
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanht_12_v4
</td>
<td style="text-align:left;">
ILLUMINA HumanHT 12 V4 probe ID(s) [e.g. ILMN_3308961]
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanref_8_v3
</td>
<td style="text-align:left;">
ILLUMINA HumanRef 8 V3 probe ID(s) [e.g. ILMN_2295987]
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanwg_6_v1
</td>
<td style="text-align:left;">
ILLUMINA HumanWG 6 V1 probe ID(s) [e.g. 0005910053]
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanwg_6_v2
</td>
<td style="text-align:left;">
ILLUMINA HumanWG 6 V2 probe ID(s) [e.g. ILMN_1779625]
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanwg_6_v3
</td>
<td style="text-align:left;">
ILLUMINA HumanWG 6 V3 probe ID(s) [e.g. ILMN_1779625]
</td>
</tr>
<tr>
<td style="text-align:left;">
phalanx_onearray
</td>
<td style="text-align:left;">
PHALANX OneArray probe ID(s) [e.g. PH_hs_0033947]
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count_greater_than
</td>
<td style="text-align:left;">
Transcript count &gt;=
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count_less_than
</td>
<td style="text-align:left;">
Transcript count &lt;=
</td>
</tr>
<tr>
<td style="text-align:left;">
biotype
</td>
<td style="text-align:left;">
Type
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_biotype
</td>
<td style="text-align:left;">
Transcript Type
</td>
</tr>
<tr>
<td style="text-align:left;">
source
</td>
<td style="text-align:left;">
Source (gene)
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_source
</td>
<td style="text-align:left;">
Source (transcript)
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_tsl
</td>
<td style="text-align:left;">
Transcript Support Level (TSL)
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_gencode_basic
</td>
<td style="text-align:left;">
GENCODE basic annotation
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_appris
</td>
<td style="text-align:left;">
APPRIS annotation
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_is_canonical
</td>
<td style="text-align:left;">
Ensembl Canonical
</td>
</tr>
<tr>
<td style="text-align:left;">
mane_select
</td>
<td style="text-align:left;">
MANE Select transcript
</td>
</tr>
<tr>
<td style="text-align:left;">
mane_plus_clinical
</td>
<td style="text-align:left;">
MANE Plus clinical
</td>
</tr>
<tr>
<td style="text-align:left;">
phenotype_description
</td>
<td style="text-align:left;">
Phenotype description
</td>
</tr>
<tr>
<td style="text-align:left;">
phenotype_source
</td>
<td style="text-align:left;">
Phenotype source
</td>
</tr>
<tr>
<td style="text-align:left;">
go_parent_term
</td>
<td style="text-align:left;">
Parent term accession
</td>
</tr>
<tr>
<td style="text-align:left;">
go_parent_name
</td>
<td style="text-align:left;">
Parent term name
</td>
</tr>
<tr>
<td style="text-align:left;">
go_evidence_code
</td>
<td style="text-align:left;">
GO Evidence code
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hsapiens_paralog
</td>
<td style="text-align:left;">
Paralogous Human Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cabingdonii_homolog
</td>
<td style="text-align:left;">
Orthologous Abingdon island giant tortoise Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_scaustralis_homolog
</td>
<td style="text-align:left;">
Orthologous African ostrich Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mspretus_homolog
</td>
<td style="text-align:left;">
Orthologous Algerian mouse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_vpacos_homolog
</td>
<td style="text-align:left;">
Orthologous Alpaca Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmmarmota_homolog
</td>
<td style="text-align:left;">
Orthologous Alpine marmot Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pformosa_homolog
</td>
<td style="text-align:left;">
Orthologous Amazon molly Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_bbbison_homolog
</td>
<td style="text-align:left;">
Orthologous American bison Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uamericanus_homolog
</td>
<td style="text-align:left;">
Orthologous American black bear Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_nvison_homolog
</td>
<td style="text-align:left;">
Orthologous American mink Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cdromedarius_homolog
</td>
<td style="text-align:left;">
Orthologous Arabian camel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_uparryii_homolog
</td>
<td style="text-align:left;">
Orthologous Arctic ground squirrel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_smerianae_homolog
</td>
<td style="text-align:left;">
Orthologous Argentine black and white tegu Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dnovemcinctus_homolog
</td>
<td style="text-align:left;">
Orthologous Armadillo Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sformosus_homolog
</td>
<td style="text-align:left;">
Orthologous Asian bonytongue Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_gmorhua_homolog
</td>
<td style="text-align:left;">
Orthologous Atlantic cod Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_charengus_homolog
</td>
<td style="text-align:left;">
Orthologous Atlantic herring Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ssalar_homolog
</td>
<td style="text-align:left;">
Orthologous Atlantic salmon Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cporosus_homolog
</td>
<td style="text-align:left;">
Orthologous Australian saltwater crocodile Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_lbergylta_homolog
</td>
<td style="text-align:left;">
Orthologous Ballan wrasse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_lcalcarifer_homolog
</td>
<td style="text-align:left;">
Orthologous Barramundi perch Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dleucas_homolog
</td>
<td style="text-align:left;">
Orthologous Beluga whale Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_spartitus_homolog
</td>
<td style="text-align:left;">
Orthologous Bicolor damselfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rbieti_homolog
</td>
<td style="text-align:left;">
Orthologous Black snub-nosed monkey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_bmusculus_homolog
</td>
<td style="text-align:left;">
Orthologous Blue whale Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_llaticaudata_homolog
</td>
<td style="text-align:left;">
Orthologous Blue-ringed sea krait Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sbboliviensis_homolog
</td>
<td style="text-align:left;">
Orthologous Bolivian squirrel monkey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ppaniscus_homolog
</td>
<td style="text-align:left;">
Orthologous Bonobo Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_strutta_homolog
</td>
<td style="text-align:left;">
Orthologous Brown trout Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hburtoni_homolog
</td>
<td style="text-align:left;">
Orthologous Burton’s mouthbrooder Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ogarnettii_homolog
</td>
<td style="text-align:left;">
Orthologous Bushbaby Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cintestinalis_homolog
</td>
<td style="text-align:left;">
Orthologous C.intestinalis Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_csavignyi_homolog
</td>
<td style="text-align:left;">
Orthologous C.savignyi Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_celegans_homolog
</td>
<td style="text-align:left;">
Orthologous Caenorhabditis elegans Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_fcatus_homolog
</td>
<td style="text-align:left;">
Orthologous Cat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cwagneri_homolog
</td>
<td style="text-align:left;">
Orthologous Chacoan peccary Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cgobio_homolog
</td>
<td style="text-align:left;">
Orthologous Channel bull blenny Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ipunctatus_homolog
</td>
<td style="text-align:left;">
Orthologous Channel catfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ggallus_homolog
</td>
<td style="text-align:left;">
Orthologous Chicken Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ptroglodytes_homolog
</td>
<td style="text-align:left;">
Orthologous Chimpanzee Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cgchok1gshd_homolog
</td>
<td style="text-align:left;">
Orthologous Chinese hamster CHOK1GS Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_osinensis_homolog
</td>
<td style="text-align:left;">
Orthologous Chinese medaka Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_psinensis_homolog
</td>
<td style="text-align:left;">
Orthologous Chinese softshell turtle Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_otshawytscha_homolog
</td>
<td style="text-align:left;">
Orthologous Chinook salmon Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_atestudineus_homolog
</td>
<td style="text-align:left;">
Orthologous Climbing perch Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_aocellaris_homolog
</td>
<td style="text-align:left;">
Orthologous Clown anemonefish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_lchalumnae_homolog
</td>
<td style="text-align:left;">
Orthologous Coelacanth Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_okisutch_homolog
</td>
<td style="text-align:left;">
Orthologous Coho salmon Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_falbicollis_homolog
</td>
<td style="text-align:left;">
Orthologous Collared flycatcher Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_scanaria_homolog
</td>
<td style="text-align:left;">
Orthologous Common canary Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ccarpio_homolog
</td>
<td style="text-align:left;">
Orthologous Common carp Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pmuralis_homolog
</td>
<td style="text-align:left;">
Orthologous Common wall lizard Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_vursinus_homolog
</td>
<td style="text-align:left;">
Orthologous Common wombat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pcoquereli_homolog
</td>
<td style="text-align:left;">
Orthologous Coquerel’s sifaka Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_btaurus_homolog
</td>
<td style="text-align:left;">
Orthologous Cow Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mfascicularis_homolog
</td>
<td style="text-align:left;">
Orthologous Crab-eating macaque Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_odegus_homolog
</td>
<td style="text-align:left;">
Orthologous Degu Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dclupeoides_homolog
</td>
<td style="text-align:left;">
Orthologous Denticle herring Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cldingo_homolog
</td>
<td style="text-align:left;">
Orthologous Dingo Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_clfamiliaris_homolog
</td>
<td style="text-align:left;">
Orthologous Dog Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ttruncatus_homolog
</td>
<td style="text-align:left;">
Orthologous Dolphin Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_bgrunniens_homolog
</td>
<td style="text-align:left;">
Orthologous Domestic yak Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_eaasinus_homolog
</td>
<td style="text-align:left;">
Orthologous Donkey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mleucophaeus_homolog
</td>
<td style="text-align:left;">
Orthologous Drill Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dmelanogaster_homolog
</td>
<td style="text-align:left;">
Orthologous Drosophila melanogaster Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_applatyrhynchos_homolog
</td>
<td style="text-align:left;">
Orthologous Duck Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ptextilis_homolog
</td>
<td style="text-align:left;">
Orthologous Eastern brown snake Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_acalliptera_homolog
</td>
<td style="text-align:left;">
Orthologous Eastern happy Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_eelectricus_homolog
</td>
<td style="text-align:left;">
Orthologous Electric eel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_lafricana_homolog
</td>
<td style="text-align:left;">
Orthologous Elephant Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cmilii_homolog
</td>
<td style="text-align:left;">
Orthologous Elephant shark Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_svulgaris_homolog
</td>
<td style="text-align:left;">
Orthologous Eurasian red squirrel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dlabrax_homolog
</td>
<td style="text-align:left;">
Orthologous European seabass Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mpfuro_homolog
</td>
<td style="text-align:left;">
Orthologous Ferret Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_trubripes_homolog
</td>
<td style="text-align:left;">
Orthologous Fugu Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_amelanoleuca_homolog
</td>
<td style="text-align:left;">
Orthologous Giant panda Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_nleucogenys_homolog
</td>
<td style="text-align:left;">
Orthologous Gibbon Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_saurata_homolog
</td>
<td style="text-align:left;">
Orthologous Gilthead seabream Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_chircus_homolog
</td>
<td style="text-align:left;">
Orthologous Goat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mauratus_homolog
</td>
<td style="text-align:left;">
Orthologous Golden Hamster Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_acchrysaetos_homolog
</td>
<td style="text-align:left;">
Orthologous Golden eagle Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rroxellana_homolog
</td>
<td style="text-align:left;">
Orthologous Golden snub-nosed monkey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sgrahami_homolog
</td>
<td style="text-align:left;">
Orthologous Golden-line barbel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cauratus_homolog
</td>
<td style="text-align:left;">
Orthologous Goldfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_gevgoodei_homolog
</td>
<td style="text-align:left;">
Orthologous Goodes thornscrub tortoise Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ggorilla_homolog
</td>
<td style="text-align:left;">
Orthologous Gorilla Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pmajor_homolog
</td>
<td style="text-align:left;">
Orthologous Great Tit Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sdumerili_homolog
</td>
<td style="text-align:left;">
Orthologous Greater amberjack Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_psimus_homolog
</td>
<td style="text-align:left;">
Orthologous Greater bamboo lemur Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rferrumequinum_homolog
</td>
<td style="text-align:left;">
Orthologous Greater horseshoe bat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_acarolinensis_homolog
</td>
<td style="text-align:left;">
Orthologous Green anole Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cporcellus_homolog
</td>
<td style="text-align:left;">
Orthologous Guinea Pig Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_preticulata_homolog
</td>
<td style="text-align:left;">
Orthologous Guppy Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_eburgeri_homolog
</td>
<td style="text-align:left;">
Orthologous Hagfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_eeuropaeus_homolog
</td>
<td style="text-align:left;">
Orthologous Hedgehog Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ecaballus_homolog
</td>
<td style="text-align:left;">
Orthologous Horse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hhucho_homolog
</td>
<td style="text-align:left;">
Orthologous Huchen Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_bihybrid_homolog
</td>
<td style="text-align:left;">
Orthologous Hybrid - Bos Indicus Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pcapensis_homolog
</td>
<td style="text-align:left;">
Orthologous Hyrax Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_nnaja_homolog
</td>
<td style="text-align:left;">
Orthologous Indian cobra Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_omelastigma_homolog
</td>
<td style="text-align:left;">
Orthologous Indian medaka Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_olatipes_homolog
</td>
<td style="text-align:left;">
Orthologous Japanese medaka HdrR Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cjaponica_homolog
</td>
<td style="text-align:left;">
Orthologous Japanese quail Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ojavanicus_homolog
</td>
<td style="text-align:left;">
Orthologous Javanese ricefish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_shabroptila_homolog
</td>
<td style="text-align:left;">
Orthologous Kakapo Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_dordii_homolog
</td>
<td style="text-align:left;">
Orthologous Kangaroo rat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pcinereus_homolog
</td>
<td style="text-align:left;">
Orthologous Koala Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pmarinus_homolog
</td>
<td style="text-align:left;">
Orthologous Lamprey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_lcrocea_homolog
</td>
<td style="text-align:left;">
Orthologous Large yellow croaker Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_lleishanense_homolog
</td>
<td style="text-align:left;">
Orthologous Leishan spiny toad Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ppardus_homolog
</td>
<td style="text-align:left;">
Orthologous Leopard Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_jjaculus_homolog
</td>
<td style="text-align:left;">
Orthologous Lesser Egyptian jerboa Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_etelfairi_homolog
</td>
<td style="text-align:left;">
Orthologous Lesser hedgehog tenrec Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pleo_homolog
</td>
<td style="text-align:left;">
Orthologous Lion Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_clanigera_homolog
</td>
<td style="text-align:left;">
Orthologous Long-tailed chinchilla Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_clumpus_homolog
</td>
<td style="text-align:left;">
Orthologous Lumpfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_nbrichardi_homolog
</td>
<td style="text-align:left;">
Orthologous Lyretail cichlid Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_anancymaae_homolog
</td>
<td style="text-align:left;">
Orthologous Ma’s night monkey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmulatta_homolog
</td>
<td style="text-align:left;">
Orthologous Macaque Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_nscutatus_homolog
</td>
<td style="text-align:left;">
Orthologous Mainland tiger snake Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pnyererei_homolog
</td>
<td style="text-align:left;">
Orthologous Makobe Island cichlid Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_kmarmoratus_homolog
</td>
<td style="text-align:left;">
Orthologous Mangrove rivulus Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cjacchus_homolog
</td>
<td style="text-align:left;">
Orthologous Marmoset Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_gfortis_homolog
</td>
<td style="text-align:left;">
Orthologous Medium ground-finch Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pvampyrus_homolog
</td>
<td style="text-align:left;">
Orthologous Megabat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_amexicanus_homolog
</td>
<td style="text-align:left;">
Orthologous Mexican tetra Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mlucifugus_homolog
</td>
<td style="text-align:left;">
Orthologous Microbat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_acitrinellus_homolog
</td>
<td style="text-align:left;">
Orthologous Midas cichlid Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmusculus_homolog
</td>
<td style="text-align:left;">
Orthologous Mouse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmurinus_homolog
</td>
<td style="text-align:left;">
Orthologous Mouse Lemur Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_fheteroclitus_homolog
</td>
<td style="text-align:left;">
Orthologous Mummichog Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hgfemale_homolog
</td>
<td style="text-align:left;">
Orthologous Naked mole-rat female Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmonoceros_homolog
</td>
<td style="text-align:left;">
Orthologous Narwhal Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_oniloticus_homolog
</td>
<td style="text-align:left;">
Orthologous Nile tilapia Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pmbairdii_homolog
</td>
<td style="text-align:left;">
Orthologous Northern American deer mouse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_elucius_homolog
</td>
<td style="text-align:left;">
Orthologous Northern pike Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_panubis_homolog
</td>
<td style="text-align:left;">
Orthologous Olive baboon Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mdomestica_homolog
</td>
<td style="text-align:left;">
Orthologous Opossum Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_apercula_homolog
</td>
<td style="text-align:left;">
Orthologous Orange clownfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pabelii_homolog
</td>
<td style="text-align:left;">
Orthologous Orangutan Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cpbellii_homolog
</td>
<td style="text-align:left;">
Orthologous Painted turtle Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pkingsleyae_homolog
</td>
<td style="text-align:left;">
Orthologous Paramormyrops kingsleyae Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sscrofa_homolog
</td>
<td style="text-align:left;">
Orthologous Pig Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mnemestrina_homolog
</td>
<td style="text-align:left;">
Orthologous Pig-tailed macaque Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_oprinceps_homolog
</td>
<td style="text-align:left;">
Orthologous Pika Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_slucioperca_homolog
</td>
<td style="text-align:left;">
Orthologous Pike-perch Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmurdjan_homolog
</td>
<td style="text-align:left;">
Orthologous Pinecone soldierfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_abrachyrhynchus_homolog
</td>
<td style="text-align:left;">
Orthologous Pink-footed goose Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_xmaculatus_homolog
</td>
<td style="text-align:left;">
Orthologous Platyfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_oanatinus_homolog
</td>
<td style="text-align:left;">
Orthologous Platypus Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_umaritimus_homolog
</td>
<td style="text-align:left;">
Orthologous Polar bear Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mochrogaster_homolog
</td>
<td style="text-align:left;">
Orthologous Prairie vole Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ocuniculus_homolog
</td>
<td style="text-align:left;">
Orthologous Rabbit Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_omykiss_homolog
</td>
<td style="text-align:left;">
Orthologous Rainbow trout Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_rnorvegicus_homolog
</td>
<td style="text-align:left;">
Orthologous Rat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_vvulpes_homolog
</td>
<td style="text-align:left;">
Orthologous Red fox Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pnattereri_homolog
</td>
<td style="text-align:left;">
Orthologous Red-bellied piranha Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ecalabaricus_homolog
</td>
<td style="text-align:left;">
Orthologous Reedfish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mcaroli_homolog
</td>
<td style="text-align:left;">
Orthologous Ryukyu mouse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_scerevisiae_homolog
</td>
<td style="text-align:left;">
Orthologous Saccharomyces cerevisiae Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_platipinna_homolog
</td>
<td style="text-align:left;">
Orthologous Sailfin molly Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_oarambouillet_homolog
</td>
<td style="text-align:left;">
Orthologous Sheep Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cvariegatus_homolog
</td>
<td style="text-align:left;">
Orthologous Sheepshead minnow Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_saraneus_homolog
</td>
<td style="text-align:left;">
Orthologous Shrew Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mpahari_homolog
</td>
<td style="text-align:left;">
Orthologous Shrew mouse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_bsplendens_homolog
</td>
<td style="text-align:left;">
Orthologous Siamese fighting fish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mmoschiferus_homolog
</td>
<td style="text-align:left;">
Orthologous Siberian musk deer Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_choffmanni_homolog
</td>
<td style="text-align:left;">
Orthologous Sloth Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_catys_homolog
</td>
<td style="text-align:left;">
Orthologous Sooty mangabey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pcatodon_homolog
</td>
<td style="text-align:left;">
Orthologous Sperm whale Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_apolyacanthus_homolog
</td>
<td style="text-align:left;">
Orthologous Spiny chromis Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_loculatus_homolog
</td>
<td style="text-align:left;">
Orthologous Spotted gar Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_itridecemlineatus_homolog
</td>
<td style="text-align:left;">
Orthologous Squirrel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mspicilegus_homolog
</td>
<td style="text-align:left;">
Orthologous Steppe mouse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_gaculeatus_homolog
</td>
<td style="text-align:left;">
Orthologous Stickleback Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_csyrichta_homolog
</td>
<td style="text-align:left;">
Orthologous Tarsier Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sharrisii_homolog
</td>
<td style="text-align:left;">
Orthologous Tasmanian devil Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_tnigroviridis_homolog
</td>
<td style="text-align:left;">
Orthologous Tetraodon Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_tctriunguis_homolog
</td>
<td style="text-align:left;">
Orthologous Three-toed box turtle Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ptaltaica_homolog
</td>
<td style="text-align:left;">
Orthologous Tiger Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hcomes_homolog
</td>
<td style="text-align:left;">
Orthologous Tiger tail seahorse Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_csemilaevis_homolog
</td>
<td style="text-align:left;">
Orthologous Tongue sole Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_tbelangeri_homolog
</td>
<td style="text-align:left;">
Orthologous Tree Shrew Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_xtropicalis_homolog
</td>
<td style="text-align:left;">
Orthologous Tropical clawed frog Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_spunctatus_homolog
</td>
<td style="text-align:left;">
Orthologous Tuatara Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_smaximus_homolog
</td>
<td style="text-align:left;">
Orthologous Turbot Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mgallopavo_homolog
</td>
<td style="text-align:left;">
Orthologous Turkey Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_nfurzeri_homolog
</td>
<td style="text-align:left;">
Orthologous Turquoise killifish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ngalili_homolog
</td>
<td style="text-align:left;">
Orthologous Upper Galilee mountains blind mole rat Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_psinus_homolog
</td>
<td style="text-align:left;">
Orthologous Vaquita Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_csabaeus_homolog
</td>
<td style="text-align:left;">
Orthologous Vervet-AGM Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_neugenii_homolog
</td>
<td style="text-align:left;">
Orthologous Wallaby Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_bmutus_homolog
</td>
<td style="text-align:left;">
Orthologous Wild yak Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_chyarkandensis_homolog
</td>
<td style="text-align:left;">
Orthologous Yarkand deer Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sldorsalis_homolog
</td>
<td style="text-align:left;">
Orthologous Yellowtail amberjack Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_tguttata_homolog
</td>
<td style="text-align:left;">
Orthologous Zebra finch Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mzebra_homolog
</td>
<td style="text-align:left;">
Orthologous Zebra mbuna Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_drerio_homolog
</td>
<td style="text-align:left;">
Orthologous Zebrafish Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_marmatus_homolog
</td>
<td style="text-align:left;">
Orthologous Zig-zag eel Genes
</td>
</tr>
<tr>
<td style="text-align:left;">
with_interpro
</td>
<td style="text-align:left;">
With Interpro ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_cdd
</td>
<td style="text-align:left;">
With CDD ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_gene3d
</td>
<td style="text-align:left;">
With Gene3D ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hamap
</td>
<td style="text-align:left;">
With HAMAP ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_mobidblite
</td>
<td style="text-align:left;">
With MobiDB lite
</td>
</tr>
<tr>
<td style="text-align:left;">
with_ncoils
</td>
<td style="text-align:left;">
With Coiled-coils (Ncoils)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_hmmpanther
</td>
<td style="text-align:left;">
With PANTHER ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pfam
</td>
<td style="text-align:left;">
With Pfam ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pirsf
</td>
<td style="text-align:left;">
With PIRSF ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_prints
</td>
<td style="text-align:left;">
With Prints ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_scanprosite
</td>
<td style="text-align:left;">
With PROSITE patterns ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_pfscan
</td>
<td style="text-align:left;">
With PROSITE profiles ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_seg
</td>
<td style="text-align:left;">
With Low complexity (Seg)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sfld
</td>
<td style="text-align:left;">
With SFLD ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_sifts_import
</td>
<td style="text-align:left;">
With PDB-ENSP mappings
</td>
</tr>
<tr>
<td style="text-align:left;">
with_signalp
</td>
<td style="text-align:left;">
With Cleavage site (Signalp)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_smart
</td>
<td style="text-align:left;">
With SMART ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_superfamily
</td>
<td style="text-align:left;">
With Superfamily ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_tigrfam
</td>
<td style="text-align:left;">
With TIGRFAM ID(s)
</td>
</tr>
<tr>
<td style="text-align:left;">
with_tmhmm
</td>
<td style="text-align:left;">
With Transmembrane helices
</td>
</tr>
<tr>
<td style="text-align:left;">
interpro
</td>
<td style="text-align:left;">
Interpro ID(s) [e.g. IPR000001]
</td>
</tr>
<tr>
<td style="text-align:left;">
cdd
</td>
<td style="text-align:left;">
CDD ID(s) [e.g. cd00009]
</td>
</tr>
<tr>
<td style="text-align:left;">
gene3d
</td>
<td style="text-align:left;">
Gene3D ID(s) [e.g. 1.10.10.10]
</td>
</tr>
<tr>
<td style="text-align:left;">
hamap
</td>
<td style="text-align:left;">
HAMAP ID(s) [e.g. MF_00001]
</td>
</tr>
<tr>
<td style="text-align:left;">
hmmpanther
</td>
<td style="text-align:left;">
PANTHER ID(s) [e.g. PTHR10003]
</td>
</tr>
<tr>
<td style="text-align:left;">
pfam
</td>
<td style="text-align:left;">
Pfam ID(s) [e.g. PF00001]
</td>
</tr>
<tr>
<td style="text-align:left;">
pirsf
</td>
<td style="text-align:left;">
PIRSF ID(s) [e.g. PIRSF000019]
</td>
</tr>
<tr>
<td style="text-align:left;">
prints
</td>
<td style="text-align:left;">
Prints ID(s) [e.g. PR00001]
</td>
</tr>
<tr>
<td style="text-align:left;">
scanprosite
</td>
<td style="text-align:left;">
PROSITE patterns ID(s) [e.g. PS00010]
</td>
</tr>
<tr>
<td style="text-align:left;">
pfscan
</td>
<td style="text-align:left;">
PROSITE profiles ID(s) [e.g. PS01031]
</td>
</tr>
<tr>
<td style="text-align:left;">
sfld
</td>
<td style="text-align:left;">
SFLD ID(s) [e.g. SFLDF00002]
</td>
</tr>
<tr>
<td style="text-align:left;">
smart
</td>
<td style="text-align:left;">
SMART ID(s) [e.g. SM00002]
</td>
</tr>
<tr>
<td style="text-align:left;">
superfamily
</td>
<td style="text-align:left;">
Superfamily ID(s) [e.g. SSF100879]
</td>
</tr>
<tr>
<td style="text-align:left;">
tigrfam
</td>
<td style="text-align:left;">
TIGRFAM ID(s) [e.g. TIGR00002]
</td>
</tr>
<tr>
<td style="text-align:left;">
germ_line_variation_source
</td>
<td style="text-align:left;">
limit to genes with germline variant data sources
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_variation_source
</td>
<td style="text-align:left;">
limit to genes with somatic variant sources
</td>
</tr>
<tr>
<td style="text-align:left;">
with_validated_snp
</td>
<td style="text-align:left;">
Variant supporting evidence
</td>
</tr>
<tr>
<td style="text-align:left;">
so_mini_parent_name
</td>
<td style="text-align:left;">
Parent term name
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Note that <code>listFilters</code> must be used with a mart object in which a dataset has been defined.<br />
Since we are mapping from Entrez ID, we choose filter <code>entrezgene_id</code>.<br />
</p>
<p>Attributes define the types of data we want in the query result. To see attributes available for the chosen dataset:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/listAttributes.html'>listAttributes</a></span><span class='op'>(</span>mart <span class='op'>=</span> <span class='va'>ensembl</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:300px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
page
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID version
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID version
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID version
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_exon_id
</td>
<td style="text-align:left;">
Exon stable ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
description
</td>
<td style="text-align:left;">
Gene description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
start_position
</td>
<td style="text-align:left;">
Gene start (bp)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
end_position
</td>
<td style="text-align:left;">
Gene end (bp)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
band
</td>
<td style="text-align:left;">
Karyotype band
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_start
</td>
<td style="text-align:left;">
Transcript start (bp)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_end
</td>
<td style="text-align:left;">
Transcript end (bp)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcription_start_site
</td>
<td style="text-align:left;">
Transcription start site (TSS)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_length
</td>
<td style="text-align:left;">
Transcript length (including UTRs and CDS)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_tsl
</td>
<td style="text-align:left;">
Transcript support level (TSL)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_gencode_basic
</td>
<td style="text-align:left;">
GENCODE basic annotation
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_appris
</td>
<td style="text-align:left;">
APPRIS annotation
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_is_canonical
</td>
<td style="text-align:left;">
Ensembl Canonical
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_mane_select
</td>
<td style="text-align:left;">
RefSeq match transcript (MANE Select)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_mane_plus_clinical
</td>
<td style="text-align:left;">
RefSeq match transcript (MANE Plus Clinical)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_source
</td>
<td style="text-align:left;">
Source of gene name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
external_transcript_name
</td>
<td style="text-align:left;">
Transcript name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
external_transcript_source_name
</td>
<td style="text-align:left;">
Source of transcript name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count
</td>
<td style="text-align:left;">
Transcript count
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
percentage_gene_gc_content
</td>
<td style="text-align:left;">
Gene % GC content
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
gene_biotype
</td>
<td style="text-align:left;">
Gene type
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_biotype
</td>
<td style="text-align:left;">
Transcript type
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
source
</td>
<td style="text-align:left;">
Source (gene)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_source
</td>
<td style="text-align:left;">
Source (transcript)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
version
</td>
<td style="text-align:left;">
Version (gene)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_version
</td>
<td style="text-align:left;">
Version (transcript)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_version
</td>
<td style="text-align:left;">
Version (protein)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
external_synonym
</td>
<td style="text-align:left;">
Gene Synonym
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
phenotype_description
</td>
<td style="text-align:left;">
Phenotype description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
source_name
</td>
<td style="text-align:left;">
Source name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
study_external_id
</td>
<td style="text-align:left;">
Study external reference
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
strain_name
</td>
<td style="text-align:left;">
Strain name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
strain_gender
</td>
<td style="text-align:left;">
Strain gender
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
p_value
</td>
<td style="text-align:left;">
P value
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
go_id
</td>
<td style="text-align:left;">
GO term accession
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
name_1006
</td>
<td style="text-align:left;">
GO term name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
definition_1006
</td>
<td style="text-align:left;">
GO term definition
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
go_linkage_type
</td>
<td style="text-align:left;">
GO term evidence code
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
namespace_1003
</td>
<td style="text-align:left;">
GO domain
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
goslim_goa_accession
</td>
<td style="text-align:left;">
GOSlim GOA Accession(s)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
goslim_goa_description
</td>
<td style="text-align:left;">
GOSlim GOA Description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
biogrid
</td>
<td style="text-align:left;">
BioGRID Interaction data, The General Repository for Interaction Datasets ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ccds
</td>
<td style="text-align:left;">
CCDS ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
chembl
</td>
<td style="text-align:left;">
ChEMBL ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass3_name
</td>
<td style="text-align:left;">
DataBase of Aberrant 3’ Splice Sites name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass3_id
</td>
<td style="text-align:left;">
DataBase of Aberrant 3’ Splice Sites ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass5_name
</td>
<td style="text-align:left;">
DataBase of Aberrant 5’ Splice Sites name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
dbass5_id
</td>
<td style="text-align:left;">
DataBase of Aberrant 5’ Splice Sites ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_hs_transcript
</td>
<td style="text-align:left;">
Ensembl Human Transcript ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_hs_translation
</td>
<td style="text-align:left;">
Ensembl Human Translation ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
entrezgene_trans_name
</td>
<td style="text-align:left;">
EntrezGene transcript name ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
embl
</td>
<td style="text-align:left;">
European Nucleotide Archive ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
arrayexpress
</td>
<td style="text-align:left;">
Expression Atlas ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
genedb
</td>
<td style="text-align:left;">
GeneDB ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hgnc_id
</td>
<td style="text-align:left;">
HGNC ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
hgnc_symbol
</td>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
HGNC symbol
</td>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hpa_accession
</td>
<td style="text-align:left;">
Human Protein Atlas accession
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hpa_id
</td>
<td style="text-align:left;">
Human Protein Atlas ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
protein_id
</td>
<td style="text-align:left;">
INSDC protein ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
kegg_enzyme
</td>
<td style="text-align:left;">
KEGG Pathway and Enzyme ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_lrg_gene
</td>
<td style="text-align:left;">
LRG display in Ensembl gene ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ens_lrg_transcript
</td>
<td style="text-align:left;">
LRG display in Ensembl transcript ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
merops
</td>
<td style="text-align:left;">
MEROPS - the Peptidase Database ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mim_gene_description
</td>
<td style="text-align:left;">
MIM gene description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mim_gene_accession
</td>
<td style="text-align:left;">
MIM gene accession
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mim_morbid_description
</td>
<td style="text-align:left;">
MIM morbid description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mim_morbid_accession
</td>
<td style="text-align:left;">
MIM morbid accession
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mirbase_accession
</td>
<td style="text-align:left;">
miRBase accession
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mirbase_id
</td>
<td style="text-align:left;">
miRBase ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mirbase_trans_name
</td>
<td style="text-align:left;">
miRBase transcript name ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
entrezgene_description
</td>
<td style="text-align:left;">
NCBI gene (formerly Entrezgene) description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
entrezgene_accession
</td>
<td style="text-align:left;">
NCBI gene (formerly Entrezgene) accession
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
entrezgene_id
</td>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
NCBI gene (formerly Entrezgene) ID
</td>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #D7261E !important;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pdb
</td>
<td style="text-align:left;">
PDB ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
reactome
</td>
<td style="text-align:left;">
Reactome ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
reactome_gene
</td>
<td style="text-align:left;">
Reactome gene ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
reactome_transcript
</td>
<td style="text-align:left;">
Reactome transcript ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_mrna
</td>
<td style="text-align:left;">
RefSeq mRNA ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_mrna_predicted
</td>
<td style="text-align:left;">
RefSeq mRNA predicted ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_ncrna
</td>
<td style="text-align:left;">
RefSeq ncRNA ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_ncrna_predicted
</td>
<td style="text-align:left;">
RefSeq ncRNA predicted ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_peptide
</td>
<td style="text-align:left;">
RefSeq peptide ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
refseq_peptide_predicted
</td>
<td style="text-align:left;">
RefSeq peptide predicted ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
rfam
</td>
<td style="text-align:left;">
RFAM ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
rfam_trans_name
</td>
<td style="text-align:left;">
RFAM transcript name ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
rnacentral
</td>
<td style="text-align:left;">
RNAcentral ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hgnc_trans_name
</td>
<td style="text-align:left;">
Transcript name ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ucsc
</td>
<td style="text-align:left;">
UCSC Stable ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
uniparc
</td>
<td style="text-align:left;">
UniParc ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprot_gn_symbol
</td>
<td style="text-align:left;">
UniProtKB Gene Name symbol
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprot_gn_id
</td>
<td style="text-align:left;">
UniProtKB Gene Name ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprot_isoform
</td>
<td style="text-align:left;">
UniProtKB isoform ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprotswissprot
</td>
<td style="text-align:left;">
UniProtKB/Swiss-Prot ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprotsptrembl
</td>
<td style="text-align:left;">
UniProtKB/TrEMBL ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
wikigene_description
</td>
<td style="text-align:left;">
WikiGene description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
wikigene_name
</td>
<td style="text-align:left;">
WikiGene name
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
wikigene_id
</td>
<td style="text-align:left;">
WikiGene ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hc_g110
</td>
<td style="text-align:left;">
AFFY HC G110 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_focus
</td>
<td style="text-align:left;">
AFFY HG Focus probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133a
</td>
<td style="text-align:left;">
AFFY HG U133A probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133a_2
</td>
<td style="text-align:left;">
AFFY HG U133A 2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133b
</td>
<td style="text-align:left;">
AFFY HG U133B probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u133_plus_2
</td>
<td style="text-align:left;">
AFFY HG U133 Plus 2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95a
</td>
<td style="text-align:left;">
AFFY HG U95A probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95av2
</td>
<td style="text-align:left;">
AFFY HG U95Av2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95b
</td>
<td style="text-align:left;">
AFFY HG U95B probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95c
</td>
<td style="text-align:left;">
AFFY HG U95C probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95d
</td>
<td style="text-align:left;">
AFFY HG U95D probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hg_u95e
</td>
<td style="text-align:left;">
AFFY HG U95E probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hta_2_0
</td>
<td style="text-align:left;">
AFFY HTA 2 0 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_huex_1_0_st_v2
</td>
<td style="text-align:left;">
AFFY HuEx 1 0 st v2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hugenefl
</td>
<td style="text-align:left;">
AFFY HuGeneFL probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hugene_1_0_st_v1
</td>
<td style="text-align:left;">
AFFY HuGene 1 0 st v1 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_hugene_2_0_st_v1
</td>
<td style="text-align:left;">
AFFY HuGene 2 0 st v1 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_primeview
</td>
<td style="text-align:left;">
AFFY PrimeView probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
affy_u133_x3p
</td>
<td style="text-align:left;">
AFFY U133 X3P probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_cgh_44b
</td>
<td style="text-align:left;">
AGILENT CGH 44b probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_gpl26966
</td>
<td style="text-align:left;">
AGILENT GPL26966 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_gpl6848
</td>
<td style="text-align:left;">
AGILENT GPL6848 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_sureprint_g3_ge_8x60k
</td>
<td style="text-align:left;">
AGILENT SurePrint G3 GE 8x60k probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_sureprint_g3_ge_8x60k_v2
</td>
<td style="text-align:left;">
AGILENT SurePrint G3 GE 8x60k v2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_wholegenome
</td>
<td style="text-align:left;">
AGILENT WholeGenome probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_wholegenome_4x44k_v1
</td>
<td style="text-align:left;">
AGILENT WholeGenome 4x44k v1 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
agilent_wholegenome_4x44k_v2
</td>
<td style="text-align:left;">
AGILENT WholeGenome 4x44k v2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
codelink_codelink
</td>
<td style="text-align:left;">
CODELINK CODELINK probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanht_12_v3
</td>
<td style="text-align:left;">
ILLUMINA HumanHT 12 V3 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanht_12_v4
</td>
<td style="text-align:left;">
ILLUMINA HumanHT 12 V4 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanref_8_v3
</td>
<td style="text-align:left;">
ILLUMINA HumanRef 8 V3 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanwg_6_v1
</td>
<td style="text-align:left;">
ILLUMINA HumanWG 6 V1 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanwg_6_v2
</td>
<td style="text-align:left;">
ILLUMINA HumanWG 6 V2 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
illumina_humanwg_6_v3
</td>
<td style="text-align:left;">
ILLUMINA HumanWG 6 V3 probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
phalanx_onearray
</td>
<td style="text-align:left;">
PHALANX OneArray probe
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
cdd
</td>
<td style="text-align:left;">
CDD ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
cdd_start
</td>
<td style="text-align:left;">
CDD start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
cdd_end
</td>
<td style="text-align:left;">
CDD end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
gene3d
</td>
<td style="text-align:left;">
Gene3D ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
gene3d_start
</td>
<td style="text-align:left;">
Gene3D start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
gene3d_end
</td>
<td style="text-align:left;">
Gene3D end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hamap
</td>
<td style="text-align:left;">
HAMAP ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hamap_start
</td>
<td style="text-align:left;">
HAMAP start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hamap_end
</td>
<td style="text-align:left;">
HAMAP end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hmmpanther
</td>
<td style="text-align:left;">
PANTHER ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hmmpanther_start
</td>
<td style="text-align:left;">
PANTHER start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
hmmpanther_end
</td>
<td style="text-align:left;">
PANTHER end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pfam
</td>
<td style="text-align:left;">
Pfam ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pfam_start
</td>
<td style="text-align:left;">
Pfam start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pfam_end
</td>
<td style="text-align:left;">
Pfam end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pirsf
</td>
<td style="text-align:left;">
PIRSF ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pirsf_start
</td>
<td style="text-align:left;">
PIRSF start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pirsf_end
</td>
<td style="text-align:left;">
PIRSF end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
prints
</td>
<td style="text-align:left;">
Prints ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
prints_start
</td>
<td style="text-align:left;">
Prints start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
prints_end
</td>
<td style="text-align:left;">
Prints end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
scanprosite
</td>
<td style="text-align:left;">
PROSITE patterns ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
scanprosite_start
</td>
<td style="text-align:left;">
PROSITE patterns start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
scanprosite_end
</td>
<td style="text-align:left;">
PROSITE patterns end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pfscan
</td>
<td style="text-align:left;">
PROSITE profiles ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pfscan_start
</td>
<td style="text-align:left;">
PROSITE profiles start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
pfscan_end
</td>
<td style="text-align:left;">
PROSITE profiles end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
sfld
</td>
<td style="text-align:left;">
SFLD ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
sfld_start
</td>
<td style="text-align:left;">
SFLD start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
sfld_end
</td>
<td style="text-align:left;">
SFLD end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
smart
</td>
<td style="text-align:left;">
SMART ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
smart_start
</td>
<td style="text-align:left;">
SMART start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
smart_end
</td>
<td style="text-align:left;">
SMART end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
superfamily
</td>
<td style="text-align:left;">
Superfamily ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
superfamily_start
</td>
<td style="text-align:left;">
Superfamily start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
superfamily_end
</td>
<td style="text-align:left;">
Superfamily end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
tigrfam
</td>
<td style="text-align:left;">
TIGRFAM ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
tigrfam_start
</td>
<td style="text-align:left;">
TIGRFAM start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
tigrfam_end
</td>
<td style="text-align:left;">
TIGRFAM end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
interpro
</td>
<td style="text-align:left;">
Interpro ID
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
interpro_short_description
</td>
<td style="text-align:left;">
Interpro Short Description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
interpro_description
</td>
<td style="text-align:left;">
Interpro Description
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
interpro_start
</td>
<td style="text-align:left;">
Interpro start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
interpro_end
</td>
<td style="text-align:left;">
Interpro end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mobidblite
</td>
<td style="text-align:left;">
MobiDB lite
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mobidblite_start
</td>
<td style="text-align:left;">
MobiDB lite start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
mobidblite_end
</td>
<td style="text-align:left;">
MobiDB lite end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ncoils
</td>
<td style="text-align:left;">
Coiled-coils (Ncoils)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ncoils_start
</td>
<td style="text-align:left;">
Coiled-coils (Ncoils) start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ncoils_end
</td>
<td style="text-align:left;">
Coiled-coils (Ncoils) end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
seg
</td>
<td style="text-align:left;">
Low complexity (Seg)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
seg_start
</td>
<td style="text-align:left;">
Low complexity (Seg) start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
seg_end
</td>
<td style="text-align:left;">
Low complexity (Seg) end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
sifts_import
</td>
<td style="text-align:left;">
PDB-ENSP mappings
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
sifts_import_start
</td>
<td style="text-align:left;">
PDB-ENSP mappings start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
sifts_import_end
</td>
<td style="text-align:left;">
PDB-ENSP mappings end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
signalp
</td>
<td style="text-align:left;">
Cleavage site (Signalp)
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
signalp_start
</td>
<td style="text-align:left;">
Cleavage site (Signalp) start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
signalp_end
</td>
<td style="text-align:left;">
Cleavage site (Signalp) end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
tmhmm
</td>
<td style="text-align:left;">
Transmembrane helices
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
tmhmm_start
</td>
<td style="text-align:left;">
Transmembrane helices start
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
tmhmm_end
</td>
<td style="text-align:left;">
Transmembrane helices end
</td>
<td style="text-align:left;">
feature_page
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID version
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
version
</td>
<td style="text-align:left;">
Version (gene)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID version
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_version
</td>
<td style="text-align:left;">
Version (transcript)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID version
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_version
</td>
<td style="text-align:left;">
Version (protein)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
start_position
</td>
<td style="text-align:left;">
Gene start (bp)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
end_position
</td>
<td style="text-align:left;">
Gene end (bp)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_start
</td>
<td style="text-align:left;">
Transcript start (bp)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_end
</td>
<td style="text-align:left;">
Transcript end (bp)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
transcription_start_site
</td>
<td style="text-align:left;">
Transcription start site (TSS)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_length
</td>
<td style="text-align:left;">
Transcript length (including UTRs and CDS)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene name
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_source
</td>
<td style="text-align:left;">
Source of gene name
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
5_utr_start
</td>
<td style="text-align:left;">
5’ UTR start
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
5_utr_end
</td>
<td style="text-align:left;">
5’ UTR end
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
3_utr_start
</td>
<td style="text-align:left;">
3’ UTR start
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
3_utr_end
</td>
<td style="text-align:left;">
3’ UTR end
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_length
</td>
<td style="text-align:left;">
CDS Length
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count
</td>
<td style="text-align:left;">
Transcript count
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
description
</td>
<td style="text-align:left;">
Gene description
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
gene_biotype
</td>
<td style="text-align:left;">
Gene type
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
exon_chrom_start
</td>
<td style="text-align:left;">
Exon region start (bp)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
exon_chrom_end
</td>
<td style="text-align:left;">
Exon region end (bp)
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
is_constitutive
</td>
<td style="text-align:left;">
Constitutive exon
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
rank
</td>
<td style="text-align:left;">
Exon rank in transcript
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
phase
</td>
<td style="text-align:left;">
Start phase
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
end_phase
</td>
<td style="text-align:left;">
End phase
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna_coding_start
</td>
<td style="text-align:left;">
cDNA coding start
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna_coding_end
</td>
<td style="text-align:left;">
cDNA coding end
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
genomic_coding_start
</td>
<td style="text-align:left;">
Genomic coding start
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
genomic_coding_end
</td>
<td style="text-align:left;">
Genomic coding end
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_exon_id
</td>
<td style="text-align:left;">
Exon stable ID
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_start
</td>
<td style="text-align:left;">
CDS start
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_end
</td>
<td style="text-align:left;">
CDS end
</td>
<td style="text-align:left;">
structure
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID version
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
version
</td>
<td style="text-align:left;">
Version (gene)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID version
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_version
</td>
<td style="text-align:left;">
Version (transcript)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID version
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_version
</td>
<td style="text-align:left;">
Version (protein)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
start_position
</td>
<td style="text-align:left;">
Gene start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
end_position
</td>
<td style="text-align:left;">
Gene end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
band
</td>
<td style="text-align:left;">
Karyotype band
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_source
</td>
<td style="text-align:left;">
Source of gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count
</td>
<td style="text-align:left;">
Transcript count
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
percentage_gene_gc_content
</td>
<td style="text-align:left;">
Gene % GC content
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
description
</td>
<td style="text-align:left;">
Gene description
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Abingdon island giant tortoise gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Abingdon island giant tortoise gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Abingdon island giant tortoise protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_chromosome
</td>
<td style="text-align:left;">
Abingdon island giant tortoise chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_chrom_start
</td>
<td style="text-align:left;">
Abingdon island giant tortoise chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_chrom_end
</td>
<td style="text-align:left;">
Abingdon island giant tortoise chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Abingdon island giant tortoise
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_orthology_type
</td>
<td style="text-align:left;">
Abingdon island giant tortoise homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Abingdon island giant tortoise gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Abingdon island giant tortoise gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_goc_score
</td>
<td style="text-align:left;">
Abingdon island giant tortoise Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Abingdon island giant tortoise Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cabingdonii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Abingdon island giant tortoise orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
African ostrich gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
African ostrich gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
African ostrich protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_chromosome
</td>
<td style="text-align:left;">
African ostrich chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_chrom_start
</td>
<td style="text-align:left;">
African ostrich chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_chrom_end
</td>
<td style="text-align:left;">
African ostrich chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with African ostrich
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_orthology_type
</td>
<td style="text-align:left;">
African ostrich homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target African ostrich gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target African ostrich gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_goc_score
</td>
<td style="text-align:left;">
African ostrich Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_wga_coverage
</td>
<td style="text-align:left;">
African ostrich Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scaustralis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
African ostrich orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Algerian mouse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Algerian mouse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Algerian mouse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_chromosome
</td>
<td style="text-align:left;">
Algerian mouse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_chrom_start
</td>
<td style="text-align:left;">
Algerian mouse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_chrom_end
</td>
<td style="text-align:left;">
Algerian mouse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Algerian mouse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_orthology_type
</td>
<td style="text-align:left;">
Algerian mouse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Algerian mouse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Algerian mouse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_goc_score
</td>
<td style="text-align:left;">
Algerian mouse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Algerian mouse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspretus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Algerian mouse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Alpaca gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Alpaca gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Alpaca protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_chromosome
</td>
<td style="text-align:left;">
Alpaca chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_chrom_start
</td>
<td style="text-align:left;">
Alpaca chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_chrom_end
</td>
<td style="text-align:left;">
Alpaca chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Alpaca
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_orthology_type
</td>
<td style="text-align:left;">
Alpaca homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Alpaca gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Alpaca gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_goc_score
</td>
<td style="text-align:left;">
Alpaca Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_wga_coverage
</td>
<td style="text-align:left;">
Alpaca Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vpacos_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Alpaca orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Alpine marmot gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Alpine marmot gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Alpine marmot protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_chromosome
</td>
<td style="text-align:left;">
Alpine marmot chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_chrom_start
</td>
<td style="text-align:left;">
Alpine marmot chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_chrom_end
</td>
<td style="text-align:left;">
Alpine marmot chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Alpine marmot
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_orthology_type
</td>
<td style="text-align:left;">
Alpine marmot homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Alpine marmot gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Alpine marmot gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_goc_score
</td>
<td style="text-align:left;">
Alpine marmot Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_wga_coverage
</td>
<td style="text-align:left;">
Alpine marmot Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmmarmota_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Alpine marmot orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Amazon molly gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Amazon molly gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Amazon molly protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_chromosome
</td>
<td style="text-align:left;">
Amazon molly chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_chrom_start
</td>
<td style="text-align:left;">
Amazon molly chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_chrom_end
</td>
<td style="text-align:left;">
Amazon molly chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Amazon molly
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_orthology_type
</td>
<td style="text-align:left;">
Amazon molly homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Amazon molly gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Amazon molly gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_goc_score
</td>
<td style="text-align:left;">
Amazon molly Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_wga_coverage
</td>
<td style="text-align:left;">
Amazon molly Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pformosa_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Amazon molly orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_ensembl_gene
</td>
<td style="text-align:left;">
American bison gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_associated_gene_name
</td>
<td style="text-align:left;">
American bison gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
American bison protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_chromosome
</td>
<td style="text-align:left;">
American bison chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_chrom_start
</td>
<td style="text-align:left;">
American bison chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_chrom_end
</td>
<td style="text-align:left;">
American bison chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with American bison
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_orthology_type
</td>
<td style="text-align:left;">
American bison homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target American bison gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target American bison gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_goc_score
</td>
<td style="text-align:left;">
American bison Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_wga_coverage
</td>
<td style="text-align:left;">
American bison Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bbbison_homolog_orthology_confidence
</td>
<td style="text-align:left;">
American bison orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
American black bear gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
American black bear gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
American black bear protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_chromosome
</td>
<td style="text-align:left;">
American black bear chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_chrom_start
</td>
<td style="text-align:left;">
American black bear chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_chrom_end
</td>
<td style="text-align:left;">
American black bear chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with American black bear
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_orthology_type
</td>
<td style="text-align:left;">
American black bear homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target American black bear gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target American black bear gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_goc_score
</td>
<td style="text-align:left;">
American black bear Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_wga_coverage
</td>
<td style="text-align:left;">
American black bear Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uamericanus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
American black bear orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_ensembl_gene
</td>
<td style="text-align:left;">
American mink gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_associated_gene_name
</td>
<td style="text-align:left;">
American mink gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
American mink protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_chromosome
</td>
<td style="text-align:left;">
American mink chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_chrom_start
</td>
<td style="text-align:left;">
American mink chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_chrom_end
</td>
<td style="text-align:left;">
American mink chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with American mink
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_orthology_type
</td>
<td style="text-align:left;">
American mink homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target American mink gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target American mink gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_goc_score
</td>
<td style="text-align:left;">
American mink Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_wga_coverage
</td>
<td style="text-align:left;">
American mink Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nvison_homolog_orthology_confidence
</td>
<td style="text-align:left;">
American mink orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Arabian camel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Arabian camel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Arabian camel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_chromosome
</td>
<td style="text-align:left;">
Arabian camel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_chrom_start
</td>
<td style="text-align:left;">
Arabian camel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_chrom_end
</td>
<td style="text-align:left;">
Arabian camel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Arabian camel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_orthology_type
</td>
<td style="text-align:left;">
Arabian camel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Arabian camel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Arabian camel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_goc_score
</td>
<td style="text-align:left;">
Arabian camel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_wga_coverage
</td>
<td style="text-align:left;">
Arabian camel Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cdromedarius_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Arabian camel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Arctic ground squirrel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Arctic ground squirrel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Arctic ground squirrel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_chromosome
</td>
<td style="text-align:left;">
Arctic ground squirrel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_chrom_start
</td>
<td style="text-align:left;">
Arctic ground squirrel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_chrom_end
</td>
<td style="text-align:left;">
Arctic ground squirrel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Arctic ground squirrel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_orthology_type
</td>
<td style="text-align:left;">
Arctic ground squirrel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Arctic ground squirrel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Arctic ground squirrel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_goc_score
</td>
<td style="text-align:left;">
Arctic ground squirrel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Arctic ground squirrel Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
uparryii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Arctic ground squirrel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Argentine black and white tegu gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Argentine black and white tegu gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Argentine black and white tegu protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_chromosome
</td>
<td style="text-align:left;">
Argentine black and white tegu chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_chrom_start
</td>
<td style="text-align:left;">
Argentine black and white tegu chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_chrom_end
</td>
<td style="text-align:left;">
Argentine black and white tegu chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Argentine black and white tegu
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_orthology_type
</td>
<td style="text-align:left;">
Argentine black and white tegu homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Argentine black and white tegu gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Argentine black and white tegu gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_goc_score
</td>
<td style="text-align:left;">
Argentine black and white tegu Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_wga_coverage
</td>
<td style="text-align:left;">
Argentine black and white tegu Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smerianae_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Argentine black and white tegu orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Armadillo gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Armadillo gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Armadillo protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_chromosome
</td>
<td style="text-align:left;">
Armadillo chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_chrom_start
</td>
<td style="text-align:left;">
Armadillo chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_chrom_end
</td>
<td style="text-align:left;">
Armadillo chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Armadillo
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_orthology_type
</td>
<td style="text-align:left;">
Armadillo homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Armadillo gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Armadillo gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_goc_score
</td>
<td style="text-align:left;">
Armadillo Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Armadillo Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dnovemcinctus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Armadillo orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Asian bonytongue gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Asian bonytongue gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Asian bonytongue protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_chromosome
</td>
<td style="text-align:left;">
Asian bonytongue chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_chrom_start
</td>
<td style="text-align:left;">
Asian bonytongue chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_chrom_end
</td>
<td style="text-align:left;">
Asian bonytongue chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Asian bonytongue
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_orthology_type
</td>
<td style="text-align:left;">
Asian bonytongue homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Asian bonytongue gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Asian bonytongue gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_goc_score
</td>
<td style="text-align:left;">
Asian bonytongue Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Asian bonytongue Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sformosus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Asian bonytongue orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Atlantic cod gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Atlantic cod gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Atlantic cod protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_chromosome
</td>
<td style="text-align:left;">
Atlantic cod chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_chrom_start
</td>
<td style="text-align:left;">
Atlantic cod chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_chrom_end
</td>
<td style="text-align:left;">
Atlantic cod chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Atlantic cod
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_orthology_type
</td>
<td style="text-align:left;">
Atlantic cod homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Atlantic cod gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Atlantic cod gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_goc_score
</td>
<td style="text-align:left;">
Atlantic cod Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gmorhua_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Atlantic cod orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Atlantic herring gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Atlantic herring gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Atlantic herring protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_chromosome
</td>
<td style="text-align:left;">
Atlantic herring chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_chrom_start
</td>
<td style="text-align:left;">
Atlantic herring chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_chrom_end
</td>
<td style="text-align:left;">
Atlantic herring chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Atlantic herring
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_orthology_type
</td>
<td style="text-align:left;">
Atlantic herring homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Atlantic herring gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Atlantic herring gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_goc_score
</td>
<td style="text-align:left;">
Atlantic herring Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Atlantic herring Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
charengus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Atlantic herring orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Atlantic salmon gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Atlantic salmon gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Atlantic salmon protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_chromosome
</td>
<td style="text-align:left;">
Atlantic salmon chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_chrom_start
</td>
<td style="text-align:left;">
Atlantic salmon chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_chrom_end
</td>
<td style="text-align:left;">
Atlantic salmon chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Atlantic salmon
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_orthology_type
</td>
<td style="text-align:left;">
Atlantic salmon homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Atlantic salmon gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Atlantic salmon gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_goc_score
</td>
<td style="text-align:left;">
Atlantic salmon Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_wga_coverage
</td>
<td style="text-align:left;">
Atlantic salmon Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ssalar_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Atlantic salmon orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Australian saltwater crocodile gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Australian saltwater crocodile gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Australian saltwater crocodile protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_chromosome
</td>
<td style="text-align:left;">
Australian saltwater crocodile chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_chrom_start
</td>
<td style="text-align:left;">
Australian saltwater crocodile chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_chrom_end
</td>
<td style="text-align:left;">
Australian saltwater crocodile chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Australian saltwater crocodile
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_orthology_type
</td>
<td style="text-align:left;">
Australian saltwater crocodile homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Australian saltwater crocodile gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Australian saltwater crocodile gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_goc_score
</td>
<td style="text-align:left;">
Australian saltwater crocodile Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Australian saltwater crocodile Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporosus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Australian saltwater crocodile orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Ballan wrasse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Ballan wrasse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Ballan wrasse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_chromosome
</td>
<td style="text-align:left;">
Ballan wrasse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_chrom_start
</td>
<td style="text-align:left;">
Ballan wrasse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_chrom_end
</td>
<td style="text-align:left;">
Ballan wrasse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Ballan wrasse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_orthology_type
</td>
<td style="text-align:left;">
Ballan wrasse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Ballan wrasse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Ballan wrasse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_goc_score
</td>
<td style="text-align:left;">
Ballan wrasse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_wga_coverage
</td>
<td style="text-align:left;">
Ballan wrasse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lbergylta_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Ballan wrasse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Barramundi perch gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Barramundi perch gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Barramundi perch protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_chromosome
</td>
<td style="text-align:left;">
Barramundi perch chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_chrom_start
</td>
<td style="text-align:left;">
Barramundi perch chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_chrom_end
</td>
<td style="text-align:left;">
Barramundi perch chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Barramundi perch
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_orthology_type
</td>
<td style="text-align:left;">
Barramundi perch homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Barramundi perch gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Barramundi perch gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_goc_score
</td>
<td style="text-align:left;">
Barramundi perch Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_wga_coverage
</td>
<td style="text-align:left;">
Barramundi perch Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcalcarifer_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Barramundi perch orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Beluga whale gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Beluga whale gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Beluga whale protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_chromosome
</td>
<td style="text-align:left;">
Beluga whale chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_chrom_start
</td>
<td style="text-align:left;">
Beluga whale chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_chrom_end
</td>
<td style="text-align:left;">
Beluga whale chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Beluga whale
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_orthology_type
</td>
<td style="text-align:left;">
Beluga whale homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Beluga whale gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Beluga whale gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_goc_score
</td>
<td style="text-align:left;">
Beluga whale Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_wga_coverage
</td>
<td style="text-align:left;">
Beluga whale Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dleucas_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Beluga whale orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Bicolor damselfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Bicolor damselfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Bicolor damselfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_chromosome
</td>
<td style="text-align:left;">
Bicolor damselfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_chrom_start
</td>
<td style="text-align:left;">
Bicolor damselfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_chrom_end
</td>
<td style="text-align:left;">
Bicolor damselfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Bicolor damselfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_orthology_type
</td>
<td style="text-align:left;">
Bicolor damselfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Bicolor damselfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Bicolor damselfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_goc_score
</td>
<td style="text-align:left;">
Bicolor damselfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Bicolor damselfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spartitus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Bicolor damselfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Black snub-nosed monkey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Black snub-nosed monkey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Black snub-nosed monkey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_chromosome
</td>
<td style="text-align:left;">
Black snub-nosed monkey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_chrom_start
</td>
<td style="text-align:left;">
Black snub-nosed monkey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_chrom_end
</td>
<td style="text-align:left;">
Black snub-nosed monkey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Black snub-nosed monkey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_orthology_type
</td>
<td style="text-align:left;">
Black snub-nosed monkey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Black snub-nosed monkey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Black snub-nosed monkey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_goc_score
</td>
<td style="text-align:left;">
Black snub-nosed monkey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_wga_coverage
</td>
<td style="text-align:left;">
Black snub-nosed monkey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rbieti_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Black snub-nosed monkey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Blue whale gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Blue whale gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Blue whale protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_chromosome
</td>
<td style="text-align:left;">
Blue whale chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_chrom_start
</td>
<td style="text-align:left;">
Blue whale chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_chrom_end
</td>
<td style="text-align:left;">
Blue whale chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Blue whale
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_orthology_type
</td>
<td style="text-align:left;">
Blue whale homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Blue whale gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Blue whale gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_goc_score
</td>
<td style="text-align:left;">
Blue whale Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Blue whale Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmusculus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Blue whale orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Blue-ringed sea krait gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Blue-ringed sea krait gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Blue-ringed sea krait protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_chromosome
</td>
<td style="text-align:left;">
Blue-ringed sea krait chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_chrom_start
</td>
<td style="text-align:left;">
Blue-ringed sea krait chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_chrom_end
</td>
<td style="text-align:left;">
Blue-ringed sea krait chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Blue-ringed sea krait
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_orthology_type
</td>
<td style="text-align:left;">
Blue-ringed sea krait homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Blue-ringed sea krait gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Blue-ringed sea krait gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_goc_score
</td>
<td style="text-align:left;">
Blue-ringed sea krait Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_wga_coverage
</td>
<td style="text-align:left;">
Blue-ringed sea krait Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
llaticaudata_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Blue-ringed sea krait orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Bolivian squirrel monkey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Bolivian squirrel monkey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Bolivian squirrel monkey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_chromosome
</td>
<td style="text-align:left;">
Bolivian squirrel monkey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_chrom_start
</td>
<td style="text-align:left;">
Bolivian squirrel monkey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_chrom_end
</td>
<td style="text-align:left;">
Bolivian squirrel monkey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Bolivian squirrel monkey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_orthology_type
</td>
<td style="text-align:left;">
Bolivian squirrel monkey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Bolivian squirrel monkey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Bolivian squirrel monkey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_goc_score
</td>
<td style="text-align:left;">
Bolivian squirrel monkey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Bolivian squirrel monkey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sbboliviensis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Bolivian squirrel monkey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Bonobo gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Bonobo gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Bonobo protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_chromosome
</td>
<td style="text-align:left;">
Bonobo chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_chrom_start
</td>
<td style="text-align:left;">
Bonobo chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_chrom_end
</td>
<td style="text-align:left;">
Bonobo chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Bonobo
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_orthology_type
</td>
<td style="text-align:left;">
Bonobo homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Bonobo gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Bonobo gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_goc_score
</td>
<td style="text-align:left;">
Bonobo Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Bonobo Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppaniscus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Bonobo orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Brown trout gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Brown trout gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Brown trout protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_chromosome
</td>
<td style="text-align:left;">
Brown trout chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_chrom_start
</td>
<td style="text-align:left;">
Brown trout chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_chrom_end
</td>
<td style="text-align:left;">
Brown trout chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Brown trout
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_orthology_type
</td>
<td style="text-align:left;">
Brown trout homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Brown trout gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Brown trout gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_goc_score
</td>
<td style="text-align:left;">
Brown trout Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_wga_coverage
</td>
<td style="text-align:left;">
Brown trout Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
strutta_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Brown trout orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Burton’s mouthbrooder gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Burton’s mouthbrooder gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Burton’s mouthbrooder protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_chromosome
</td>
<td style="text-align:left;">
Burton’s mouthbrooder chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_chrom_start
</td>
<td style="text-align:left;">
Burton’s mouthbrooder chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_chrom_end
</td>
<td style="text-align:left;">
Burton’s mouthbrooder chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Burton’s mouthbrooder
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_orthology_type
</td>
<td style="text-align:left;">
Burton’s mouthbrooder homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Burton’s mouthbrooder gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Burton’s mouthbrooder gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_goc_score
</td>
<td style="text-align:left;">
Burton’s mouthbrooder Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_wga_coverage
</td>
<td style="text-align:left;">
Burton’s mouthbrooder Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hburtoni_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Burton’s mouthbrooder orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Bushbaby gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Bushbaby gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Bushbaby protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_chromosome
</td>
<td style="text-align:left;">
Bushbaby chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_chrom_start
</td>
<td style="text-align:left;">
Bushbaby chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_chrom_end
</td>
<td style="text-align:left;">
Bushbaby chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Bushbaby
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_orthology_type
</td>
<td style="text-align:left;">
Bushbaby homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Bushbaby gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Bushbaby gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_goc_score
</td>
<td style="text-align:left;">
Bushbaby Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Bushbaby Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ogarnettii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Bushbaby orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
C.intestinalis gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
C.intestinalis gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
C.intestinalis protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_chromosome
</td>
<td style="text-align:left;">
C.intestinalis chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_chrom_start
</td>
<td style="text-align:left;">
C.intestinalis chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_chrom_end
</td>
<td style="text-align:left;">
C.intestinalis chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with C.intestinalis
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_orthology_type
</td>
<td style="text-align:left;">
C.intestinalis homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target C.intestinalis gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target C.intestinalis gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_wga_coverage
</td>
<td style="text-align:left;">
C.intestinalis Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cintestinalis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
C.intestinalis orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_ensembl_gene
</td>
<td style="text-align:left;">
C.savignyi gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_associated_gene_name
</td>
<td style="text-align:left;">
C.savignyi gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
C.savignyi protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_chromosome
</td>
<td style="text-align:left;">
C.savignyi chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_chrom_start
</td>
<td style="text-align:left;">
C.savignyi chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_chrom_end
</td>
<td style="text-align:left;">
C.savignyi chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with C.savignyi
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_orthology_type
</td>
<td style="text-align:left;">
C.savignyi homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target C.savignyi gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target C.savignyi gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_wga_coverage
</td>
<td style="text-align:left;">
C.savignyi Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csavignyi_homolog_orthology_confidence
</td>
<td style="text-align:left;">
C.savignyi orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Caenorhabditis elegans gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Caenorhabditis elegans gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Caenorhabditis elegans protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_chromosome
</td>
<td style="text-align:left;">
Caenorhabditis elegans chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_chrom_start
</td>
<td style="text-align:left;">
Caenorhabditis elegans chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_chrom_end
</td>
<td style="text-align:left;">
Caenorhabditis elegans chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Caenorhabditis elegans
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_orthology_type
</td>
<td style="text-align:left;">
Caenorhabditis elegans homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Caenorhabditis elegans gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Caenorhabditis elegans gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
celegans_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Caenorhabditis elegans orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Cat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Cat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Cat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_chromosome
</td>
<td style="text-align:left;">
Cat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Cat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Cat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Cat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Cat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Cat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Cat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_goc_score
</td>
<td style="text-align:left;">
Cat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Cat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fcatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Cat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chacoan peccary gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chacoan peccary gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chacoan peccary protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_chromosome
</td>
<td style="text-align:left;">
Chacoan peccary chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_chrom_start
</td>
<td style="text-align:left;">
Chacoan peccary chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_chrom_end
</td>
<td style="text-align:left;">
Chacoan peccary chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chacoan peccary
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_orthology_type
</td>
<td style="text-align:left;">
Chacoan peccary homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chacoan peccary gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chacoan peccary gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_goc_score
</td>
<td style="text-align:left;">
Chacoan peccary Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chacoan peccary Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cwagneri_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chacoan peccary orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Channel bull blenny gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Channel bull blenny gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Channel bull blenny protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_chromosome
</td>
<td style="text-align:left;">
Channel bull blenny chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_chrom_start
</td>
<td style="text-align:left;">
Channel bull blenny chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_chrom_end
</td>
<td style="text-align:left;">
Channel bull blenny chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Channel bull blenny
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_orthology_type
</td>
<td style="text-align:left;">
Channel bull blenny homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Channel bull blenny gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Channel bull blenny gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_goc_score
</td>
<td style="text-align:left;">
Channel bull blenny Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_wga_coverage
</td>
<td style="text-align:left;">
Channel bull blenny Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgobio_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Channel bull blenny orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Channel catfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Channel catfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Channel catfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_chromosome
</td>
<td style="text-align:left;">
Channel catfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Channel catfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Channel catfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Channel catfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Channel catfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Channel catfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Channel catfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_goc_score
</td>
<td style="text-align:left;">
Channel catfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Channel catfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ipunctatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Channel catfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chicken gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chicken gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chicken protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_chromosome
</td>
<td style="text-align:left;">
Chicken chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_chrom_start
</td>
<td style="text-align:left;">
Chicken chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_chrom_end
</td>
<td style="text-align:left;">
Chicken chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chicken
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_orthology_type
</td>
<td style="text-align:left;">
Chicken homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chicken gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chicken gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_goc_score
</td>
<td style="text-align:left;">
Chicken Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chicken Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggallus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chicken orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chimpanzee gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chimpanzee gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chimpanzee protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_chromosome
</td>
<td style="text-align:left;">
Chimpanzee chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_chrom_start
</td>
<td style="text-align:left;">
Chimpanzee chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_chrom_end
</td>
<td style="text-align:left;">
Chimpanzee chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chimpanzee
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_orthology_type
</td>
<td style="text-align:left;">
Chimpanzee homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chimpanzee gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chimpanzee gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_goc_score
</td>
<td style="text-align:left;">
Chimpanzee Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chimpanzee Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptroglodytes_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chimpanzee orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_chromosome
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_chrom_start
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_chrom_end
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chinese hamster CHOK1GS
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_orthology_type
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chinese hamster CHOK1GS gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chinese hamster CHOK1GS gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_goc_score
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cgchok1gshd_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chinese hamster CHOK1GS orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chinese medaka gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chinese medaka gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chinese medaka protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_chromosome
</td>
<td style="text-align:left;">
Chinese medaka chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_chrom_start
</td>
<td style="text-align:left;">
Chinese medaka chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_chrom_end
</td>
<td style="text-align:left;">
Chinese medaka chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chinese medaka
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_orthology_type
</td>
<td style="text-align:left;">
Chinese medaka homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chinese medaka gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chinese medaka gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_goc_score
</td>
<td style="text-align:left;">
Chinese medaka Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chinese medaka Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
osinensis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chinese medaka orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chinese softshell turtle gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chinese softshell turtle gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chinese softshell turtle protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_chromosome
</td>
<td style="text-align:left;">
Chinese softshell turtle chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_chrom_start
</td>
<td style="text-align:left;">
Chinese softshell turtle chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_chrom_end
</td>
<td style="text-align:left;">
Chinese softshell turtle chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chinese softshell turtle
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_orthology_type
</td>
<td style="text-align:left;">
Chinese softshell turtle homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chinese softshell turtle gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chinese softshell turtle gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_goc_score
</td>
<td style="text-align:left;">
Chinese softshell turtle Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chinese softshell turtle Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinensis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chinese softshell turtle orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Chinook salmon gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Chinook salmon gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Chinook salmon protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_chromosome
</td>
<td style="text-align:left;">
Chinook salmon chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_chrom_start
</td>
<td style="text-align:left;">
Chinook salmon chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_chrom_end
</td>
<td style="text-align:left;">
Chinook salmon chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Chinook salmon
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_orthology_type
</td>
<td style="text-align:left;">
Chinook salmon homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Chinook salmon gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Chinook salmon gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_goc_score
</td>
<td style="text-align:left;">
Chinook salmon Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_wga_coverage
</td>
<td style="text-align:left;">
Chinook salmon Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
otshawytscha_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Chinook salmon orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Climbing perch gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Climbing perch gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Climbing perch protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_chromosome
</td>
<td style="text-align:left;">
Climbing perch chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_chrom_start
</td>
<td style="text-align:left;">
Climbing perch chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_chrom_end
</td>
<td style="text-align:left;">
Climbing perch chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Climbing perch
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_orthology_type
</td>
<td style="text-align:left;">
Climbing perch homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Climbing perch gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Climbing perch gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_goc_score
</td>
<td style="text-align:left;">
Climbing perch Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
atestudineus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Climbing perch orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Clown anemonefish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Clown anemonefish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Clown anemonefish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_chromosome
</td>
<td style="text-align:left;">
Clown anemonefish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_chrom_start
</td>
<td style="text-align:left;">
Clown anemonefish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_chrom_end
</td>
<td style="text-align:left;">
Clown anemonefish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Clown anemonefish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_orthology_type
</td>
<td style="text-align:left;">
Clown anemonefish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Clown anemonefish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Clown anemonefish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_goc_score
</td>
<td style="text-align:left;">
Clown anemonefish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_wga_coverage
</td>
<td style="text-align:left;">
Clown anemonefish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
aocellaris_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Clown anemonefish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Coelacanth gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Coelacanth gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Coelacanth protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_chromosome
</td>
<td style="text-align:left;">
Coelacanth chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_chrom_start
</td>
<td style="text-align:left;">
Coelacanth chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_chrom_end
</td>
<td style="text-align:left;">
Coelacanth chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Coelacanth
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_orthology_type
</td>
<td style="text-align:left;">
Coelacanth homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Coelacanth gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Coelacanth gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_goc_score
</td>
<td style="text-align:left;">
Coelacanth Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_wga_coverage
</td>
<td style="text-align:left;">
Coelacanth Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lchalumnae_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Coelacanth orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Coho salmon gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Coho salmon gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Coho salmon protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_chromosome
</td>
<td style="text-align:left;">
Coho salmon chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_chrom_start
</td>
<td style="text-align:left;">
Coho salmon chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_chrom_end
</td>
<td style="text-align:left;">
Coho salmon chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Coho salmon
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_orthology_type
</td>
<td style="text-align:left;">
Coho salmon homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Coho salmon gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Coho salmon gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_goc_score
</td>
<td style="text-align:left;">
Coho salmon Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_wga_coverage
</td>
<td style="text-align:left;">
Coho salmon Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
okisutch_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Coho salmon orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Collared flycatcher gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Collared flycatcher gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Collared flycatcher protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_chromosome
</td>
<td style="text-align:left;">
Collared flycatcher chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_chrom_start
</td>
<td style="text-align:left;">
Collared flycatcher chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_chrom_end
</td>
<td style="text-align:left;">
Collared flycatcher chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Collared flycatcher
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_orthology_type
</td>
<td style="text-align:left;">
Collared flycatcher homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Collared flycatcher gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Collared flycatcher gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_goc_score
</td>
<td style="text-align:left;">
Collared flycatcher Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
falbicollis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Collared flycatcher orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Common canary gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Common canary gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Common canary protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_chromosome
</td>
<td style="text-align:left;">
Common canary chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_chrom_start
</td>
<td style="text-align:left;">
Common canary chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_chrom_end
</td>
<td style="text-align:left;">
Common canary chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Common canary
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_orthology_type
</td>
<td style="text-align:left;">
Common canary homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Common canary gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Common canary gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_goc_score
</td>
<td style="text-align:left;">
Common canary Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_wga_coverage
</td>
<td style="text-align:left;">
Common canary Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scanaria_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Common canary orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Common carp gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Common carp gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Common carp protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_chromosome
</td>
<td style="text-align:left;">
Common carp chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_chrom_start
</td>
<td style="text-align:left;">
Common carp chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_chrom_end
</td>
<td style="text-align:left;">
Common carp chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Common carp
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_orthology_type
</td>
<td style="text-align:left;">
Common carp homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Common carp gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Common carp gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_goc_score
</td>
<td style="text-align:left;">
Common carp Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_wga_coverage
</td>
<td style="text-align:left;">
Common carp Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ccarpio_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Common carp orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Common wall lizard gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Common wall lizard gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Common wall lizard protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_chromosome
</td>
<td style="text-align:left;">
Common wall lizard chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_chrom_start
</td>
<td style="text-align:left;">
Common wall lizard chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_chrom_end
</td>
<td style="text-align:left;">
Common wall lizard chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Common wall lizard
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_orthology_type
</td>
<td style="text-align:left;">
Common wall lizard homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Common wall lizard gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Common wall lizard gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_goc_score
</td>
<td style="text-align:left;">
Common wall lizard Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Common wall lizard Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmuralis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Common wall lizard orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Common wombat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Common wombat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Common wombat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_chromosome
</td>
<td style="text-align:left;">
Common wombat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_chrom_start
</td>
<td style="text-align:left;">
Common wombat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_chrom_end
</td>
<td style="text-align:left;">
Common wombat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Common wombat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_orthology_type
</td>
<td style="text-align:left;">
Common wombat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Common wombat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Common wombat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_goc_score
</td>
<td style="text-align:left;">
Common wombat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Common wombat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vursinus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Common wombat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Coquerel’s sifaka gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Coquerel’s sifaka gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Coquerel’s sifaka protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_chromosome
</td>
<td style="text-align:left;">
Coquerel’s sifaka chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_chrom_start
</td>
<td style="text-align:left;">
Coquerel’s sifaka chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_chrom_end
</td>
<td style="text-align:left;">
Coquerel’s sifaka chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Coquerel’s sifaka
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_orthology_type
</td>
<td style="text-align:left;">
Coquerel’s sifaka homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Coquerel’s sifaka gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Coquerel’s sifaka gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_goc_score
</td>
<td style="text-align:left;">
Coquerel’s sifaka Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_wga_coverage
</td>
<td style="text-align:left;">
Coquerel’s sifaka Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcoquereli_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Coquerel’s sifaka orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Cow gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Cow gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Cow protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_chromosome
</td>
<td style="text-align:left;">
Cow chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_chrom_start
</td>
<td style="text-align:left;">
Cow chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_chrom_end
</td>
<td style="text-align:left;">
Cow chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Cow
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_orthology_type
</td>
<td style="text-align:left;">
Cow homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Cow gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Cow gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_goc_score
</td>
<td style="text-align:left;">
Cow Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Cow Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
btaurus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Cow orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Crab-eating macaque gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Crab-eating macaque gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Crab-eating macaque protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_chromosome
</td>
<td style="text-align:left;">
Crab-eating macaque chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_chrom_start
</td>
<td style="text-align:left;">
Crab-eating macaque chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_chrom_end
</td>
<td style="text-align:left;">
Crab-eating macaque chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Crab-eating macaque
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_orthology_type
</td>
<td style="text-align:left;">
Crab-eating macaque homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Crab-eating macaque gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Crab-eating macaque gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_goc_score
</td>
<td style="text-align:left;">
Crab-eating macaque Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Crab-eating macaque Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mfascicularis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Crab-eating macaque orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Degu gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Degu gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Degu protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_chromosome
</td>
<td style="text-align:left;">
Degu chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_chrom_start
</td>
<td style="text-align:left;">
Degu chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_chrom_end
</td>
<td style="text-align:left;">
Degu chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Degu
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_orthology_type
</td>
<td style="text-align:left;">
Degu homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Degu gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Degu gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_goc_score
</td>
<td style="text-align:left;">
Degu Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Degu Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
odegus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Degu orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Denticle herring gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Denticle herring gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Denticle herring protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_chromosome
</td>
<td style="text-align:left;">
Denticle herring chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_chrom_start
</td>
<td style="text-align:left;">
Denticle herring chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_chrom_end
</td>
<td style="text-align:left;">
Denticle herring chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Denticle herring
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_orthology_type
</td>
<td style="text-align:left;">
Denticle herring homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Denticle herring gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Denticle herring gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_goc_score
</td>
<td style="text-align:left;">
Denticle herring Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_wga_coverage
</td>
<td style="text-align:left;">
Denticle herring Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dclupeoides_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Denticle herring orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Dingo gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Dingo gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Dingo protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_chromosome
</td>
<td style="text-align:left;">
Dingo chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_chrom_start
</td>
<td style="text-align:left;">
Dingo chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_chrom_end
</td>
<td style="text-align:left;">
Dingo chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Dingo
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_orthology_type
</td>
<td style="text-align:left;">
Dingo homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Dingo gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Dingo gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_goc_score
</td>
<td style="text-align:left;">
Dingo Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_wga_coverage
</td>
<td style="text-align:left;">
Dingo Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cldingo_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Dingo orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Dog gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Dog gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Dog protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_chromosome
</td>
<td style="text-align:left;">
Dog chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_chrom_start
</td>
<td style="text-align:left;">
Dog chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_chrom_end
</td>
<td style="text-align:left;">
Dog chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Dog
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_orthology_type
</td>
<td style="text-align:left;">
Dog homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Dog gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Dog gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_goc_score
</td>
<td style="text-align:left;">
Dog Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_wga_coverage
</td>
<td style="text-align:left;">
Dog Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clfamiliaris_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Dog orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Dolphin gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Dolphin gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Dolphin protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_chromosome
</td>
<td style="text-align:left;">
Dolphin chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Dolphin chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Dolphin chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Dolphin
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Dolphin homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Dolphin gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Dolphin gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_goc_score
</td>
<td style="text-align:left;">
Dolphin Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Dolphin Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ttruncatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Dolphin orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Domestic yak gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Domestic yak gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Domestic yak protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_chromosome
</td>
<td style="text-align:left;">
Domestic yak chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_chrom_start
</td>
<td style="text-align:left;">
Domestic yak chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_chrom_end
</td>
<td style="text-align:left;">
Domestic yak chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Domestic yak
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_orthology_type
</td>
<td style="text-align:left;">
Domestic yak homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Domestic yak gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Domestic yak gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_goc_score
</td>
<td style="text-align:left;">
Domestic yak Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_wga_coverage
</td>
<td style="text-align:left;">
Domestic yak Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bgrunniens_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Domestic yak orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Donkey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Donkey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Donkey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_chromosome
</td>
<td style="text-align:left;">
Donkey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_chrom_start
</td>
<td style="text-align:left;">
Donkey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_chrom_end
</td>
<td style="text-align:left;">
Donkey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Donkey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_orthology_type
</td>
<td style="text-align:left;">
Donkey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Donkey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Donkey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_goc_score
</td>
<td style="text-align:left;">
Donkey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Donkey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eaasinus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Donkey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Drill gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Drill gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Drill protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_chromosome
</td>
<td style="text-align:left;">
Drill chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_chrom_start
</td>
<td style="text-align:left;">
Drill chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_chrom_end
</td>
<td style="text-align:left;">
Drill chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Drill
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_orthology_type
</td>
<td style="text-align:left;">
Drill homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Drill gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Drill gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_goc_score
</td>
<td style="text-align:left;">
Drill Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Drill Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mleucophaeus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Drill orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Drosophila melanogaster gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Drosophila melanogaster gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Drosophila melanogaster protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_chromosome
</td>
<td style="text-align:left;">
Drosophila melanogaster chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_chrom_start
</td>
<td style="text-align:left;">
Drosophila melanogaster chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_chrom_end
</td>
<td style="text-align:left;">
Drosophila melanogaster chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Drosophila melanogaster
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_orthology_type
</td>
<td style="text-align:left;">
Drosophila melanogaster homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Drosophila melanogaster gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Drosophila melanogaster gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dmelanogaster_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Drosophila melanogaster orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Duck gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Duck gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Duck protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_chromosome
</td>
<td style="text-align:left;">
Duck chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_chrom_start
</td>
<td style="text-align:left;">
Duck chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_chrom_end
</td>
<td style="text-align:left;">
Duck chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Duck
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_orthology_type
</td>
<td style="text-align:left;">
Duck homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Duck gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Duck gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_goc_score
</td>
<td style="text-align:left;">
Duck Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_wga_coverage
</td>
<td style="text-align:left;">
Duck Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
applatyrhynchos_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Duck orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Eastern brown snake gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Eastern brown snake gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Eastern brown snake protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_chromosome
</td>
<td style="text-align:left;">
Eastern brown snake chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_chrom_start
</td>
<td style="text-align:left;">
Eastern brown snake chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_chrom_end
</td>
<td style="text-align:left;">
Eastern brown snake chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Eastern brown snake
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_orthology_type
</td>
<td style="text-align:left;">
Eastern brown snake homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Eastern brown snake gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Eastern brown snake gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_goc_score
</td>
<td style="text-align:left;">
Eastern brown snake Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Eastern brown snake Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptextilis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Eastern brown snake orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Eastern happy gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Eastern happy gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Eastern happy protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_chromosome
</td>
<td style="text-align:left;">
Eastern happy chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_chrom_start
</td>
<td style="text-align:left;">
Eastern happy chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_chrom_end
</td>
<td style="text-align:left;">
Eastern happy chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Eastern happy
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_orthology_type
</td>
<td style="text-align:left;">
Eastern happy homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Eastern happy gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Eastern happy gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_goc_score
</td>
<td style="text-align:left;">
Eastern happy Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_wga_coverage
</td>
<td style="text-align:left;">
Eastern happy Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acalliptera_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Eastern happy orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Electric eel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Electric eel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Electric eel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_chromosome
</td>
<td style="text-align:left;">
Electric eel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_chrom_start
</td>
<td style="text-align:left;">
Electric eel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_chrom_end
</td>
<td style="text-align:left;">
Electric eel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Electric eel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_orthology_type
</td>
<td style="text-align:left;">
Electric eel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Electric eel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Electric eel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_goc_score
</td>
<td style="text-align:left;">
Electric eel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Electric eel Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eelectricus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Electric eel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Elephant gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Elephant gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Elephant protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_chromosome
</td>
<td style="text-align:left;">
Elephant chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_chrom_start
</td>
<td style="text-align:left;">
Elephant chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_chrom_end
</td>
<td style="text-align:left;">
Elephant chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Elephant
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_orthology_type
</td>
<td style="text-align:left;">
Elephant homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Elephant gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Elephant gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_goc_score
</td>
<td style="text-align:left;">
Elephant Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_wga_coverage
</td>
<td style="text-align:left;">
Elephant Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lafricana_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Elephant orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Elephant shark gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Elephant shark gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Elephant shark protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_chromosome
</td>
<td style="text-align:left;">
Elephant shark chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_chrom_start
</td>
<td style="text-align:left;">
Elephant shark chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_chrom_end
</td>
<td style="text-align:left;">
Elephant shark chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Elephant shark
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_orthology_type
</td>
<td style="text-align:left;">
Elephant shark homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Elephant shark gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Elephant shark gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Elephant shark Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cmilii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Elephant shark orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Eurasian red squirrel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Eurasian red squirrel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Eurasian red squirrel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_chromosome
</td>
<td style="text-align:left;">
Eurasian red squirrel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_chrom_start
</td>
<td style="text-align:left;">
Eurasian red squirrel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_chrom_end
</td>
<td style="text-align:left;">
Eurasian red squirrel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Eurasian red squirrel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_orthology_type
</td>
<td style="text-align:left;">
Eurasian red squirrel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Eurasian red squirrel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Eurasian red squirrel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_goc_score
</td>
<td style="text-align:left;">
Eurasian red squirrel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_wga_coverage
</td>
<td style="text-align:left;">
Eurasian red squirrel Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
svulgaris_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Eurasian red squirrel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_ensembl_gene
</td>
<td style="text-align:left;">
European seabass gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_associated_gene_name
</td>
<td style="text-align:left;">
European seabass gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
European seabass protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_chromosome
</td>
<td style="text-align:left;">
European seabass chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_chrom_start
</td>
<td style="text-align:left;">
European seabass chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_chrom_end
</td>
<td style="text-align:left;">
European seabass chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with European seabass
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_orthology_type
</td>
<td style="text-align:left;">
European seabass homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target European seabass gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target European seabass gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_goc_score
</td>
<td style="text-align:left;">
European seabass Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_wga_coverage
</td>
<td style="text-align:left;">
European seabass Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dlabrax_homolog_orthology_confidence
</td>
<td style="text-align:left;">
European seabass orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Ferret gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Ferret gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Ferret protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_chromosome
</td>
<td style="text-align:left;">
Ferret chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_chrom_start
</td>
<td style="text-align:left;">
Ferret chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_chrom_end
</td>
<td style="text-align:left;">
Ferret chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Ferret
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_orthology_type
</td>
<td style="text-align:left;">
Ferret homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Ferret gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Ferret gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_goc_score
</td>
<td style="text-align:left;">
Ferret Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_wga_coverage
</td>
<td style="text-align:left;">
Ferret Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpfuro_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Ferret orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Fugu gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Fugu gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Fugu protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_chromosome
</td>
<td style="text-align:left;">
Fugu chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_chrom_start
</td>
<td style="text-align:left;">
Fugu chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_chrom_end
</td>
<td style="text-align:left;">
Fugu chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Fugu
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_orthology_type
</td>
<td style="text-align:left;">
Fugu homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Fugu gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Fugu gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_goc_score
</td>
<td style="text-align:left;">
Fugu Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_wga_coverage
</td>
<td style="text-align:left;">
Fugu Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
trubripes_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Fugu orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Giant panda gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Giant panda gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Giant panda protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_chromosome
</td>
<td style="text-align:left;">
Giant panda chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_chrom_start
</td>
<td style="text-align:left;">
Giant panda chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_chrom_end
</td>
<td style="text-align:left;">
Giant panda chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Giant panda
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_orthology_type
</td>
<td style="text-align:left;">
Giant panda homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Giant panda gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Giant panda gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_goc_score
</td>
<td style="text-align:left;">
Giant panda Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amelanoleuca_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Giant panda orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Gibbon gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Gibbon gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Gibbon protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_chromosome
</td>
<td style="text-align:left;">
Gibbon chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_chrom_start
</td>
<td style="text-align:left;">
Gibbon chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_chrom_end
</td>
<td style="text-align:left;">
Gibbon chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Gibbon
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_orthology_type
</td>
<td style="text-align:left;">
Gibbon homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Gibbon gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Gibbon gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_goc_score
</td>
<td style="text-align:left;">
Gibbon Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_wga_coverage
</td>
<td style="text-align:left;">
Gibbon Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nleucogenys_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Gibbon orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Gilthead seabream gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Gilthead seabream gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Gilthead seabream protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_chromosome
</td>
<td style="text-align:left;">
Gilthead seabream chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_chrom_start
</td>
<td style="text-align:left;">
Gilthead seabream chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_chrom_end
</td>
<td style="text-align:left;">
Gilthead seabream chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Gilthead seabream
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_orthology_type
</td>
<td style="text-align:left;">
Gilthead seabream homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Gilthead seabream gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Gilthead seabream gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_goc_score
</td>
<td style="text-align:left;">
Gilthead seabream Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_wga_coverage
</td>
<td style="text-align:left;">
Gilthead seabream Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saurata_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Gilthead seabream orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Goat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Goat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Goat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_chromosome
</td>
<td style="text-align:left;">
Goat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_chrom_start
</td>
<td style="text-align:left;">
Goat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_chrom_end
</td>
<td style="text-align:left;">
Goat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Goat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_orthology_type
</td>
<td style="text-align:left;">
Goat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Goat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Goat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_goc_score
</td>
<td style="text-align:left;">
Goat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Goat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chircus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Goat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Golden Hamster gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Golden Hamster gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Golden Hamster protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_chromosome
</td>
<td style="text-align:left;">
Golden Hamster chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_chrom_start
</td>
<td style="text-align:left;">
Golden Hamster chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_chrom_end
</td>
<td style="text-align:left;">
Golden Hamster chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Golden Hamster
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_orthology_type
</td>
<td style="text-align:left;">
Golden Hamster homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Golden Hamster gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Golden Hamster gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_goc_score
</td>
<td style="text-align:left;">
Golden Hamster Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Golden Hamster Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mauratus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Golden Hamster orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Golden eagle gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Golden eagle gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Golden eagle protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_chromosome
</td>
<td style="text-align:left;">
Golden eagle chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_chrom_start
</td>
<td style="text-align:left;">
Golden eagle chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_chrom_end
</td>
<td style="text-align:left;">
Golden eagle chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Golden eagle
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_orthology_type
</td>
<td style="text-align:left;">
Golden eagle homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Golden eagle gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Golden eagle gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_goc_score
</td>
<td style="text-align:left;">
Golden eagle Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_wga_coverage
</td>
<td style="text-align:left;">
Golden eagle Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acchrysaetos_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Golden eagle orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Golden snub-nosed monkey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Golden snub-nosed monkey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Golden snub-nosed monkey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_chromosome
</td>
<td style="text-align:left;">
Golden snub-nosed monkey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_chrom_start
</td>
<td style="text-align:left;">
Golden snub-nosed monkey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_chrom_end
</td>
<td style="text-align:left;">
Golden snub-nosed monkey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Golden snub-nosed monkey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_orthology_type
</td>
<td style="text-align:left;">
Golden snub-nosed monkey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Golden snub-nosed monkey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Golden snub-nosed monkey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_goc_score
</td>
<td style="text-align:left;">
Golden snub-nosed monkey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_wga_coverage
</td>
<td style="text-align:left;">
Golden snub-nosed monkey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rroxellana_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Golden snub-nosed monkey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Golden-line barbel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Golden-line barbel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Golden-line barbel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_chromosome
</td>
<td style="text-align:left;">
Golden-line barbel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_chrom_start
</td>
<td style="text-align:left;">
Golden-line barbel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_chrom_end
</td>
<td style="text-align:left;">
Golden-line barbel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Golden-line barbel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_orthology_type
</td>
<td style="text-align:left;">
Golden-line barbel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Golden-line barbel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Golden-line barbel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_goc_score
</td>
<td style="text-align:left;">
Golden-line barbel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_wga_coverage
</td>
<td style="text-align:left;">
Golden-line barbel Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sgrahami_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Golden-line barbel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Goldfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Goldfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Goldfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_chromosome
</td>
<td style="text-align:left;">
Goldfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_chrom_start
</td>
<td style="text-align:left;">
Goldfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_chrom_end
</td>
<td style="text-align:left;">
Goldfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Goldfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_orthology_type
</td>
<td style="text-align:left;">
Goldfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Goldfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Goldfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_goc_score
</td>
<td style="text-align:left;">
Goldfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Goldfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cauratus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Goldfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_chromosome
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_chrom_start
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_chrom_end
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Goodes thornscrub tortoise
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_orthology_type
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Goodes thornscrub tortoise gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Goodes thornscrub tortoise gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_goc_score
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_wga_coverage
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gevgoodei_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Goodes thornscrub tortoise orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Gorilla gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Gorilla gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Gorilla protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_chromosome
</td>
<td style="text-align:left;">
Gorilla chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_chrom_start
</td>
<td style="text-align:left;">
Gorilla chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_chrom_end
</td>
<td style="text-align:left;">
Gorilla chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Gorilla
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_orthology_type
</td>
<td style="text-align:left;">
Gorilla homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Gorilla gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Gorilla gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_goc_score
</td>
<td style="text-align:left;">
Gorilla Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_wga_coverage
</td>
<td style="text-align:left;">
Gorilla Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ggorilla_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Gorilla orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Great Tit gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Great Tit gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Great Tit protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_chromosome
</td>
<td style="text-align:left;">
Great Tit chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_chrom_start
</td>
<td style="text-align:left;">
Great Tit chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_chrom_end
</td>
<td style="text-align:left;">
Great Tit chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Great Tit
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_orthology_type
</td>
<td style="text-align:left;">
Great Tit homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Great Tit gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Great Tit gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_goc_score
</td>
<td style="text-align:left;">
Great Tit Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_wga_coverage
</td>
<td style="text-align:left;">
Great Tit Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmajor_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Great Tit orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Greater amberjack gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Greater amberjack gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Greater amberjack protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_chromosome
</td>
<td style="text-align:left;">
Greater amberjack chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_chrom_start
</td>
<td style="text-align:left;">
Greater amberjack chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_chrom_end
</td>
<td style="text-align:left;">
Greater amberjack chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Greater amberjack
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_orthology_type
</td>
<td style="text-align:left;">
Greater amberjack homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Greater amberjack gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Greater amberjack gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_goc_score
</td>
<td style="text-align:left;">
Greater amberjack Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_wga_coverage
</td>
<td style="text-align:left;">
Greater amberjack Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sdumerili_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Greater amberjack orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Greater bamboo lemur gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Greater bamboo lemur gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Greater bamboo lemur protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_chromosome
</td>
<td style="text-align:left;">
Greater bamboo lemur chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_chrom_start
</td>
<td style="text-align:left;">
Greater bamboo lemur chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_chrom_end
</td>
<td style="text-align:left;">
Greater bamboo lemur chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Greater bamboo lemur
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_orthology_type
</td>
<td style="text-align:left;">
Greater bamboo lemur homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Greater bamboo lemur gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Greater bamboo lemur gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_goc_score
</td>
<td style="text-align:left;">
Greater bamboo lemur Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Greater bamboo lemur Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psimus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Greater bamboo lemur orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Greater horseshoe bat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Greater horseshoe bat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Greater horseshoe bat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_chromosome
</td>
<td style="text-align:left;">
Greater horseshoe bat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_chrom_start
</td>
<td style="text-align:left;">
Greater horseshoe bat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_chrom_end
</td>
<td style="text-align:left;">
Greater horseshoe bat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Greater horseshoe bat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_orthology_type
</td>
<td style="text-align:left;">
Greater horseshoe bat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Greater horseshoe bat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Greater horseshoe bat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_goc_score
</td>
<td style="text-align:left;">
Greater horseshoe bat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_wga_coverage
</td>
<td style="text-align:left;">
Greater horseshoe bat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rferrumequinum_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Greater horseshoe bat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Green anole gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Green anole gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Green anole protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_chromosome
</td>
<td style="text-align:left;">
Green anole chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_chrom_start
</td>
<td style="text-align:left;">
Green anole chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_chrom_end
</td>
<td style="text-align:left;">
Green anole chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Green anole
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_orthology_type
</td>
<td style="text-align:left;">
Green anole homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Green anole gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Green anole gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_goc_score
</td>
<td style="text-align:left;">
Green anole Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acarolinensis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Green anole orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Guinea Pig gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Guinea Pig gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Guinea Pig protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_chromosome
</td>
<td style="text-align:left;">
Guinea Pig chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_chrom_start
</td>
<td style="text-align:left;">
Guinea Pig chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_chrom_end
</td>
<td style="text-align:left;">
Guinea Pig chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Guinea Pig
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_orthology_type
</td>
<td style="text-align:left;">
Guinea Pig homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Guinea Pig gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Guinea Pig gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_goc_score
</td>
<td style="text-align:left;">
Guinea Pig Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Guinea Pig Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cporcellus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Guinea Pig orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Guppy gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Guppy gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Guppy protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_chromosome
</td>
<td style="text-align:left;">
Guppy chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_chrom_start
</td>
<td style="text-align:left;">
Guppy chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_chrom_end
</td>
<td style="text-align:left;">
Guppy chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Guppy
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_orthology_type
</td>
<td style="text-align:left;">
Guppy homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Guppy gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Guppy gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_goc_score
</td>
<td style="text-align:left;">
Guppy Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_wga_coverage
</td>
<td style="text-align:left;">
Guppy Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
preticulata_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Guppy orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Hagfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Hagfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Hagfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_chromosome
</td>
<td style="text-align:left;">
Hagfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_chrom_start
</td>
<td style="text-align:left;">
Hagfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_chrom_end
</td>
<td style="text-align:left;">
Hagfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Hagfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_orthology_type
</td>
<td style="text-align:left;">
Hagfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Hagfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Hagfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_wga_coverage
</td>
<td style="text-align:left;">
Hagfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eburgeri_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Hagfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Hedgehog gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Hedgehog gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Hedgehog protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_chromosome
</td>
<td style="text-align:left;">
Hedgehog chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_chrom_start
</td>
<td style="text-align:left;">
Hedgehog chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_chrom_end
</td>
<td style="text-align:left;">
Hedgehog chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Hedgehog
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_orthology_type
</td>
<td style="text-align:left;">
Hedgehog homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Hedgehog gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Hedgehog gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_goc_score
</td>
<td style="text-align:left;">
Hedgehog Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Hedgehog Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
eeuropaeus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Hedgehog orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Horse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Horse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Horse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_chromosome
</td>
<td style="text-align:left;">
Horse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_chrom_start
</td>
<td style="text-align:left;">
Horse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_chrom_end
</td>
<td style="text-align:left;">
Horse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Horse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_orthology_type
</td>
<td style="text-align:left;">
Horse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Horse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Horse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_goc_score
</td>
<td style="text-align:left;">
Horse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Horse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecaballus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Horse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Huchen gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Huchen gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Huchen protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_chromosome
</td>
<td style="text-align:left;">
Huchen chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_chrom_start
</td>
<td style="text-align:left;">
Huchen chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_chrom_end
</td>
<td style="text-align:left;">
Huchen chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Huchen
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_orthology_type
</td>
<td style="text-align:left;">
Huchen homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Huchen gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Huchen gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_goc_score
</td>
<td style="text-align:left;">
Huchen Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_wga_coverage
</td>
<td style="text-align:left;">
Huchen Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hhucho_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Huchen orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_chromosome
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_chrom_start
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_chrom_end
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Hybrid - Bos Indicus
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_orthology_type
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Hybrid - Bos Indicus gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Hybrid - Bos Indicus gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_goc_score
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_wga_coverage
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bihybrid_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Hybrid - Bos Indicus orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Hyrax gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Hyrax gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Hyrax protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_chromosome
</td>
<td style="text-align:left;">
Hyrax chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_chrom_start
</td>
<td style="text-align:left;">
Hyrax chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_chrom_end
</td>
<td style="text-align:left;">
Hyrax chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Hyrax
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_orthology_type
</td>
<td style="text-align:left;">
Hyrax homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Hyrax gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Hyrax gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_goc_score
</td>
<td style="text-align:left;">
Hyrax Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Hyrax Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcapensis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Hyrax orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Indian cobra gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Indian cobra gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Indian cobra protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_chromosome
</td>
<td style="text-align:left;">
Indian cobra chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_chrom_start
</td>
<td style="text-align:left;">
Indian cobra chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_chrom_end
</td>
<td style="text-align:left;">
Indian cobra chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Indian cobra
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_orthology_type
</td>
<td style="text-align:left;">
Indian cobra homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Indian cobra gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Indian cobra gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_goc_score
</td>
<td style="text-align:left;">
Indian cobra Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_wga_coverage
</td>
<td style="text-align:left;">
Indian cobra Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nnaja_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Indian cobra orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Indian medaka gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Indian medaka gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Indian medaka protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_chromosome
</td>
<td style="text-align:left;">
Indian medaka chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_chrom_start
</td>
<td style="text-align:left;">
Indian medaka chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_chrom_end
</td>
<td style="text-align:left;">
Indian medaka chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Indian medaka
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_orthology_type
</td>
<td style="text-align:left;">
Indian medaka homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Indian medaka gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Indian medaka gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_goc_score
</td>
<td style="text-align:left;">
Indian medaka Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_wga_coverage
</td>
<td style="text-align:left;">
Indian medaka Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omelastigma_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Indian medaka orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Japanese medaka HdrR gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Japanese medaka HdrR gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Japanese medaka HdrR protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_chromosome
</td>
<td style="text-align:left;">
Japanese medaka HdrR chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_chrom_start
</td>
<td style="text-align:left;">
Japanese medaka HdrR chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_chrom_end
</td>
<td style="text-align:left;">
Japanese medaka HdrR chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Japanese medaka HdrR
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_orthology_type
</td>
<td style="text-align:left;">
Japanese medaka HdrR homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Japanese medaka HdrR gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Japanese medaka HdrR gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_goc_score
</td>
<td style="text-align:left;">
Japanese medaka HdrR Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_wga_coverage
</td>
<td style="text-align:left;">
Japanese medaka HdrR Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
olatipes_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Japanese medaka HdrR orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Japanese quail gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Japanese quail gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Japanese quail protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_chromosome
</td>
<td style="text-align:left;">
Japanese quail chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_chrom_start
</td>
<td style="text-align:left;">
Japanese quail chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_chrom_end
</td>
<td style="text-align:left;">
Japanese quail chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Japanese quail
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_orthology_type
</td>
<td style="text-align:left;">
Japanese quail homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Japanese quail gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Japanese quail gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_goc_score
</td>
<td style="text-align:left;">
Japanese quail Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_wga_coverage
</td>
<td style="text-align:left;">
Japanese quail Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjaponica_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Japanese quail orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Javanese ricefish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Javanese ricefish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Javanese ricefish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_chromosome
</td>
<td style="text-align:left;">
Javanese ricefish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_chrom_start
</td>
<td style="text-align:left;">
Javanese ricefish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_chrom_end
</td>
<td style="text-align:left;">
Javanese ricefish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Javanese ricefish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_orthology_type
</td>
<td style="text-align:left;">
Javanese ricefish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Javanese ricefish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Javanese ricefish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_goc_score
</td>
<td style="text-align:left;">
Javanese ricefish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Javanese ricefish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ojavanicus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Javanese ricefish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Kakapo gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Kakapo gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Kakapo protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_chromosome
</td>
<td style="text-align:left;">
Kakapo chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_chrom_start
</td>
<td style="text-align:left;">
Kakapo chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_chrom_end
</td>
<td style="text-align:left;">
Kakapo chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Kakapo
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_orthology_type
</td>
<td style="text-align:left;">
Kakapo homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Kakapo gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Kakapo gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_goc_score
</td>
<td style="text-align:left;">
Kakapo Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_wga_coverage
</td>
<td style="text-align:left;">
Kakapo Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
shabroptila_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Kakapo orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Kangaroo rat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Kangaroo rat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Kangaroo rat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_chromosome
</td>
<td style="text-align:left;">
Kangaroo rat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_chrom_start
</td>
<td style="text-align:left;">
Kangaroo rat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_chrom_end
</td>
<td style="text-align:left;">
Kangaroo rat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Kangaroo rat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_orthology_type
</td>
<td style="text-align:left;">
Kangaroo rat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Kangaroo rat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Kangaroo rat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_goc_score
</td>
<td style="text-align:left;">
Kangaroo rat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Kangaroo rat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
dordii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Kangaroo rat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Koala gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Koala gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Koala protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_chromosome
</td>
<td style="text-align:left;">
Koala chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_chrom_start
</td>
<td style="text-align:left;">
Koala chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_chrom_end
</td>
<td style="text-align:left;">
Koala chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Koala
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_orthology_type
</td>
<td style="text-align:left;">
Koala homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Koala gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Koala gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_goc_score
</td>
<td style="text-align:left;">
Koala Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Koala Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcinereus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Koala orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Lamprey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Lamprey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Lamprey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_chromosome
</td>
<td style="text-align:left;">
Lamprey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_chrom_start
</td>
<td style="text-align:left;">
Lamprey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_chrom_end
</td>
<td style="text-align:left;">
Lamprey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Lamprey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_orthology_type
</td>
<td style="text-align:left;">
Lamprey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Lamprey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Lamprey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Lamprey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmarinus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Lamprey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Large yellow croaker gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Large yellow croaker gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Large yellow croaker protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_chromosome
</td>
<td style="text-align:left;">
Large yellow croaker chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_chrom_start
</td>
<td style="text-align:left;">
Large yellow croaker chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_chrom_end
</td>
<td style="text-align:left;">
Large yellow croaker chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Large yellow croaker
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_orthology_type
</td>
<td style="text-align:left;">
Large yellow croaker homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Large yellow croaker gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Large yellow croaker gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_goc_score
</td>
<td style="text-align:left;">
Large yellow croaker Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_wga_coverage
</td>
<td style="text-align:left;">
Large yellow croaker Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lcrocea_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Large yellow croaker orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Leishan spiny toad gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Leishan spiny toad gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Leishan spiny toad protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_chromosome
</td>
<td style="text-align:left;">
Leishan spiny toad chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_chrom_start
</td>
<td style="text-align:left;">
Leishan spiny toad chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_chrom_end
</td>
<td style="text-align:left;">
Leishan spiny toad chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Leishan spiny toad
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_orthology_type
</td>
<td style="text-align:left;">
Leishan spiny toad homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Leishan spiny toad gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Leishan spiny toad gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_goc_score
</td>
<td style="text-align:left;">
Leishan spiny toad Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_wga_coverage
</td>
<td style="text-align:left;">
Leishan spiny toad Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
lleishanense_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Leishan spiny toad orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Leopard gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Leopard gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Leopard protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_chromosome
</td>
<td style="text-align:left;">
Leopard chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_chrom_start
</td>
<td style="text-align:left;">
Leopard chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_chrom_end
</td>
<td style="text-align:left;">
Leopard chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Leopard
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_orthology_type
</td>
<td style="text-align:left;">
Leopard homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Leopard gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Leopard gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_goc_score
</td>
<td style="text-align:left;">
Leopard Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Leopard Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ppardus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Leopard orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_chromosome
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_chrom_start
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_chrom_end
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Lesser Egyptian jerboa
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_orthology_type
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Lesser Egyptian jerboa gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Lesser Egyptian jerboa gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_goc_score
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
jjaculus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Lesser Egyptian jerboa orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_chromosome
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_chrom_start
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_chrom_end
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Lesser hedgehog tenrec
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_orthology_type
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Lesser hedgehog tenrec gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Lesser hedgehog tenrec gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_goc_score
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_wga_coverage
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
etelfairi_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Lesser hedgehog tenrec orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Lion gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Lion gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Lion protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_chromosome
</td>
<td style="text-align:left;">
Lion chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_chrom_start
</td>
<td style="text-align:left;">
Lion chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_chrom_end
</td>
<td style="text-align:left;">
Lion chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Lion
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_orthology_type
</td>
<td style="text-align:left;">
Lion homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Lion gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Lion gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_goc_score
</td>
<td style="text-align:left;">
Lion Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_wga_coverage
</td>
<td style="text-align:left;">
Lion Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pleo_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Lion orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Long-tailed chinchilla gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Long-tailed chinchilla gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Long-tailed chinchilla protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_chromosome
</td>
<td style="text-align:left;">
Long-tailed chinchilla chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_chrom_start
</td>
<td style="text-align:left;">
Long-tailed chinchilla chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_chrom_end
</td>
<td style="text-align:left;">
Long-tailed chinchilla chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Long-tailed chinchilla
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_orthology_type
</td>
<td style="text-align:left;">
Long-tailed chinchilla homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Long-tailed chinchilla gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Long-tailed chinchilla gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_goc_score
</td>
<td style="text-align:left;">
Long-tailed chinchilla Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_wga_coverage
</td>
<td style="text-align:left;">
Long-tailed chinchilla Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clanigera_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Long-tailed chinchilla orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Lumpfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Lumpfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Lumpfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_chromosome
</td>
<td style="text-align:left;">
Lumpfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_chrom_start
</td>
<td style="text-align:left;">
Lumpfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_chrom_end
</td>
<td style="text-align:left;">
Lumpfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Lumpfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_orthology_type
</td>
<td style="text-align:left;">
Lumpfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Lumpfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Lumpfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_goc_score
</td>
<td style="text-align:left;">
Lumpfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Lumpfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
clumpus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Lumpfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Lyretail cichlid gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Lyretail cichlid gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Lyretail cichlid protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_chromosome
</td>
<td style="text-align:left;">
Lyretail cichlid chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_chrom_start
</td>
<td style="text-align:left;">
Lyretail cichlid chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_chrom_end
</td>
<td style="text-align:left;">
Lyretail cichlid chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Lyretail cichlid
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_orthology_type
</td>
<td style="text-align:left;">
Lyretail cichlid homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Lyretail cichlid gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Lyretail cichlid gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_goc_score
</td>
<td style="text-align:left;">
Lyretail cichlid Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_wga_coverage
</td>
<td style="text-align:left;">
Lyretail cichlid Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nbrichardi_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Lyretail cichlid orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Ma’s night monkey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Ma’s night monkey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Ma’s night monkey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_chromosome
</td>
<td style="text-align:left;">
Ma’s night monkey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_chrom_start
</td>
<td style="text-align:left;">
Ma’s night monkey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_chrom_end
</td>
<td style="text-align:left;">
Ma’s night monkey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Ma’s night monkey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_orthology_type
</td>
<td style="text-align:left;">
Ma’s night monkey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Ma’s night monkey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Ma’s night monkey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_goc_score
</td>
<td style="text-align:left;">
Ma’s night monkey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_wga_coverage
</td>
<td style="text-align:left;">
Ma’s night monkey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
anancymaae_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Ma’s night monkey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Macaque gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Macaque gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Macaque protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_chromosome
</td>
<td style="text-align:left;">
Macaque chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_chrom_start
</td>
<td style="text-align:left;">
Macaque chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_chrom_end
</td>
<td style="text-align:left;">
Macaque chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Macaque
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_orthology_type
</td>
<td style="text-align:left;">
Macaque homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Macaque gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Macaque gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_goc_score
</td>
<td style="text-align:left;">
Macaque Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_wga_coverage
</td>
<td style="text-align:left;">
Macaque Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmulatta_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Macaque orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Mainland tiger snake gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Mainland tiger snake gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Mainland tiger snake protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_chromosome
</td>
<td style="text-align:left;">
Mainland tiger snake chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Mainland tiger snake chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Mainland tiger snake chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Mainland tiger snake
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Mainland tiger snake homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Mainland tiger snake gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Mainland tiger snake gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_goc_score
</td>
<td style="text-align:left;">
Mainland tiger snake Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Mainland tiger snake Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nscutatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Mainland tiger snake orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Makobe Island cichlid gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Makobe Island cichlid gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Makobe Island cichlid protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_chromosome
</td>
<td style="text-align:left;">
Makobe Island cichlid chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_chrom_start
</td>
<td style="text-align:left;">
Makobe Island cichlid chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_chrom_end
</td>
<td style="text-align:left;">
Makobe Island cichlid chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Makobe Island cichlid
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_orthology_type
</td>
<td style="text-align:left;">
Makobe Island cichlid homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Makobe Island cichlid gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Makobe Island cichlid gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_goc_score
</td>
<td style="text-align:left;">
Makobe Island cichlid Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_wga_coverage
</td>
<td style="text-align:left;">
Makobe Island cichlid Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnyererei_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Makobe Island cichlid orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Mangrove rivulus gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Mangrove rivulus gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Mangrove rivulus protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_chromosome
</td>
<td style="text-align:left;">
Mangrove rivulus chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_chrom_start
</td>
<td style="text-align:left;">
Mangrove rivulus chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_chrom_end
</td>
<td style="text-align:left;">
Mangrove rivulus chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Mangrove rivulus
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_orthology_type
</td>
<td style="text-align:left;">
Mangrove rivulus homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Mangrove rivulus gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Mangrove rivulus gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_goc_score
</td>
<td style="text-align:left;">
Mangrove rivulus Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Mangrove rivulus Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
kmarmoratus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Mangrove rivulus orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Marmoset gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Marmoset gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Marmoset protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_chromosome
</td>
<td style="text-align:left;">
Marmoset chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_chrom_start
</td>
<td style="text-align:left;">
Marmoset chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_chrom_end
</td>
<td style="text-align:left;">
Marmoset chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Marmoset
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_orthology_type
</td>
<td style="text-align:left;">
Marmoset homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Marmoset gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Marmoset gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_goc_score
</td>
<td style="text-align:left;">
Marmoset Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Marmoset Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cjacchus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Marmoset orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Medium ground-finch gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Medium ground-finch gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Medium ground-finch protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_chromosome
</td>
<td style="text-align:left;">
Medium ground-finch chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_chrom_start
</td>
<td style="text-align:left;">
Medium ground-finch chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_chrom_end
</td>
<td style="text-align:left;">
Medium ground-finch chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Medium ground-finch
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_orthology_type
</td>
<td style="text-align:left;">
Medium ground-finch homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Medium ground-finch gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Medium ground-finch gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_goc_score
</td>
<td style="text-align:left;">
Medium ground-finch Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Medium ground-finch Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gfortis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Medium ground-finch orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Megabat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Megabat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Megabat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_chromosome
</td>
<td style="text-align:left;">
Megabat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_chrom_start
</td>
<td style="text-align:left;">
Megabat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_chrom_end
</td>
<td style="text-align:left;">
Megabat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Megabat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_orthology_type
</td>
<td style="text-align:left;">
Megabat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Megabat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Megabat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_goc_score
</td>
<td style="text-align:left;">
Megabat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Megabat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pvampyrus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Megabat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Mexican tetra gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Mexican tetra gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Mexican tetra protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_chromosome
</td>
<td style="text-align:left;">
Mexican tetra chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_chrom_start
</td>
<td style="text-align:left;">
Mexican tetra chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_chrom_end
</td>
<td style="text-align:left;">
Mexican tetra chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Mexican tetra
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_orthology_type
</td>
<td style="text-align:left;">
Mexican tetra homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Mexican tetra gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Mexican tetra gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_goc_score
</td>
<td style="text-align:left;">
Mexican tetra Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Mexican tetra Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
amexicanus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Mexican tetra orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Microbat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Microbat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Microbat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_chromosome
</td>
<td style="text-align:left;">
Microbat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_chrom_start
</td>
<td style="text-align:left;">
Microbat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_chrom_end
</td>
<td style="text-align:left;">
Microbat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Microbat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_orthology_type
</td>
<td style="text-align:left;">
Microbat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Microbat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Microbat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_goc_score
</td>
<td style="text-align:left;">
Microbat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Microbat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mlucifugus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Microbat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Midas cichlid gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Midas cichlid gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Midas cichlid protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_chromosome
</td>
<td style="text-align:left;">
Midas cichlid chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_chrom_start
</td>
<td style="text-align:left;">
Midas cichlid chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_chrom_end
</td>
<td style="text-align:left;">
Midas cichlid chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Midas cichlid
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_orthology_type
</td>
<td style="text-align:left;">
Midas cichlid homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Midas cichlid gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Midas cichlid gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_goc_score
</td>
<td style="text-align:left;">
Midas cichlid Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Midas cichlid Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
acitrinellus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Midas cichlid orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Mouse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Mouse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Mouse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_chromosome
</td>
<td style="text-align:left;">
Mouse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_chrom_start
</td>
<td style="text-align:left;">
Mouse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_chrom_end
</td>
<td style="text-align:left;">
Mouse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Mouse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_orthology_type
</td>
<td style="text-align:left;">
Mouse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Mouse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Mouse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_goc_score
</td>
<td style="text-align:left;">
Mouse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Mouse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmusculus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Mouse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Mouse Lemur gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Mouse Lemur gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Mouse Lemur protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_chromosome
</td>
<td style="text-align:left;">
Mouse Lemur chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_chrom_start
</td>
<td style="text-align:left;">
Mouse Lemur chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_chrom_end
</td>
<td style="text-align:left;">
Mouse Lemur chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Mouse Lemur
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_orthology_type
</td>
<td style="text-align:left;">
Mouse Lemur homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Mouse Lemur gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Mouse Lemur gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_goc_score
</td>
<td style="text-align:left;">
Mouse Lemur Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Mouse Lemur Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurinus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Mouse Lemur orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Mummichog gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Mummichog gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Mummichog protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_chromosome
</td>
<td style="text-align:left;">
Mummichog chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_chrom_start
</td>
<td style="text-align:left;">
Mummichog chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_chrom_end
</td>
<td style="text-align:left;">
Mummichog chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Mummichog
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_orthology_type
</td>
<td style="text-align:left;">
Mummichog homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Mummichog gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Mummichog gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_goc_score
</td>
<td style="text-align:left;">
Mummichog Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Mummichog Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
fheteroclitus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Mummichog orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Naked mole-rat female gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Naked mole-rat female gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Naked mole-rat female protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_chromosome
</td>
<td style="text-align:left;">
Naked mole-rat female chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_chrom_start
</td>
<td style="text-align:left;">
Naked mole-rat female chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_chrom_end
</td>
<td style="text-align:left;">
Naked mole-rat female chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Naked mole-rat female
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_orthology_type
</td>
<td style="text-align:left;">
Naked mole-rat female homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Naked mole-rat female gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Naked mole-rat female gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_goc_score
</td>
<td style="text-align:left;">
Naked mole-rat female Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_wga_coverage
</td>
<td style="text-align:left;">
Naked mole-rat female Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hgfemale_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Naked mole-rat female orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Narwhal gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Narwhal gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Narwhal protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_chromosome
</td>
<td style="text-align:left;">
Narwhal chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_chrom_start
</td>
<td style="text-align:left;">
Narwhal chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_chrom_end
</td>
<td style="text-align:left;">
Narwhal chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Narwhal
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_orthology_type
</td>
<td style="text-align:left;">
Narwhal homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Narwhal gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Narwhal gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_goc_score
</td>
<td style="text-align:left;">
Narwhal Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_wga_coverage
</td>
<td style="text-align:left;">
Narwhal Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmonoceros_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Narwhal orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Nile tilapia gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Nile tilapia gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Nile tilapia protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_chromosome
</td>
<td style="text-align:left;">
Nile tilapia chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_chrom_start
</td>
<td style="text-align:left;">
Nile tilapia chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_chrom_end
</td>
<td style="text-align:left;">
Nile tilapia chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Nile tilapia
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_orthology_type
</td>
<td style="text-align:left;">
Nile tilapia homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Nile tilapia gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Nile tilapia gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_goc_score
</td>
<td style="text-align:left;">
Nile tilapia Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Nile tilapia Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oniloticus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Nile tilapia orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Northern American deer mouse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Northern American deer mouse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Northern American deer mouse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_chromosome
</td>
<td style="text-align:left;">
Northern American deer mouse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_chrom_start
</td>
<td style="text-align:left;">
Northern American deer mouse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_chrom_end
</td>
<td style="text-align:left;">
Northern American deer mouse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Northern American deer mouse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_orthology_type
</td>
<td style="text-align:left;">
Northern American deer mouse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Northern American deer mouse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Northern American deer mouse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_goc_score
</td>
<td style="text-align:left;">
Northern American deer mouse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Northern American deer mouse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pmbairdii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Northern American deer mouse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Northern pike gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Northern pike gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Northern pike protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_chromosome
</td>
<td style="text-align:left;">
Northern pike chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_chrom_start
</td>
<td style="text-align:left;">
Northern pike chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_chrom_end
</td>
<td style="text-align:left;">
Northern pike chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Northern pike
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_orthology_type
</td>
<td style="text-align:left;">
Northern pike homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Northern pike gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Northern pike gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_goc_score
</td>
<td style="text-align:left;">
Northern pike Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_wga_coverage
</td>
<td style="text-align:left;">
Northern pike Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
elucius_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Northern pike orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Olive baboon gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Olive baboon gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Olive baboon protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_chromosome
</td>
<td style="text-align:left;">
Olive baboon chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_chrom_start
</td>
<td style="text-align:left;">
Olive baboon chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_chrom_end
</td>
<td style="text-align:left;">
Olive baboon chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Olive baboon
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_orthology_type
</td>
<td style="text-align:left;">
Olive baboon homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Olive baboon gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Olive baboon gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_goc_score
</td>
<td style="text-align:left;">
Olive baboon Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Olive baboon Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
panubis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Olive baboon orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Opossum gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Opossum gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Opossum protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_chromosome
</td>
<td style="text-align:left;">
Opossum chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_chrom_start
</td>
<td style="text-align:left;">
Opossum chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_chrom_end
</td>
<td style="text-align:left;">
Opossum chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Opossum
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_orthology_type
</td>
<td style="text-align:left;">
Opossum homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Opossum gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Opossum gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_goc_score
</td>
<td style="text-align:left;">
Opossum Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_wga_coverage
</td>
<td style="text-align:left;">
Opossum Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mdomestica_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Opossum orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Orange clownfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Orange clownfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Orange clownfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_chromosome
</td>
<td style="text-align:left;">
Orange clownfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_chrom_start
</td>
<td style="text-align:left;">
Orange clownfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_chrom_end
</td>
<td style="text-align:left;">
Orange clownfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Orange clownfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_orthology_type
</td>
<td style="text-align:left;">
Orange clownfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Orange clownfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Orange clownfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_goc_score
</td>
<td style="text-align:left;">
Orange clownfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_wga_coverage
</td>
<td style="text-align:left;">
Orange clownfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apercula_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Orange clownfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Orangutan gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Orangutan gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Orangutan protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_chromosome
</td>
<td style="text-align:left;">
Orangutan chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_chrom_start
</td>
<td style="text-align:left;">
Orangutan chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_chrom_end
</td>
<td style="text-align:left;">
Orangutan chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Orangutan
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_orthology_type
</td>
<td style="text-align:left;">
Orangutan homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Orangutan gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Orangutan gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_goc_score
</td>
<td style="text-align:left;">
Orangutan Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Orangutan Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pabelii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Orangutan orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Painted turtle gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Painted turtle gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Painted turtle protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_chromosome
</td>
<td style="text-align:left;">
Painted turtle chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_chrom_start
</td>
<td style="text-align:left;">
Painted turtle chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_chrom_end
</td>
<td style="text-align:left;">
Painted turtle chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Painted turtle
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_orthology_type
</td>
<td style="text-align:left;">
Painted turtle homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Painted turtle gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Painted turtle gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_goc_score
</td>
<td style="text-align:left;">
Painted turtle Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Painted turtle Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cpbellii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Painted turtle orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_chromosome
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_chrom_start
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_chrom_end
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Paramormyrops kingsleyae
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_orthology_type
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Paramormyrops kingsleyae gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Paramormyrops kingsleyae gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_goc_score
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_wga_coverage
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pkingsleyae_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Paramormyrops kingsleyae orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Pig gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Pig gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Pig protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_chromosome
</td>
<td style="text-align:left;">
Pig chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_chrom_start
</td>
<td style="text-align:left;">
Pig chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_chrom_end
</td>
<td style="text-align:left;">
Pig chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Pig
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_orthology_type
</td>
<td style="text-align:left;">
Pig homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Pig gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Pig gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_goc_score
</td>
<td style="text-align:left;">
Pig Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_wga_coverage
</td>
<td style="text-align:left;">
Pig Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sscrofa_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Pig orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Pig-tailed macaque gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Pig-tailed macaque gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Pig-tailed macaque protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_chromosome
</td>
<td style="text-align:left;">
Pig-tailed macaque chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_chrom_start
</td>
<td style="text-align:left;">
Pig-tailed macaque chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_chrom_end
</td>
<td style="text-align:left;">
Pig-tailed macaque chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Pig-tailed macaque
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_orthology_type
</td>
<td style="text-align:left;">
Pig-tailed macaque homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Pig-tailed macaque gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Pig-tailed macaque gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_goc_score
</td>
<td style="text-align:left;">
Pig-tailed macaque Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_wga_coverage
</td>
<td style="text-align:left;">
Pig-tailed macaque Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mnemestrina_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Pig-tailed macaque orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Pika gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Pika gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Pika protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_chromosome
</td>
<td style="text-align:left;">
Pika chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_chrom_start
</td>
<td style="text-align:left;">
Pika chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_chrom_end
</td>
<td style="text-align:left;">
Pika chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Pika
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_orthology_type
</td>
<td style="text-align:left;">
Pika homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Pika gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Pika gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_goc_score
</td>
<td style="text-align:left;">
Pika Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_wga_coverage
</td>
<td style="text-align:left;">
Pika Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oprinceps_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Pika orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Pike-perch gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Pike-perch gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Pike-perch protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_chromosome
</td>
<td style="text-align:left;">
Pike-perch chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_chrom_start
</td>
<td style="text-align:left;">
Pike-perch chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_chrom_end
</td>
<td style="text-align:left;">
Pike-perch chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Pike-perch
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_orthology_type
</td>
<td style="text-align:left;">
Pike-perch homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Pike-perch gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Pike-perch gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_goc_score
</td>
<td style="text-align:left;">
Pike-perch Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_wga_coverage
</td>
<td style="text-align:left;">
Pike-perch Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
slucioperca_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Pike-perch orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Pinecone soldierfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Pinecone soldierfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Pinecone soldierfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_chromosome
</td>
<td style="text-align:left;">
Pinecone soldierfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_chrom_start
</td>
<td style="text-align:left;">
Pinecone soldierfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_chrom_end
</td>
<td style="text-align:left;">
Pinecone soldierfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Pinecone soldierfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_orthology_type
</td>
<td style="text-align:left;">
Pinecone soldierfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Pinecone soldierfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Pinecone soldierfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_goc_score
</td>
<td style="text-align:left;">
Pinecone soldierfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_wga_coverage
</td>
<td style="text-align:left;">
Pinecone soldierfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmurdjan_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Pinecone soldierfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Pink-footed goose gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Pink-footed goose gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Pink-footed goose protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_chromosome
</td>
<td style="text-align:left;">
Pink-footed goose chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_chrom_start
</td>
<td style="text-align:left;">
Pink-footed goose chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_chrom_end
</td>
<td style="text-align:left;">
Pink-footed goose chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Pink-footed goose
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_orthology_type
</td>
<td style="text-align:left;">
Pink-footed goose homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Pink-footed goose gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Pink-footed goose gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_goc_score
</td>
<td style="text-align:left;">
Pink-footed goose Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Pink-footed goose Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
abrachyrhynchus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Pink-footed goose orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Platyfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Platyfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Platyfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_chromosome
</td>
<td style="text-align:left;">
Platyfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Platyfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Platyfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Platyfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Platyfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Platyfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Platyfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_goc_score
</td>
<td style="text-align:left;">
Platyfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Platyfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xmaculatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Platyfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Platypus gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Platypus gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Platypus protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_chromosome
</td>
<td style="text-align:left;">
Platypus chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_chrom_start
</td>
<td style="text-align:left;">
Platypus chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_chrom_end
</td>
<td style="text-align:left;">
Platypus chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Platypus
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_orthology_type
</td>
<td style="text-align:left;">
Platypus homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Platypus gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Platypus gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_goc_score
</td>
<td style="text-align:left;">
Platypus Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Platypus Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oanatinus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Platypus orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Polar bear gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Polar bear gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Polar bear protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_chromosome
</td>
<td style="text-align:left;">
Polar bear chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_chrom_start
</td>
<td style="text-align:left;">
Polar bear chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_chrom_end
</td>
<td style="text-align:left;">
Polar bear chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Polar bear
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_orthology_type
</td>
<td style="text-align:left;">
Polar bear homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Polar bear gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Polar bear gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_goc_score
</td>
<td style="text-align:left;">
Polar bear Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Polar bear Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
umaritimus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Polar bear orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Prairie vole gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Prairie vole gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Prairie vole protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_chromosome
</td>
<td style="text-align:left;">
Prairie vole chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_chrom_start
</td>
<td style="text-align:left;">
Prairie vole chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_chrom_end
</td>
<td style="text-align:left;">
Prairie vole chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Prairie vole
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_orthology_type
</td>
<td style="text-align:left;">
Prairie vole homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Prairie vole gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Prairie vole gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_goc_score
</td>
<td style="text-align:left;">
Prairie vole Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_wga_coverage
</td>
<td style="text-align:left;">
Prairie vole Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mochrogaster_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Prairie vole orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Rabbit gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Rabbit gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Rabbit protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_chromosome
</td>
<td style="text-align:left;">
Rabbit chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_chrom_start
</td>
<td style="text-align:left;">
Rabbit chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_chrom_end
</td>
<td style="text-align:left;">
Rabbit chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Rabbit
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_orthology_type
</td>
<td style="text-align:left;">
Rabbit homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Rabbit gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Rabbit gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_goc_score
</td>
<td style="text-align:left;">
Rabbit Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Rabbit Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ocuniculus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Rabbit orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Rainbow trout gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Rainbow trout gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Rainbow trout protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_chromosome
</td>
<td style="text-align:left;">
Rainbow trout chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_chrom_start
</td>
<td style="text-align:left;">
Rainbow trout chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_chrom_end
</td>
<td style="text-align:left;">
Rainbow trout chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Rainbow trout
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_orthology_type
</td>
<td style="text-align:left;">
Rainbow trout homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Rainbow trout gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Rainbow trout gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_goc_score
</td>
<td style="text-align:left;">
Rainbow trout Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_wga_coverage
</td>
<td style="text-align:left;">
Rainbow trout Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
omykiss_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Rainbow trout orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Rat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Rat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Rat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_chromosome
</td>
<td style="text-align:left;">
Rat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_chrom_start
</td>
<td style="text-align:left;">
Rat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_chrom_end
</td>
<td style="text-align:left;">
Rat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Rat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_orthology_type
</td>
<td style="text-align:left;">
Rat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Rat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Rat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_goc_score
</td>
<td style="text-align:left;">
Rat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Rat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
rnorvegicus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Rat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Red fox gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Red fox gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Red fox protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_chromosome
</td>
<td style="text-align:left;">
Red fox chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_chrom_start
</td>
<td style="text-align:left;">
Red fox chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_chrom_end
</td>
<td style="text-align:left;">
Red fox chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Red fox
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_orthology_type
</td>
<td style="text-align:left;">
Red fox homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Red fox gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Red fox gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_goc_score
</td>
<td style="text-align:left;">
Red fox Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_wga_coverage
</td>
<td style="text-align:left;">
Red fox Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
vvulpes_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Red fox orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Red-bellied piranha gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Red-bellied piranha gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Red-bellied piranha protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_chromosome
</td>
<td style="text-align:left;">
Red-bellied piranha chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_chrom_start
</td>
<td style="text-align:left;">
Red-bellied piranha chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_chrom_end
</td>
<td style="text-align:left;">
Red-bellied piranha chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Red-bellied piranha
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_orthology_type
</td>
<td style="text-align:left;">
Red-bellied piranha homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Red-bellied piranha gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Red-bellied piranha gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_goc_score
</td>
<td style="text-align:left;">
Red-bellied piranha Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_wga_coverage
</td>
<td style="text-align:left;">
Red-bellied piranha Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pnattereri_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Red-bellied piranha orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Reedfish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Reedfish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Reedfish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_chromosome
</td>
<td style="text-align:left;">
Reedfish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_chrom_start
</td>
<td style="text-align:left;">
Reedfish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_chrom_end
</td>
<td style="text-align:left;">
Reedfish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Reedfish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_orthology_type
</td>
<td style="text-align:left;">
Reedfish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Reedfish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Reedfish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_goc_score
</td>
<td style="text-align:left;">
Reedfish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Reedfish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ecalabaricus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Reedfish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Ryukyu mouse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Ryukyu mouse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Ryukyu mouse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_chromosome
</td>
<td style="text-align:left;">
Ryukyu mouse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_chrom_start
</td>
<td style="text-align:left;">
Ryukyu mouse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_chrom_end
</td>
<td style="text-align:left;">
Ryukyu mouse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Ryukyu mouse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_orthology_type
</td>
<td style="text-align:left;">
Ryukyu mouse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Ryukyu mouse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Ryukyu mouse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_goc_score
</td>
<td style="text-align:left;">
Ryukyu mouse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_wga_coverage
</td>
<td style="text-align:left;">
Ryukyu mouse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mcaroli_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Ryukyu mouse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_chromosome
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_chrom_start
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_chrom_end
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Saccharomyces cerevisiae
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_orthology_type
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Saccharomyces cerevisiae gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Saccharomyces cerevisiae gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
scerevisiae_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Saccharomyces cerevisiae orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Sailfin molly gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Sailfin molly gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Sailfin molly protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_chromosome
</td>
<td style="text-align:left;">
Sailfin molly chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_chrom_start
</td>
<td style="text-align:left;">
Sailfin molly chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_chrom_end
</td>
<td style="text-align:left;">
Sailfin molly chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Sailfin molly
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_orthology_type
</td>
<td style="text-align:left;">
Sailfin molly homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Sailfin molly gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Sailfin molly gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_goc_score
</td>
<td style="text-align:left;">
Sailfin molly Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_wga_coverage
</td>
<td style="text-align:left;">
Sailfin molly Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
platipinna_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Sailfin molly orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Sheep gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Sheep gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Sheep protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_chromosome
</td>
<td style="text-align:left;">
Sheep chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_chrom_start
</td>
<td style="text-align:left;">
Sheep chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_chrom_end
</td>
<td style="text-align:left;">
Sheep chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Sheep
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_orthology_type
</td>
<td style="text-align:left;">
Sheep homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Sheep gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Sheep gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_goc_score
</td>
<td style="text-align:left;">
Sheep Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_wga_coverage
</td>
<td style="text-align:left;">
Sheep Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
oarambouillet_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Sheep orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Sheepshead minnow gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Sheepshead minnow gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Sheepshead minnow protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_chromosome
</td>
<td style="text-align:left;">
Sheepshead minnow chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Sheepshead minnow chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Sheepshead minnow chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Sheepshead minnow
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Sheepshead minnow homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Sheepshead minnow gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Sheepshead minnow gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_goc_score
</td>
<td style="text-align:left;">
Sheepshead minnow Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Sheepshead minnow Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
cvariegatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Sheepshead minnow orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Shrew gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Shrew gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Shrew protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_chromosome
</td>
<td style="text-align:left;">
Shrew chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_chrom_start
</td>
<td style="text-align:left;">
Shrew chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_chrom_end
</td>
<td style="text-align:left;">
Shrew chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Shrew
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_orthology_type
</td>
<td style="text-align:left;">
Shrew homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Shrew gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Shrew gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_goc_score
</td>
<td style="text-align:left;">
Shrew Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Shrew Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
saraneus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Shrew orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Shrew mouse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Shrew mouse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Shrew mouse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_chromosome
</td>
<td style="text-align:left;">
Shrew mouse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_chrom_start
</td>
<td style="text-align:left;">
Shrew mouse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_chrom_end
</td>
<td style="text-align:left;">
Shrew mouse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Shrew mouse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_orthology_type
</td>
<td style="text-align:left;">
Shrew mouse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Shrew mouse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Shrew mouse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_goc_score
</td>
<td style="text-align:left;">
Shrew mouse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_wga_coverage
</td>
<td style="text-align:left;">
Shrew mouse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mpahari_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Shrew mouse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Siamese fighting fish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Siamese fighting fish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Siamese fighting fish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_chromosome
</td>
<td style="text-align:left;">
Siamese fighting fish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_chrom_start
</td>
<td style="text-align:left;">
Siamese fighting fish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_chrom_end
</td>
<td style="text-align:left;">
Siamese fighting fish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Siamese fighting fish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_orthology_type
</td>
<td style="text-align:left;">
Siamese fighting fish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Siamese fighting fish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Siamese fighting fish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_goc_score
</td>
<td style="text-align:left;">
Siamese fighting fish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_wga_coverage
</td>
<td style="text-align:left;">
Siamese fighting fish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bsplendens_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Siamese fighting fish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Siberian musk deer gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Siberian musk deer gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Siberian musk deer protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_chromosome
</td>
<td style="text-align:left;">
Siberian musk deer chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_chrom_start
</td>
<td style="text-align:left;">
Siberian musk deer chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_chrom_end
</td>
<td style="text-align:left;">
Siberian musk deer chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Siberian musk deer
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_orthology_type
</td>
<td style="text-align:left;">
Siberian musk deer homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Siberian musk deer gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Siberian musk deer gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_goc_score
</td>
<td style="text-align:left;">
Siberian musk deer Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Siberian musk deer Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mmoschiferus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Siberian musk deer orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Sloth gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Sloth gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Sloth protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_chromosome
</td>
<td style="text-align:left;">
Sloth chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_chrom_start
</td>
<td style="text-align:left;">
Sloth chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_chrom_end
</td>
<td style="text-align:left;">
Sloth chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Sloth
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_orthology_type
</td>
<td style="text-align:left;">
Sloth homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Sloth gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Sloth gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_goc_score
</td>
<td style="text-align:left;">
Sloth Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_wga_coverage
</td>
<td style="text-align:left;">
Sloth Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
choffmanni_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Sloth orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Sooty mangabey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Sooty mangabey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Sooty mangabey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_chromosome
</td>
<td style="text-align:left;">
Sooty mangabey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_chrom_start
</td>
<td style="text-align:left;">
Sooty mangabey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_chrom_end
</td>
<td style="text-align:left;">
Sooty mangabey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Sooty mangabey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_orthology_type
</td>
<td style="text-align:left;">
Sooty mangabey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Sooty mangabey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Sooty mangabey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_goc_score
</td>
<td style="text-align:left;">
Sooty mangabey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_wga_coverage
</td>
<td style="text-align:left;">
Sooty mangabey Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
catys_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Sooty mangabey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Sperm whale gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Sperm whale gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Sperm whale protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_chromosome
</td>
<td style="text-align:left;">
Sperm whale chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_chrom_start
</td>
<td style="text-align:left;">
Sperm whale chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_chrom_end
</td>
<td style="text-align:left;">
Sperm whale chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Sperm whale
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_orthology_type
</td>
<td style="text-align:left;">
Sperm whale homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Sperm whale gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Sperm whale gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_goc_score
</td>
<td style="text-align:left;">
Sperm whale Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_wga_coverage
</td>
<td style="text-align:left;">
Sperm whale Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
pcatodon_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Sperm whale orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Spiny chromis gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Spiny chromis gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Spiny chromis protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_chromosome
</td>
<td style="text-align:left;">
Spiny chromis chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_chrom_start
</td>
<td style="text-align:left;">
Spiny chromis chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_chrom_end
</td>
<td style="text-align:left;">
Spiny chromis chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Spiny chromis
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_orthology_type
</td>
<td style="text-align:left;">
Spiny chromis homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Spiny chromis gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Spiny chromis gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_goc_score
</td>
<td style="text-align:left;">
Spiny chromis Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Spiny chromis Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
apolyacanthus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Spiny chromis orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Spotted gar gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Spotted gar gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Spotted gar protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_chromosome
</td>
<td style="text-align:left;">
Spotted gar chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Spotted gar chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Spotted gar chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Spotted gar
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Spotted gar homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Spotted gar gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Spotted gar gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_goc_score
</td>
<td style="text-align:left;">
Spotted gar Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Spotted gar Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
loculatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Spotted gar orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Squirrel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Squirrel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Squirrel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_chromosome
</td>
<td style="text-align:left;">
Squirrel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Squirrel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Squirrel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Squirrel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Squirrel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Squirrel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Squirrel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_goc_score
</td>
<td style="text-align:left;">
Squirrel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Squirrel Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
itridecemlineatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Squirrel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Steppe mouse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Steppe mouse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Steppe mouse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_chromosome
</td>
<td style="text-align:left;">
Steppe mouse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_chrom_start
</td>
<td style="text-align:left;">
Steppe mouse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_chrom_end
</td>
<td style="text-align:left;">
Steppe mouse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Steppe mouse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_orthology_type
</td>
<td style="text-align:left;">
Steppe mouse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Steppe mouse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Steppe mouse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_goc_score
</td>
<td style="text-align:left;">
Steppe mouse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Steppe mouse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mspicilegus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Steppe mouse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Stickleback gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Stickleback gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Stickleback protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_chromosome
</td>
<td style="text-align:left;">
Stickleback chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Stickleback chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Stickleback chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Stickleback
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Stickleback homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Stickleback gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Stickleback gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_goc_score
</td>
<td style="text-align:left;">
Stickleback Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Stickleback Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
gaculeatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Stickleback orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tarsier gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tarsier gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tarsier protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_chromosome
</td>
<td style="text-align:left;">
Tarsier chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_chrom_start
</td>
<td style="text-align:left;">
Tarsier chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_chrom_end
</td>
<td style="text-align:left;">
Tarsier chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tarsier
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_orthology_type
</td>
<td style="text-align:left;">
Tarsier homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tarsier gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tarsier gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_goc_score
</td>
<td style="text-align:left;">
Tarsier Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tarsier Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csyrichta_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tarsier orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tasmanian devil gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tasmanian devil gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tasmanian devil protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_chromosome
</td>
<td style="text-align:left;">
Tasmanian devil chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_chrom_start
</td>
<td style="text-align:left;">
Tasmanian devil chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_chrom_end
</td>
<td style="text-align:left;">
Tasmanian devil chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tasmanian devil
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_orthology_type
</td>
<td style="text-align:left;">
Tasmanian devil homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tasmanian devil gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tasmanian devil gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_goc_score
</td>
<td style="text-align:left;">
Tasmanian devil Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sharrisii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tasmanian devil orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tetraodon gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tetraodon gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tetraodon protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_chromosome
</td>
<td style="text-align:left;">
Tetraodon chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_chrom_start
</td>
<td style="text-align:left;">
Tetraodon chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_chrom_end
</td>
<td style="text-align:left;">
Tetraodon chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tetraodon
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_orthology_type
</td>
<td style="text-align:left;">
Tetraodon homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tetraodon gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tetraodon gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_goc_score
</td>
<td style="text-align:left;">
Tetraodon Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tetraodon Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tnigroviridis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tetraodon orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Three-toed box turtle gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Three-toed box turtle gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Three-toed box turtle protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_chromosome
</td>
<td style="text-align:left;">
Three-toed box turtle chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_chrom_start
</td>
<td style="text-align:left;">
Three-toed box turtle chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_chrom_end
</td>
<td style="text-align:left;">
Three-toed box turtle chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Three-toed box turtle
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_orthology_type
</td>
<td style="text-align:left;">
Three-toed box turtle homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Three-toed box turtle gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Three-toed box turtle gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_goc_score
</td>
<td style="text-align:left;">
Three-toed box turtle Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Three-toed box turtle Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tctriunguis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Three-toed box turtle orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tiger gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tiger gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tiger protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_chromosome
</td>
<td style="text-align:left;">
Tiger chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_chrom_start
</td>
<td style="text-align:left;">
Tiger chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_chrom_end
</td>
<td style="text-align:left;">
Tiger chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tiger
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_orthology_type
</td>
<td style="text-align:left;">
Tiger homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tiger gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tiger gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_goc_score
</td>
<td style="text-align:left;">
Tiger Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tiger Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ptaltaica_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tiger orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tiger tail seahorse gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tiger tail seahorse gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tiger tail seahorse protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_chromosome
</td>
<td style="text-align:left;">
Tiger tail seahorse chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_chrom_start
</td>
<td style="text-align:left;">
Tiger tail seahorse chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_chrom_end
</td>
<td style="text-align:left;">
Tiger tail seahorse chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tiger tail seahorse
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_orthology_type
</td>
<td style="text-align:left;">
Tiger tail seahorse homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tiger tail seahorse gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tiger tail seahorse gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_goc_score
</td>
<td style="text-align:left;">
Tiger tail seahorse Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tiger tail seahorse Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hcomes_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tiger tail seahorse orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tongue sole gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tongue sole gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tongue sole protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_chromosome
</td>
<td style="text-align:left;">
Tongue sole chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_chrom_start
</td>
<td style="text-align:left;">
Tongue sole chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_chrom_end
</td>
<td style="text-align:left;">
Tongue sole chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tongue sole
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_orthology_type
</td>
<td style="text-align:left;">
Tongue sole homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tongue sole gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tongue sole gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_goc_score
</td>
<td style="text-align:left;">
Tongue sole Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tongue sole Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csemilaevis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tongue sole orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tree Shrew gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tree Shrew gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tree Shrew protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_chromosome
</td>
<td style="text-align:left;">
Tree Shrew chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_chrom_start
</td>
<td style="text-align:left;">
Tree Shrew chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_chrom_end
</td>
<td style="text-align:left;">
Tree Shrew chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tree Shrew
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_orthology_type
</td>
<td style="text-align:left;">
Tree Shrew homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tree Shrew gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tree Shrew gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_goc_score
</td>
<td style="text-align:left;">
Tree Shrew Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tree Shrew Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tbelangeri_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tree Shrew orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tropical clawed frog gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tropical clawed frog gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tropical clawed frog protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_chromosome
</td>
<td style="text-align:left;">
Tropical clawed frog chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_chrom_start
</td>
<td style="text-align:left;">
Tropical clawed frog chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_chrom_end
</td>
<td style="text-align:left;">
Tropical clawed frog chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tropical clawed frog
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_orthology_type
</td>
<td style="text-align:left;">
Tropical clawed frog homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tropical clawed frog gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tropical clawed frog gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_goc_score
</td>
<td style="text-align:left;">
Tropical clawed frog Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tropical clawed frog Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
xtropicalis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tropical clawed frog orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Tuatara gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Tuatara gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Tuatara protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_chromosome
</td>
<td style="text-align:left;">
Tuatara chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Tuatara chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Tuatara chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Tuatara
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Tuatara homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Tuatara gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Tuatara gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_goc_score
</td>
<td style="text-align:left;">
Tuatara Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Tuatara Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
spunctatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Tuatara orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Turbot gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Turbot gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Turbot protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_chromosome
</td>
<td style="text-align:left;">
Turbot chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_chrom_start
</td>
<td style="text-align:left;">
Turbot chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_chrom_end
</td>
<td style="text-align:left;">
Turbot chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Turbot
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_orthology_type
</td>
<td style="text-align:left;">
Turbot homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Turbot gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Turbot gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_goc_score
</td>
<td style="text-align:left;">
Turbot Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
smaximus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Turbot orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Turkey gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Turkey gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Turkey protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_chromosome
</td>
<td style="text-align:left;">
Turkey chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_chrom_start
</td>
<td style="text-align:left;">
Turkey chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_chrom_end
</td>
<td style="text-align:left;">
Turkey chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Turkey
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_orthology_type
</td>
<td style="text-align:left;">
Turkey homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Turkey gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Turkey gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_goc_score
</td>
<td style="text-align:left;">
Turkey Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mgallopavo_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Turkey orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Turquoise killifish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Turquoise killifish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Turquoise killifish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_chromosome
</td>
<td style="text-align:left;">
Turquoise killifish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_chrom_start
</td>
<td style="text-align:left;">
Turquoise killifish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_chrom_end
</td>
<td style="text-align:left;">
Turquoise killifish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Turquoise killifish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_orthology_type
</td>
<td style="text-align:left;">
Turquoise killifish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Turquoise killifish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Turquoise killifish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_goc_score
</td>
<td style="text-align:left;">
Turquoise killifish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_wga_coverage
</td>
<td style="text-align:left;">
Turquoise killifish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
nfurzeri_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Turquoise killifish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_chromosome
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_chrom_start
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_chrom_end
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Upper Galilee mountains blind mole rat
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_orthology_type
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Upper Galilee mountains blind mole rat gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Upper Galilee mountains blind mole rat gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_goc_score
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_wga_coverage
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ngalili_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Upper Galilee mountains blind mole rat orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Vaquita gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Vaquita gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Vaquita protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_chromosome
</td>
<td style="text-align:left;">
Vaquita chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_chrom_start
</td>
<td style="text-align:left;">
Vaquita chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_chrom_end
</td>
<td style="text-align:left;">
Vaquita chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Vaquita
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_orthology_type
</td>
<td style="text-align:left;">
Vaquita homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Vaquita gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Vaquita gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_goc_score
</td>
<td style="text-align:left;">
Vaquita Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Vaquita Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
psinus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Vaquita orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Vervet-AGM gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Vervet-AGM gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Vervet-AGM protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_chromosome
</td>
<td style="text-align:left;">
Vervet-AGM chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_chrom_start
</td>
<td style="text-align:left;">
Vervet-AGM chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_chrom_end
</td>
<td style="text-align:left;">
Vervet-AGM chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Vervet-AGM
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_orthology_type
</td>
<td style="text-align:left;">
Vervet-AGM homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Vervet-AGM gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Vervet-AGM gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_goc_score
</td>
<td style="text-align:left;">
Vervet-AGM Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Vervet-AGM Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
csabaeus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Vervet-AGM orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Wallaby gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Wallaby gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Wallaby protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_chromosome
</td>
<td style="text-align:left;">
Wallaby chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_chrom_start
</td>
<td style="text-align:left;">
Wallaby chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_chrom_end
</td>
<td style="text-align:left;">
Wallaby chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Wallaby
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_orthology_type
</td>
<td style="text-align:left;">
Wallaby homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Wallaby gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Wallaby gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_goc_score
</td>
<td style="text-align:left;">
Wallaby Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_wga_coverage
</td>
<td style="text-align:left;">
Wallaby Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
neugenii_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Wallaby orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Wild yak gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Wild yak gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Wild yak protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_chromosome
</td>
<td style="text-align:left;">
Wild yak chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_chrom_start
</td>
<td style="text-align:left;">
Wild yak chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_chrom_end
</td>
<td style="text-align:left;">
Wild yak chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Wild yak
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_orthology_type
</td>
<td style="text-align:left;">
Wild yak homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Wild yak gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Wild yak gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_goc_score
</td>
<td style="text-align:left;">
Wild yak Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_wga_coverage
</td>
<td style="text-align:left;">
Wild yak Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
bmutus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Wild yak orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Yarkand deer gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Yarkand deer gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Yarkand deer protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_chromosome
</td>
<td style="text-align:left;">
Yarkand deer chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_chrom_start
</td>
<td style="text-align:left;">
Yarkand deer chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_chrom_end
</td>
<td style="text-align:left;">
Yarkand deer chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Yarkand deer
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_orthology_type
</td>
<td style="text-align:left;">
Yarkand deer homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Yarkand deer gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Yarkand deer gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_goc_score
</td>
<td style="text-align:left;">
Yarkand deer Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Yarkand deer Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
chyarkandensis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Yarkand deer orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Yellowtail amberjack gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Yellowtail amberjack gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Yellowtail amberjack protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_chromosome
</td>
<td style="text-align:left;">
Yellowtail amberjack chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_chrom_start
</td>
<td style="text-align:left;">
Yellowtail amberjack chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_chrom_end
</td>
<td style="text-align:left;">
Yellowtail amberjack chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Yellowtail amberjack
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_orthology_type
</td>
<td style="text-align:left;">
Yellowtail amberjack homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Yellowtail amberjack gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Yellowtail amberjack gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_goc_score
</td>
<td style="text-align:left;">
Yellowtail amberjack Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_wga_coverage
</td>
<td style="text-align:left;">
Yellowtail amberjack Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
sldorsalis_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Yellowtail amberjack orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Zebra finch gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Zebra finch gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Zebra finch protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_chromosome
</td>
<td style="text-align:left;">
Zebra finch chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_chrom_start
</td>
<td style="text-align:left;">
Zebra finch chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_chrom_end
</td>
<td style="text-align:left;">
Zebra finch chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Zebra finch
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_orthology_type
</td>
<td style="text-align:left;">
Zebra finch homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Zebra finch gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Zebra finch gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_goc_score
</td>
<td style="text-align:left;">
Zebra finch Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_wga_coverage
</td>
<td style="text-align:left;">
Zebra finch Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
tguttata_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Zebra finch orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Zebra mbuna gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Zebra mbuna gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Zebra mbuna protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_chromosome
</td>
<td style="text-align:left;">
Zebra mbuna chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_chrom_start
</td>
<td style="text-align:left;">
Zebra mbuna chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_chrom_end
</td>
<td style="text-align:left;">
Zebra mbuna chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Zebra mbuna
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_orthology_type
</td>
<td style="text-align:left;">
Zebra mbuna homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Zebra mbuna gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Zebra mbuna gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_goc_score
</td>
<td style="text-align:left;">
Zebra mbuna Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_wga_coverage
</td>
<td style="text-align:left;">
Zebra mbuna Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
mzebra_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Zebra mbuna orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Zebrafish gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Zebrafish gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Zebrafish protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_chromosome
</td>
<td style="text-align:left;">
Zebrafish chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_chrom_start
</td>
<td style="text-align:left;">
Zebrafish chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_chrom_end
</td>
<td style="text-align:left;">
Zebrafish chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Zebrafish
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_orthology_type
</td>
<td style="text-align:left;">
Zebrafish homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Zebrafish gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Zebrafish gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_goc_score
</td>
<td style="text-align:left;">
Zebrafish Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_wga_coverage
</td>
<td style="text-align:left;">
Zebrafish Whole-genome alignment coverage
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
drerio_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Zebrafish orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_ensembl_gene
</td>
<td style="text-align:left;">
Zig-zag eel gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_associated_gene_name
</td>
<td style="text-align:left;">
Zig-zag eel gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_ensembl_peptide
</td>
<td style="text-align:left;">
Zig-zag eel protein or transcript stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_chromosome
</td>
<td style="text-align:left;">
Zig-zag eel chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_chrom_start
</td>
<td style="text-align:left;">
Zig-zag eel chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_chrom_end
</td>
<td style="text-align:left;">
Zig-zag eel chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_subtype
</td>
<td style="text-align:left;">
Last common ancestor with Zig-zag eel
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_orthology_type
</td>
<td style="text-align:left;">
Zig-zag eel homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_perc_id
</td>
<td style="text-align:left;">
%id. target Zig-zag eel gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_perc_id_r1
</td>
<td style="text-align:left;">
%id. query gene identical to target Zig-zag eel gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_goc_score
</td>
<td style="text-align:left;">
Zig-zag eel Gene-order conservation score
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
marmatus_homolog_orthology_confidence
</td>
<td style="text-align:left;">
Zig-zag eel orthology confidence [0 low, 1 high]
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_ensembl_gene
</td>
<td style="text-align:left;">
Human paralogue gene stable ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_associated_gene_name
</td>
<td style="text-align:left;">
Human paralogue associated gene name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_ensembl_peptide
</td>
<td style="text-align:left;">
Human paralogue protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_chromosome
</td>
<td style="text-align:left;">
Human paralogue chromosome/scaffold name
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_chrom_start
</td>
<td style="text-align:left;">
Human paralogue chromosome/scaffold start (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_chrom_end
</td>
<td style="text-align:left;">
Human paralogue chromosome/scaffold end (bp)
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_canonical_transcript_protein
</td>
<td style="text-align:left;">
Paralogue query protein or transcript ID
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_subtype
</td>
<td style="text-align:left;">
Paralogue last common ancestor with Human
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_orthology_type
</td>
<td style="text-align:left;">
Human paralogue homology type
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_perc_id
</td>
<td style="text-align:left;">
Paralogue %id. target Human gene identical to query gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
hsapiens_paralog_perc_id_r1
</td>
<td style="text-align:left;">
Paralogue %id. query gene identical to target Human gene
</td>
<td style="text-align:left;">
homologs
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID version
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
version
</td>
<td style="text-align:left;">
Version (gene)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID version
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_version
</td>
<td style="text-align:left;">
Version (transcript)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID version
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_version
</td>
<td style="text-align:left;">
Version (protein)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
start_position
</td>
<td style="text-align:left;">
Gene start (bp)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
end_position
</td>
<td style="text-align:left;">
Gene end (bp)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
band
</td>
<td style="text-align:left;">
Karyotype band
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene name
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_source
</td>
<td style="text-align:left;">
Source of gene name
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count
</td>
<td style="text-align:left;">
Transcript count
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
percentage_gene_gc_content
</td>
<td style="text-align:left;">
Gene % GC content
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
description
</td>
<td style="text-align:left;">
Gene description
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
variation_name
</td>
<td style="text-align:left;">
Variant name
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
germ_line_variation_source
</td>
<td style="text-align:left;">
Variant source
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
source_description
</td>
<td style="text-align:left;">
Variant source description
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
allele
</td>
<td style="text-align:left;">
Variant alleles
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
validated
</td>
<td style="text-align:left;">
Variant supporting evidence
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
mapweight
</td>
<td style="text-align:left;">
Mapweight
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
minor_allele
</td>
<td style="text-align:left;">
Minor allele
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
minor_allele_freq
</td>
<td style="text-align:left;">
Minor allele frequency
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
minor_allele_count
</td>
<td style="text-align:left;">
Minor allele count
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
clinical_significance
</td>
<td style="text-align:left;">
Clinical significance
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_location
</td>
<td style="text-align:left;">
Transcript location (bp)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
snp_chromosome_strand
</td>
<td style="text-align:left;">
Variant chromosome Strand
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_location
</td>
<td style="text-align:left;">
Protein location (aa)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_start
</td>
<td style="text-align:left;">
chromosome/scaffold position start (bp)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_end
</td>
<td style="text-align:left;">
Chromosome/scaffold position end (bp)
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
polyphen_prediction_2076
</td>
<td style="text-align:left;">
PolyPhen prediction
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
polyphen_score_2076
</td>
<td style="text-align:left;">
PolyPhen score
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
sift_prediction_2076
</td>
<td style="text-align:left;">
SIFT prediction
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
sift_score_2076
</td>
<td style="text-align:left;">
SIFT score
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
distance_to_transcript_2076
</td>
<td style="text-align:left;">
Distance to transcript
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_start_2076
</td>
<td style="text-align:left;">
CDS start
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_end_2076
</td>
<td style="text-align:left;">
CDS end
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_shift
</td>
<td style="text-align:left;">
Protein allele
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
synonymous_status
</td>
<td style="text-align:left;">
Variant consequence
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
allele_string_2076
</td>
<td style="text-align:left;">
Consequence specific allele
</td>
<td style="text-align:left;">
snp
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID version
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
version
</td>
<td style="text-align:left;">
Version (gene)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID version
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_version
</td>
<td style="text-align:left;">
Version (transcript)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID version
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_version
</td>
<td style="text-align:left;">
Version (protein)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
start_position
</td>
<td style="text-align:left;">
Gene start (bp)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
end_position
</td>
<td style="text-align:left;">
Gene end (bp)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
band
</td>
<td style="text-align:left;">
Karyotype band
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene name
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_source
</td>
<td style="text-align:left;">
Source of gene name
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_count
</td>
<td style="text-align:left;">
Transcript count
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
percentage_gene_gc_content
</td>
<td style="text-align:left;">
Gene % GC content
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
description
</td>
<td style="text-align:left;">
Gene description
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_variation_name
</td>
<td style="text-align:left;">
Variant name
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_source_name
</td>
<td style="text-align:left;">
Variant source
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_source_description
</td>
<td style="text-align:left;">
Variant source description
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_allele
</td>
<td style="text-align:left;">
Variant alleles
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_validated
</td>
<td style="text-align:left;">
Variant supporting evidence
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_mapweight
</td>
<td style="text-align:left;">
Mapweight
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_transcript_location
</td>
<td style="text-align:left;">
Transcript location (bp)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_snp_chromosome_strand
</td>
<td style="text-align:left;">
Variant chromosome/scaffold strand
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_peptide_location
</td>
<td style="text-align:left;">
Protein location (aa)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_chromosome_start
</td>
<td style="text-align:left;">
Chromosome/scaffold position start (bp)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_chromosome_end
</td>
<td style="text-align:left;">
Chromosome/scaffold position end (bp)
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
mart_transcript_variation_som__dm_distance_to_transcript_2076
</td>
<td style="text-align:left;">
Distance to transcript
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_cds_start_2076
</td>
<td style="text-align:left;">
CDS start
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_cds_end_2076
</td>
<td style="text-align:left;">
CDS end
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
somatic_synonymous_status
</td>
<td style="text-align:left;">
Variant consequence
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
mart_transcript_variation_som__dm_allele_string_2076
</td>
<td style="text-align:left;">
Consequence specific allele
</td>
<td style="text-align:left;">
snp_somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_exon_intron
</td>
<td style="text-align:left;">
Unspliced (Transcript)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
gene_exon_intron
</td>
<td style="text-align:left;">
Unspliced (Gene)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_flank
</td>
<td style="text-align:left;">
Flank (Transcript)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
gene_flank
</td>
<td style="text-align:left;">
Flank (Gene)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
coding_transcript_flank
</td>
<td style="text-align:left;">
Flank-coding region (Transcript)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
coding_gene_flank
</td>
<td style="text-align:left;">
Flank-coding region (Gene)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
5utr
</td>
<td style="text-align:left;">
5’ UTR
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
3utr
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
gene_exon
</td>
<td style="text-align:left;">
Exon sequences
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna
</td>
<td style="text-align:left;">
cDNA sequences
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
coding
</td>
<td style="text-align:left;">
Coding sequence
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide
</td>
<td style="text-align:left;">
Peptide
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
upstream_flank
</td>
<td style="text-align:left;">
upstream_flank
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
downstream_flank
</td>
<td style="text-align:left;">
downstream_flank
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id
</td>
<td style="text-align:left;">
Gene stable ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_gene_id_version
</td>
<td style="text-align:left;">
Gene stable ID version
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
description
</td>
<td style="text-align:left;">
Gene description
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_name
</td>
<td style="text-align:left;">
Gene name
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
external_gene_source
</td>
<td style="text-align:left;">
Source of gene name
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
chromosome_name
</td>
<td style="text-align:left;">
Chromosome/scaffold name
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
start_position
</td>
<td style="text-align:left;">
Gene start (bp)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
end_position
</td>
<td style="text-align:left;">
Gene end (bp)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
gene_biotype
</td>
<td style="text-align:left;">
Gene type
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
version
</td>
<td style="text-align:left;">
Version (gene)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
uniparc
</td>
<td style="text-align:left;">
UniParc ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprotswissprot
</td>
<td style="text-align:left;">
UniProtKB/Swiss-Prot ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
uniprotsptrembl
</td>
<td style="text-align:left;">
UniProtKB/TrEMBL ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna_coding_start
</td>
<td style="text-align:left;">
CDS start (within cDNA)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna_coding_end
</td>
<td style="text-align:left;">
CDS end (within cDNA)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
5_utr_start
</td>
<td style="text-align:left;">
5’ UTR start
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
5_utr_end
</td>
<td style="text-align:left;">
5’ UTR end
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
3_utr_start
</td>
<td style="text-align:left;">
3’ UTR start
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
3_utr_end
</td>
<td style="text-align:left;">
3’ UTR end
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id
</td>
<td style="text-align:left;">
Transcript stable ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_transcript_id_version
</td>
<td style="text-align:left;">
Transcript stable ID version
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id
</td>
<td style="text-align:left;">
Protein stable ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_peptide_id_version
</td>
<td style="text-align:left;">
Protein stable ID version
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_biotype
</td>
<td style="text-align:left;">
Transcript type
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_version
</td>
<td style="text-align:left;">
Version (transcript)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
peptide_version
</td>
<td style="text-align:left;">
Version (protein)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_start
</td>
<td style="text-align:left;">
Transcript start (bp)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_end
</td>
<td style="text-align:left;">
Transcript end (bp)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcription_start_site
</td>
<td style="text-align:left;">
Transcription start site (TSS)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
transcript_length
</td>
<td style="text-align:left;">
Transcript length (including UTRs and CDS)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_length
</td>
<td style="text-align:left;">
CDS Length
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_start
</td>
<td style="text-align:left;">
CDS start
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cds_end
</td>
<td style="text-align:left;">
CDS end
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
ensembl_exon_id
</td>
<td style="text-align:left;">
Exon stable ID
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
exon_chrom_start
</td>
<td style="text-align:left;">
Exon region start (bp)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
exon_chrom_end
</td>
<td style="text-align:left;">
Exon region end (bp)
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
strand
</td>
<td style="text-align:left;">
Strand
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
rank
</td>
<td style="text-align:left;">
Exon rank in transcript
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
phase
</td>
<td style="text-align:left;">
Start phase
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
end_phase
</td>
<td style="text-align:left;">
End phase
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna_coding_start
</td>
<td style="text-align:left;">
cDNA coding start
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
cdna_coding_end
</td>
<td style="text-align:left;">
cDNA coding end
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
genomic_coding_start
</td>
<td style="text-align:left;">
Genomic coding start
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
genomic_coding_end
</td>
<td style="text-align:left;">
Genomic coding end
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
<tr>
<td style="text-align:left;">
is_constitutive
</td>
<td style="text-align:left;">
Constitutive exon
</td>
<td style="text-align:left;">
sequences
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Since we want to create a mapping from Entrez gene IDs to HGNC symbols, we choose <code>entrezgene_id</code> and <code>hgnc_symbol</code> for the attributes of our biomart query. Knowing values, filters, and attributes, we are now able to build a biomaRt query and execute it using biomaRt’s primary query function:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_biomaRt</span> <span class='op'>&lt;-</span> <span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/getBM.html'>getBM</a></span><span class='op'>(</span>
    mart       <span class='op'>=</span> <span class='va'>ensembl</span>,
    attributes <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span><span class='op'>(</span><span class='st'>"entrezgene_id"</span>, <span class='st'>"hgnc_symbol"</span><span class='op'>)</span>,  <span class='co'>## Entrez ID to HGNC symbol</span>
    filters    <span class='op'>=</span> <span class='st'>"entrezgene_id"</span>,       <span class='co'>## filter by the Entrez IDs in the data</span>
    values     <span class='op'>=</span> <span class='va'>entrezID</span>
    <span class='op'>)</span>
</code></pre>
</div>
</div>
<p>Now we have a mapping from Entrez gene IDs to HGNC symbols for the genes in GSE152641:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_biomaRt</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>[1] &quot;entrezgene_id&quot; &quot;hgnc_symbol&quot;  </code></pre>
</div>
<h3 id="result">Result</h3>
<p>Assessing the quality of the mapping is always needed, as there could be genes that cannot be mapped by the database, resulting in missing values:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_biomaRt</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>[1] 14448</code></pre>
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>[1] 14426</code></pre>
</div>
<p>Or duplicates…<br />
</p>
<p>To check how many genes biomaRt failed to map:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'># Remove rows that have no HGNC symbols mapped to Entrez IDs</span>
<span class='va'>entrez2Hgnc_biomaRt</span> <span class='op'>&lt;-</span> <span class='va'>entrez2Hgnc_biomaRt</span><span class='op'>[</span>
    <span class='va'>entrez2Hgnc_biomaRt</span><span class='op'>$</span><span class='va'>hgnc_symbol</span> <span class='op'>!=</span> <span class='st'>""</span>, <span class='op'>]</span>

<span class='co'># Check how many Entrez IDs remain unmapped</span>
<span class='kw'>if</span> <span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_biomaRt</span><span class='op'>)</span> <span class='op'>!=</span> <span class='fu'><a href='https://rdrr.io/r/base/length.html'>length</a></span><span class='op'>(</span><span class='va'>entrezID</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>{</span>
    <span class='va'>miss</span> <span class='op'>&lt;-</span> <span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/length.html'>length</a></span><span class='op'>(</span><span class='va'>entrezID</span><span class='op'>)</span> <span class='op'>-</span> <span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_biomaRt</span><span class='op'>)</span><span class='op'>)</span>
    <span class='fu'><a href='https://rdrr.io/r/base/sprintf.html'>sprintf</a></span><span class='op'>(</span><span class='st'>"%i Entrez IDs not mapped to HGNC symbols."</span>, <span class='va'>miss</span><span class='op'>)</span>
<span class='op'>}</span>
</code></pre>
</div>
<pre><code>[1] &quot;83 Entrez IDs not mapped to HGNC symbols.&quot;</code></pre>
</div>
<p>How come that so many genes were not mapped? The reason is that biomaRt first maps Entrez IDs to Ensembl IDs, and then to HGNC symbols. However, there doesn’t really exist a simple 1-to-1 mapping from Entrez IDs to Ensembl IDs. Hence we are not getting a full translation of the Entrez IDs.<br />
</p>
<p>This means we would have to manually check the HGNC symbols for the 83 genes. This is <del>lame</del> time-consuming and inefficient. We turn to another database and see if we can get a better mapping.</p>
<h2 id="bridgedbr">BridgeDbR</h2>
<p><code>BridgeDbR</code><span class="citation" data-cites="BridgeDbR">(<a href="#ref-BridgeDbR" role="doc-biblioref">Christ Leemans &lt;Christleemans@Gmail.Com&gt;, Egon Willighagen&lt;Egon.Willighagen@Gmail.Com&gt;, Anwesha Bohler&lt;Anweshabohler@Gmail.Com&gt;, Lars Eijssen&lt;L.Eijssen@Maastrichtuniversity.Nl&gt; 2017</a>)</span> is an R interface to the BridgeDb framework, which provides a standard interface layer through which bioinformatics tools can be connected to different identifier mapping services such as Ensembl BioMart and PICR. The BridgeDb framework combines multiple mapping services to get broader coverage. <span class="citation" data-cites="BridgeDb">(<a href="#ref-BridgeDb" role="doc-biblioref">Van Iersel et al. 2010</a>)</span></p>
<p>This means we can expect a better mapping by BridgeDbR than biomaRt.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'>## install BridgeDbR from Bioconductor</span>
<span class='kw'>if</span> <span class='op'>(</span><span class='op'>!</span><span class='fu'><a href='https://rdrr.io/r/base/ns-load.html'>requireNamespace</a></span><span class='op'>(</span><span class='st'>"BridgeDbR"</span>, quietly <span class='op'>=</span> <span class='cn'>TRUE</span><span class='op'>)</span><span class='op'>)</span>
    <span class='fu'>BiocManager</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BiocManager/man/install.html'>install</a></span><span class='op'>(</span><span class='st'>"BridgeDbR"</span>, ask <span class='op'>=</span> <span class='cn'>FALSE</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<h3 id="database">Database</h3>
<p>BridgeDb provides its own ready-to-go mapping services in the form of local relational databases. These database files are created per species for genes and proteins. BridgeDb identifies organisms by their latin name and with a two character code. <span class="citation" data-cites="BridgeDb">(<a href="#ref-BridgeDb" role="doc-biblioref">Van Iersel et al. 2010</a>)</span></p>
<p>To get the name of the database file for human gene, we first need to figure out the organism code for homo sapiens:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'>## converts an organism latin name into a code.</span>
<span class='fu'>BridgeDbR</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BridgeDbR/man/getOrganismCode.html'>getOrganismCode</a></span><span class='op'>(</span>name <span class='op'>=</span> <span class='st'>"Homo sapiens"</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>[1] &quot;Hs&quot;</code></pre>
</div>
<p>The name of the Bridge database file:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='op'>(</span><span class='va'>dbname</span> <span class='op'>&lt;-</span> <span class='fu'>BridgeDbR</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BridgeDbR/man/getBridgeNames.html'>getBridgeNames</a></span><span class='op'>(</span>code <span class='op'>=</span> <span class='st'>"Hs"</span><span class='op'>)</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>[1] &quot;Hs_Derby_Ensembl_91.bridge&quot;</code></pre>
</div>
<p>Then to download the Bridge database file:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'>## file path to Bridge database</span>
<span class='va'>dbLocation</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/file.path.html'>file.path</a></span><span class='op'>(</span><span class='st'>"."</span>, <span class='st'>"data"</span>, <span class='va'>dbname</span><span class='op'>)</span>
<span class='co'>## download only if it hasn't been downloaded</span>
<span class='kw'>if</span> <span class='op'>(</span><span class='op'>!</span><span class='fu'><a href='https://rdrr.io/r/base/files.html'>file.exists</a></span><span class='op'>(</span><span class='va'>dbLocation</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>{</span>
    <span class='va'>dbLocation</span> <span class='op'>&lt;-</span> <span class='fu'>BridgeDbR</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BridgeDbR/man/getDatabase.html'>getDatabase</a></span><span class='op'>(</span>organism <span class='op'>=</span> <span class='st'>"Homo sapiens"</span>,
                                         location <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/file.path.html'>file.path</a></span><span class='op'>(</span><span class='st'>"."</span>, <span class='st'>"data"</span><span class='op'>)</span><span class='op'>)</span>
<span class='op'>}</span>
</code></pre>
</div>
</div>
<h3 id="mapping-identifiers">Mapping identifiers</h3>
<p>To load the downloaded Bridge database:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>mapper</span> <span class='op'>&lt;-</span> <span class='fu'>BridgeDbR</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BridgeDbR/man/loadDatabase.html'>loadDatabase</a></span><span class='op'>(</span><span class='va'>dbLocation</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>BridgeDbR’s mapping function requires the identifiers that are to be converted to be stored in a data frame with 2 specific columns named <code>source</code> and <code>identifier</code>, where <code>source</code> indicates the data source of gene identifiers it maps from. Each such data source is also identified by a human readable long name and by a short system code.<br />
</p>
<p><a href="https://bridgedb.github.io/pages/system-codes.html">Full list of BridgeDb system codes</a><br />
</p>
<p>For Entrez gene IDs, the system code is <code>L</code>.</p>
<p>To create a such data frame:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>input</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
    source    <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/rep.html'>rep</a></span><span class='op'>(</span><span class='st'>"L"</span>, <span class='fu'><a href='https://rdrr.io/r/base/length.html'>length</a></span><span class='op'>(</span><span class='va'>entrezID</span><span class='op'>)</span><span class='op'>)</span>,
    identifer <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/character.html'>as.character</a></span><span class='op'>(</span><span class='va'>entrezID</span><span class='op'>)</span>
<span class='op'>)</span>
</code></pre>
</div>
</div>
<p>The mapping function also requires an argument that specifies a data source for the target identifiers it maps to. For HGNC symbol it will be <code>H</code></p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_bridge</span> <span class='op'>&lt;-</span> <span class='fu'>BridgeDbR</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BridgeDbR/man/maps.html'>maps</a></span><span class='op'>(</span>
    mapper      <span class='op'>=</span> <span class='va'>mapper</span>,
    identifiers <span class='op'>=</span> <span class='va'>input</span>,
    target      <span class='op'>=</span> <span class='st'>"H"</span>
    <span class='op'>)</span>
</code></pre>
</div>
</div>
<p>And it throws an error:</p>
<pre>
Error in map(mapper, source = source, identifier = identifier, target = target) :
java.lang.NoSuchMethodError: &lt;init&gt; 
</pre>
<p>Oddly, this worked back in February. I tried to fix this problem myself but couldn’t even identify the cause of this. I’ve submitted this issue in <a href="https://github.com/bridgedb/BridgeDbR/issues/19">the BridgeDbR GitHub tracker</a>.</p>
<h3 id="result-1">Result</h3>
<p>I will update the result as soon as the issue above is resolved.</p>
<h2 id="orgdb">OrgDb</h2>
<p>OrgDb is a class of gene-centric <code>AnnotationDbi</code> packages at the organism level that contains annotation data for the genes of an organism. Specifically, the one we will use for mapping:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='kw'>if</span> <span class='op'>(</span><span class='op'>!</span><span class='fu'><a href='https://rdrr.io/r/base/ns-load.html'>requireNamespace</a></span><span class='op'>(</span><span class='st'>"org.Hs.eg.db"</span>, quietly <span class='op'>=</span> <span class='cn'>TRUE</span><span class='op'>)</span><span class='op'>)</span>
    <span class='fu'>BiocManager</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BiocManager/man/install.html'>install</a></span><span class='op'>(</span><span class='st'>"org.Hs.eg.db"</span>, ask <span class='op'>=</span> <span class='cn'>FALSE</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>This package contains genome wide annotation for Human, primarily based on mapping using Entrez Gene identifiers.<span class="citation" data-cites="org">(<a href="#ref-org" role="doc-biblioref">Carlson et al. 2019</a>)</span><br />
</p>
<p>Many Bioconductor packages for annotation support OrgDb, and <code>AnnotationDbi</code><span class="citation" data-cites="annotationdbi">(<a href="#ref-annotationdbi" role="doc-biblioref">Hervé Pagès 2017</a>)</span> is most commonly used:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'># Use org.Hs.eg.db database to map identifiers</span>
<span class='kw'>if</span> <span class='op'>(</span><span class='op'>!</span><span class='fu'><a href='https://rdrr.io/r/base/ns-load.html'>requireNamespace</a></span><span class='op'>(</span><span class='st'>"AnnotationDbi"</span>, quietly <span class='op'>=</span> <span class='cn'>TRUE</span><span class='op'>)</span><span class='op'>)</span>
    <span class='fu'>BiocManager</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BiocManager/man/install.html'>install</a></span><span class='op'>(</span><span class='st'>"AnnotationDbi"</span>, ask <span class='op'>=</span> <span class='cn'>FALSE</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<h3 id="org.hs.eg.db">org.Hs.eg.db</h3>
<p>Unlike other packages, which we can use their functions without loading them, annotation packages need to be loaded first:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'><a href='https://rdrr.io/r/base/message.html'>suppressPackageStartupMessages</a></span><span class='op'>(</span><span class='op'>{</span>
    <span class='kw'><a href='https://rdrr.io/r/base/library.html'>library</a></span><span class='op'>(</span><span class='va'>org.Hs.eg.db</span><span class='op'>)</span>
<span class='op'>}</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>Types of annotation data retriveable from <code>org.Hs.eg.db</code>:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>AnnotationDbi</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/AnnotationDbi/man/AnnotationDb-class.html'>columns</a></span><span class='op'>(</span><span class='va'>org.Hs.eg.db</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code> [1] &quot;ACCNUM&quot;       &quot;ALIAS&quot;        &quot;ENSEMBL&quot;      &quot;ENSEMBLPROT&quot; 
 [5] &quot;ENSEMBLTRANS&quot; &quot;ENTREZID&quot;     &quot;ENZYME&quot;       &quot;EVIDENCE&quot;    
 [9] &quot;EVIDENCEALL&quot;  &quot;GENENAME&quot;     &quot;GENETYPE&quot;     &quot;GO&quot;          
[13] &quot;GOALL&quot;        &quot;IPI&quot;          &quot;MAP&quot;          &quot;OMIM&quot;        
[17] &quot;ONTOLOGY&quot;     &quot;ONTOLOGYALL&quot;  &quot;PATH&quot;         &quot;PFAM&quot;        
[21] &quot;PMID&quot;         &quot;PROSITE&quot;      &quot;REFSEQ&quot;       &quot;SYMBOL&quot;      
[25] &quot;UCSCKG&quot;       &quot;UNIPROT&quot;     </code></pre>
</div>
<p>Or <strong>fields</strong> that can be used as keys to query the database:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>AnnotationDbi</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/AnnotationDbi/man/AnnotationDb-class.html'>keytypes</a></span><span class='op'>(</span><span class='va'>org.Hs.eg.db</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code> [1] &quot;ACCNUM&quot;       &quot;ALIAS&quot;        &quot;ENSEMBL&quot;      &quot;ENSEMBLPROT&quot; 
 [5] &quot;ENSEMBLTRANS&quot; &quot;ENTREZID&quot;     &quot;ENZYME&quot;       &quot;EVIDENCE&quot;    
 [9] &quot;EVIDENCEALL&quot;  &quot;GENENAME&quot;     &quot;GENETYPE&quot;     &quot;GO&quot;          
[13] &quot;GOALL&quot;        &quot;IPI&quot;          &quot;MAP&quot;          &quot;OMIM&quot;        
[17] &quot;ONTOLOGY&quot;     &quot;ONTOLOGYALL&quot;  &quot;PATH&quot;         &quot;PFAM&quot;        
[21] &quot;PMID&quot;         &quot;PROSITE&quot;      &quot;REFSEQ&quot;       &quot;SYMBOL&quot;      
[25] &quot;UCSCKG&quot;       &quot;UNIPROT&quot;     </code></pre>
</div>
<p>It’s not always the case that an annotation package has the same keytypes as its types of annotations; some annotation data can’t be used as keys for querying.<br />
</p>
<p>The fields we’re interested in are <code>ENTREZID</code> and <code>SYMBOL</code>.</p>
<h3 id="mapping">Mapping</h3>
<p>Querying an OrgDb annotation package with <code>AnnotationDbi</code> to create a mapping is much simpler and straightforward:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>cols</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span><span class='op'>(</span>
    <span class='st'>"ENTREZID"</span>, <span class='co'>## from</span>
    <span class='st'>"SYMBOL"</span>    <span class='co'>## to</span>
    <span class='op'>)</span>           <span class='co'>## types of data we want in the query result</span>
<span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='fu'>AnnotationDbi</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/AnnotationDbi/man/AnnotationDb-class.html'>select</a></span><span class='op'>(</span><span class='va'>org.Hs.eg.db</span>,
                                           keys    <span class='op'>=</span> <span class='va'>entrezID</span>,
                                           keytype <span class='op'>=</span> <span class='st'>"ENTREZID"</span>,
                                           columns <span class='op'>=</span> <span class='va'>cols</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>&#39;select()&#39; returned 1:1 mapping between keys and columns</code></pre>
</div>
<h3 id="result-2">Result</h3>
<p>First we check whether there are missing values in the mapping:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'># Remove rows that have no HGNC symbols mapped to Entrez IDs</span>
<span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>SYMBOL</span> <span class='op'>!=</span> <span class='st'>""</span>, <span class='op'>]</span>
<span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='op'>!</span><span class='fu'><a href='https://rdrr.io/r/base/NA.html'>is.na</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>SYMBOL</span><span class='op'>)</span>, <span class='op'>]</span>

<span class='co'># Check how many Entrez IDs remain unmapped</span>
<span class='kw'>if</span> <span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>)</span> <span class='op'>!=</span> <span class='fu'><a href='https://rdrr.io/r/base/length.html'>length</a></span><span class='op'>(</span><span class='va'>entrezID</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>{</span>
    <span class='va'>miss</span> <span class='op'>&lt;-</span> <span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/length.html'>length</a></span><span class='op'>(</span><span class='va'>entrezID</span><span class='op'>)</span> <span class='op'>-</span> <span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>)</span><span class='op'>)</span>
    <span class='fu'><a href='https://rdrr.io/r/base/sprintf.html'>sprintf</a></span><span class='op'>(</span><span class='st'>"%i Entrez IDs not mapped to HGNC symbols."</span>, <span class='va'>miss</span><span class='op'>)</span>
<span class='op'>}</span> <span class='kw'>else</span> <span class='op'>{</span>
    <span class='fu'><a href='https://rdrr.io/r/base/sprintf.html'>sprintf</a></span><span class='op'>(</span><span class='st'>"No missing values."</span><span class='op'>)</span>
<span class='op'>}</span>
</code></pre>
</div>
<pre><code>[1] &quot;13 Entrez IDs not mapped to HGNC symbols.&quot;</code></pre>
</div>
<p>Then we look for duplicates in the mapping, since if there are duplicates, then there will be Entrez gene IDs mepped more than once:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>hgnc_occur</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/as.data.frame.html'>as.data.frame</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/table.html'>table</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>SYMBOL</span><span class='op'>)</span><span class='op'>)</span>
<span class='fu'><a href='https://rdrr.io/r/base/nrow.html'>nrow</a></span><span class='op'>(</span><span class='va'>hgnc_occur</span><span class='op'>[</span><span class='va'>hgnc_occur</span><span class='op'>$</span><span class='va'>Freq</span> <span class='op'>&gt;</span> <span class='fl'>1</span>, <span class='op'>]</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>[1] 0</code></pre>
</div>
<p>So there are no gene IDs repetitively mapped. This means we have only 13 genes to check. This is much more manageable than 83 genes.</p>
<h2 id="manual-clean-up">Manual Clean-up</h2>
<p>Since there were only two mappings available by BioMart and OrgDb at the time of writing, we will use the mapping by OrgDb as it has the least missing values and manually check those missing values by hands. After all, some due diligence is required:)</p>
<h3 id="enterz-gene-ids-missing-hgnc-symbols">Enterz gene IDs missing HGNC symbols</h3>
<p>The Entrez gene IDs that couldn’t be mapped to HGNC symbols by OrgDb are the following:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrezID</span><span class='op'>[</span><span class='op'>!</span><span class='op'>(</span><span class='va'>entrezID</span> <span class='op'>%in%</span> <span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span><span class='op'>)</span><span class='op'>]</span>
</code></pre>
</div>
<pre><code> [1] &quot;23285&quot;     &quot;84953&quot;     &quot;285464&quot;    &quot;338809&quot;    &quot;388289&quot;   
 [6] &quot;723788&quot;    &quot;102724957&quot; &quot;102724985&quot; &quot;102724993&quot; &quot;107983991&quot;
[11] &quot;107984125&quot; &quot;107984138&quot; &quot;107986084&quot;</code></pre>
</div>
<p>One might retrieve their information by using the Entrez Programming Utilities, but now the fastest way to know what happened to these genes is directly look them up in <a href="https://www.ncbi.nlm.nih.gov/gene/">NCBI Gene</a>.</p>
<table>
<colgroup>
<col style="width: 34%" />
<col style="width: 7%" />
<col style="width: 57%" />
</colgroup>
<thead>
<tr class="header">
<th>Entrez Gene ID</th>
<th>HGNC Symbol</th>
<th>Current Status</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/23285">23285</a></td>
<td>KIAA1107</td>
<td>replaced with GeneID: <a href="https://www.ncbi.nlm.nih.gov/gene/284697">284697</a>. HGNC symbol: BTBD8</td>
</tr>
<tr class="even">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84953">84953</a></td>
<td>MICALCL</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/9645">9645</a>. HGNC symbol: MICAL2</td>
</tr>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=285464">285464</a></td>
<td>CRIPAK</td>
<td>has been withdrawn by NCBI. It was decided that this locus is not an independent gene.</td>
</tr>
<tr class="even">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=338809">338809</a></td>
<td>C12orf74</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/440107">440107</a>. HGNC symbol: PLEKHG7</td>
</tr>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=388289">388289</a></td>
<td>C16orf47</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/463">463</a>. HGNC symbol: ZFHX3</td>
</tr>
<tr class="even">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=723788">723788</a></td>
<td>MIG7</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/8412">8412</a>. HGNC symbol: BCAR3</td>
</tr>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=102724957">102724957</a></td>
<td>LOC102724957</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/105376569">105376569</a>. HGNC symbol: LINC02751</td>
</tr>
<tr class="even">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=102724985">102724985</a></td>
<td>LOC102724985</td>
<td>has been withdrawn by NCBI staff. This record represented a gene that is not currently annotated by NCBI.</td>
</tr>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=102724993">102724993</a></td>
<td>LOC102724993</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/642799">642799</a>. HGNC symbol: NPIPA2</td>
</tr>
<tr class="even">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=107983991">107983991</a></td>
<td>LOC107983991</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/105372267">105372267</a>. HGNC symbol: NFILZ</td>
</tr>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=107984125">107984125</a></td>
<td>LOC107984125</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/6101">6101</a>. HGNC symbol: RP1</td>
</tr>
<tr class="even">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=107984138">107984138</a></td>
<td>LOC107984138</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/100506060">100506060</a>. HGNC symbol: SMG1P7</td>
</tr>
<tr class="odd">
<td><a href="https://www.ncbi.nlm.nih.gov/gene/?term=107986084">107986084</a></td>
<td>LOC107986084</td>
<td>replaced with Gene ID: <a href="https://www.ncbi.nlm.nih.gov/gene/7375">7375</a>. HGNC symbol: USP4</td>
</tr>
</tbody>
</table>
<p>Then we check whether the Entrez gene IDs missing HGNC symbols and the IDs they were replaced with have the same counts for every biological replicate. If they have identical gene counts, then we treat the unmapped gene ID as a duplicate and remove it.</p>
<h4 id="section">23285</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"23285"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"284697"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"284697"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>      ENTREZID SYMBOL
13318   284697  BTBD8</code></pre>
</div>
<h4 id="section-1">84953</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"84953"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"9645"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"9645"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>     ENTREZID SYMBOL
4930     9645 MICAL2</code></pre>
</div>
<h4 id="section-2">338809</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"338809"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"440107"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"440107"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>      ENTREZID  SYMBOL
13889   440107 PLEKHG7</code></pre>
</div>
<h4 id="section-3">388289</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"388289"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"463"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"463"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>    ENTREZID SYMBOL
230      463  ZFHX3</code></pre>
</div>
<h4 id="section-4">723788</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"723788"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"8412"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"8412"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>     ENTREZID SYMBOL
4170     8412  BCAR3</code></pre>
</div>
<h4 id="section-5">102724957</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"102724957"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"105376569"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
FALSE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Their expression values are no identical. This means we still need to treat it as an independent gene.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/cbind.html'>rbind</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span>,
                           <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
                               ENTREZID <span class='op'>=</span> <span class='st'>"102724957"</span>,
                               SYMBOL   <span class='op'>=</span> <span class='st'>"LOC102724957"</span>
                               <span class='op'>)</span>
                           <span class='op'>)</span>
</code></pre>
</div>
</div>
<h4 id="section-6">102724993</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"102724993"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"642799"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00001
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00002
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00003
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00004
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00005
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00006
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00007
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00008
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00009
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00010
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00011
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00012
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00013
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00014
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00015
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00016
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00017
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00018
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00019
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00020
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00021
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00022
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00023
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00024
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00025
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00026
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00027
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00028
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00029
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00030
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00031
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00032
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00033
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00034
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00035
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00036
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00037
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00038
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00039
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00040
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00041
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00042
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00043
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00044
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00045
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00046
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00047
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00048
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00049
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00050
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00051
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00052
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00053
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00054
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00055
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00056
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00057
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00058
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00059
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00060
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00061
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00062
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00063
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00064
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00065
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00066
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00067
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00068
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00069
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00070
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00071
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00072
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00073
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00074
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00075
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00076
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00077
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00078
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00079
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00080
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00081
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00082
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00083
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00084
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00085
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00086
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<p>No results returned. This means the replacing Entrez ID 642799 is not present in the dataset. In this case we map it to the HGNC symbol of the replacing gene ID:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/cbind.html'>rbind</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span>,
                           <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
                               ENTREZID <span class='op'>=</span> <span class='st'>"102724993"</span>,
                               SYMBOL   <span class='op'>=</span> <span class='st'>"NPIPA2"</span>
                               <span class='op'>)</span>
                           <span class='op'>)</span>
</code></pre>
</div>
</div>
<h4 id="section-7">107983991</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"107983991"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"105372267"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"105372267"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>       ENTREZID SYMBOL
14350 105372267  NFILZ</code></pre>
</div>
<h4 id="section-8">107984125</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"107984125"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"6101"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"6101"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>     ENTREZID SYMBOL
3046     6101    RP1</code></pre>
</div>
<h4 id="section-9">107984138</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"107984138"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"100506060"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00001
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00002
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00003
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00004
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00005
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00006
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00007
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00008
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00009
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00010
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00011
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00012
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00013
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00014
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00015
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00016
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00017
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00018
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00019
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00020
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00021
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00022
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00023
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00024
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00025
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00026
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00027
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00028
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00029
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00030
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00031
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00032
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00033
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00034
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00035
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00036
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00037
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00038
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00039
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00040
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00041
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00042
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00043
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00044
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00045
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00046
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00047
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00048
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00049
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00050
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00051
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00052
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00053
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00054
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00055
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00056
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00057
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00058
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00059
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00060
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00061
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00062
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00063
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00064
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00065
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00066
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00067
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00068
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00069
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00070
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00071
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00072
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00073
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00074
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00075
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00076
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00077
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00078
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00079
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00080
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00081
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00082
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00083
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00084
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00085
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
IMX_sample00086
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<p>This one also returns no values. We treat it the same as the way we did to 102724993:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/cbind.html'>rbind</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span>,
                           <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
                               ENTREZID <span class='op'>=</span> <span class='st'>"107984138"</span>,
                               SYMBOL   <span class='op'>=</span> <span class='st'>"SMG1P7"</span>
                               <span class='op'>)</span>
                           <span class='op'>)</span>
</code></pre>
</div>
</div>
<h4 id="section-10">107986084</h4>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"107986084"</span>,<span class='op'>]</span> <span class='op'>==</span>
    <span class='va'>normalised_counts</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>==</span> <span class='st'>"7375"</span>,<span class='op'>]</span>
</code></pre>
</div>
</div>
<div class="layout-chunk" data-layout="l-body">
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:500px; ">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
x
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IMX_sample00001
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00002
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00003
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00004
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00005
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00006
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00007
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00008
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00009
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00010
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00011
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00012
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00013
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00014
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00015
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00016
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00017
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00018
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00019
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00020
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00021
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00022
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00023
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00024
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00025
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00026
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00027
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00028
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00029
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00030
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00031
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00032
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00033
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00034
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00035
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00036
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00037
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00038
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00039
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00040
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00041
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00042
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00043
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00044
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00045
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00046
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00047
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00048
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00049
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00050
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00051
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00052
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00053
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00054
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00055
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00056
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00057
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00058
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00059
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00060
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00061
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00062
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00063
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00064
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00065
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00066
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00067
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00068
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00069
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00070
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00071
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00072
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00073
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00074
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00075
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00076
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00077
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00078
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00079
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00080
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00081
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00082
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00083
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00084
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00085
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IMX_sample00086
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check whether the mapped HGNC symbol of the replacing Entrez gene ID matches the NCBI gene record:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>[</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span> <span class='op'>==</span> <span class='st'>"7375"</span>,<span class='op'>]</span>
</code></pre>
</div>
<pre><code>     ENTREZID SYMBOL
3765     7375   USP4</code></pre>
</div>
<p>And we manually map the gene IDs withdrawn by NCBI:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>withdrawnID</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
    ENTREZID <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span><span class='op'>(</span><span class='st'>"285464"</span>, <span class='st'>"102724985"</span><span class='op'>)</span>,
    SYMBOL   <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span><span class='op'>(</span><span class='st'>"CRIPAK"</span>, <span class='st'>"LOC102724985"</span><span class='op'>)</span>
<span class='op'>)</span>
<span class='va'>entrez2Hgnc_OrgDb</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/cbind.html'>rbind</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span>, <span class='va'>withdrawnID</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>For now, we disregard the fact that these gene IDs were withdrawn by NCBI staff as they were still used by the original experiment.</p>
<h3 id="replcaing-entrez-id-in-the-count-matrix">Replcaing Entrez ID in the Count Matrix</h3>
<p>Create a named vector from the data frame with the mapping.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='va'>entrez2Hgnc</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/stats/setNames.html'>setNames</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/character.html'>as.character</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>SYMBOL</span><span class='op'>)</span>,
                        nm <span class='op'>=</span> <span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>We will use it like a dictionary object in Python for mapping:<br />
</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'>## Select genes that have their Entrez IDs mapped to HGNC symbols</span>
<span class='va'>select</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/which.html'>which</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>%in%</span> <span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span><span class='op'>(</span><span class='va'>entrez2Hgnc</span><span class='op'>)</span><span class='op'>)</span>
<span class='co'>## ID conversion</span>
<span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span><span class='op'>[</span><span class='va'>select</span><span class='op'>]</span> <span class='op'>&lt;-</span> <span class='va'>entrez2Hgnc</span><span class='op'>[</span>
    <span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span><span class='op'>[</span><span class='va'>select</span><span class='op'>]</span>
    <span class='op'>]</span>
</code></pre>
</div>
</div>
<p>Finally, take out the genes considered to be duplicates from the count matrix:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='co'>## Since we didn't put them in the mapping</span>
<span class='va'>dup_gene</span>          <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/sets.html'>setdiff</a></span><span class='op'>(</span><span class='va'>entrezID</span>, <span class='va'>entrez2Hgnc_OrgDb</span><span class='op'>$</span><span class='va'>ENTREZID</span><span class='op'>)</span>
<span class='co'>## Select rows to delete</span>
<span class='va'>select</span>            <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/which.html'>which</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>rownames</a></span><span class='op'>(</span><span class='va'>normalised_counts</span><span class='op'>)</span> <span class='op'>%in%</span> <span class='va'>dup_gene</span><span class='op'>)</span>
<span class='va'>normalised_counts</span> <span class='op'>&lt;-</span> <span class='va'>normalised_counts</span><span class='op'>[</span><span class='op'>-</span><span class='va'>select</span>, <span class='op'>]</span>
</code></pre>
</div>
</div>
<p>We have mapped every Entrez gene IDs in the count matrix to their HGNC symbols.</p>
<p>Save the normalised count matrix with the gene names mapped for later analyses:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'><a href='https://rdrr.io/r/base/readRDS.html'>saveRDS</a></span><span class='op'>(</span>object <span class='op'>=</span> <span class='va'>normalised_counts</span>,
        file   <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/file.path.html'>file.path</a></span><span class='op'>(</span><span class='st'>"."</span>, <span class='st'>"data"</span>, <span class='st'>"normalised_counts.rds"</span><span class='op'>)</span><span class='op'>)</span>
</code></pre>
</div>
</div>
<p>This concludes the initial processing of the bulk RNA-seq count data of GSE152641:</p>
<h2 id="conclusion">Conclusion</h2>
<p>When mapping gene identifiers, it is always important to assess the quality of mappings produced by different databases. As shown above, in the case of mapping from Entrez gene IDs to HGNC symbols, OrgDb might be preferable to biomaRt. Here the quality of these mappings were only assessed based on the number of gene identifiers mapped. One thing we haven’t checked in the mappings is whether there are genes mismatched to wrong HGNC symbols. Yes, this could happen. For example, gene <code>100913187</code> in this dataset is being incorrectly mapped by biomaRt:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>biomaRt</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/biomaRt/man/getBM.html'>getBM</a></span><span class='op'>(</span>
    mart       <span class='op'>=</span> <span class='va'>ensembl</span>,
    attributes <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span><span class='op'>(</span><span class='st'>"entrezgene_id"</span>, <span class='st'>"hgnc_symbol"</span><span class='op'>)</span>,  <span class='co'>## Entrez ID to HGNC symbol</span>
    filters    <span class='op'>=</span> <span class='st'>"entrezgene_id"</span>,       <span class='co'>## filter by the Entrez IDs in the data</span>
    values     <span class='op'>=</span> <span class='st'>"100913187"</span>
    <span class='op'>)</span>
</code></pre>
</div>
<pre><code>  entrezgene_id hgnc_symbol
1     100913187    APOBEC3A</code></pre>
</div>
<p>And the query result returned by OrgDb:</p>
<div class="layout-chunk" data-layout="l-body">
<div class="sourceCode">
<pre class="sourceCode r"><code class="sourceCode r"><span class='fu'>AnnotationDbi</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/AnnotationDbi/man/AnnotationDb-class.html'>select</a></span><span class='op'>(</span><span class='va'>org.Hs.eg.db</span>,
                      keys    <span class='op'>=</span> <span class='st'>"100913187"</span>,
                      keytype <span class='op'>=</span> <span class='st'>"ENTREZID"</span>,
                      columns <span class='op'>=</span> <span class='va'>cols</span><span class='op'>)</span>
</code></pre>
</div>
<pre><code>   ENTREZID     SYMBOL
1 100913187 APOBEC3A_B</code></pre>
</div>
<p>If we look it up in <a href="https://www.ncbi.nlm.nih.gov/gene/100913187">NCBI gene</a>, <code>APOBEC3A_B</code> is the right HGNC symbol. The Entrez gene ID for <code>APOBEC3A</code> is <a href="https://www.ncbi.nlm.nih.gov/gene/200315"><code>200315</code></a>.<br />
</p>
<p>Incorrect mappings of gene identifiers can result in significant impacts on downstream analyses. Differential gene expression analysis of a study on the efficacy of the co-therapy of birinapant and platinum-based carboplatin against platinum resistance in CA125-negative high-grade serous cancer (HGSC) cells showed significant difference in expression of DNA repair and apoptosis genes between therapy-resistant CA125-negative and differentiated CA125 positive chemotherapy-naïve human HGSC cells. <span class="citation" data-cites="janzen2015retracted">(<a href="#ref-janzen2015retracted" role="doc-biblioref">D. Janzen et al. 2015</a>)</span> However, this finding of difference in expression of the genes was a result of 341056 of its 730620 expression values mapped to incorrect gene names. As this problem was fixed, none of these genes were shown to be significantly differentially expressed between the two groups.<span class="citation" data-cites="janzen2016">(<a href="#ref-janzen2016" role="doc-biblioref">D. M. Janzen et al. 2016</a>)</span></p>
<h2 id="to-do">To Do</h2>
<p>I will update the result of mapping by BridgeDbR as soon as the problem is fixed. Differential expression analysis will be performed on this count matrix.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r distill-force-highlighting-css"><code class="sourceCode r"></code></pre></div>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-org" class="csl-entry" role="doc-biblioentry">
Carlson, Marc, S Falcon, H Pages, and N Li. 2019. <span>“Org. Hs. Eg. Db: Genome Wide Annotation for Human.”</span> <em>R Package Version</em> 3 (2).
</div>
<div id="ref-BridgeDbR" class="csl-entry" role="doc-biblioentry">
Christ Leemans &lt;Christleemans@Gmail.Com&gt;, Egon Willighagen&lt;Egon.Willighagen@Gmail.Com&gt;, Anwesha Bohler&lt;Anweshabohler@Gmail.Com&gt;, Lars Eijssen&lt;L.Eijssen@Maastrichtuniversity.Nl&gt;. 2017. <span>“BridgeDbR.”</span> Bioconductor. <a href="https://doi.org/10.18129/B9.BIOC.BRIDGEDBR">https://doi.org/10.18129/B9.BIOC.BRIDGEDBR</a>.
</div>
<div id="ref-biomart1" class="csl-entry" role="doc-biblioentry">
Durinck, Steffen, Yves Moreau, Arek Kasprzyk, Sean Davis, Bart De Moor, Alvis Brazma, and Wolfgang Huber. 2005. <span>“<span class="nocase">BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis</span>.”</span> <em>Bioinformatics</em> 21 (16): 3439–40. <a href="https://doi.org/10.1093/bioinformatics/bti525">https://doi.org/10.1093/bioinformatics/bti525</a>.
</div>
<div id="ref-annotationdbi" class="csl-entry" role="doc-biblioentry">
Hervé Pagès, Marc Carlson. 2017. <span>“AnnotationDbi.”</span> Bioconductor. <a href="https://doi.org/10.18129/B9.BIOC.ANNOTATIONDBI">https://doi.org/10.18129/B9.BIOC.ANNOTATIONDBI</a>.
</div>
<div id="ref-janzen2016" class="csl-entry" role="doc-biblioentry">
Janzen, D. M., E. Tiourin, J. A. Salehi, D. Y. Paik, J. Lu, M. Pellegrini, and S. Memarzadeh. 2016. <span>“Correction: Corrigendum: An Apoptosis-Enhancing Drug Overcomes Platinum Resistance in a Tumour-Initiating Subpopulation of Ovarian Cancer.”</span> <em>Nature Communications</em> 7 (1): 10703. <a href="https://doi.org/10.1038/ncomms10703">https://doi.org/10.1038/ncomms10703</a>.
</div>
<div id="ref-janzen2015retracted" class="csl-entry" role="doc-biblioentry">
Janzen, DM, E Tiourin, JA Salehi, DY Paik, J Lu, M Pellegrini, and S Memarzadeh. 2015. <span>“RETRACTED ARTICLE: An Apoptosis-Enhancing Drug Overcomes Platinum Resistance in a Tumour-Initiating Subpopulation of Ovarian Cancer.”</span> <em>Nature Communications</em> 6 (1): 1–18.
</div>
<div id="ref-BridgeDb" class="csl-entry" role="doc-biblioentry">
Van Iersel, Martijn, Alexander Pico, Thomas Kelder, Jianjiong Gao, Isaac Ho, Kristina Hanspers, Bruce Conklin, and Chris Evelo. 2010. <span>“The BridgeDb Framework: Standardized Access to Gene, Protein and Metabolite Identifier Mapping Services.”</span> <em>BMC Bioinformatics</em> 11 (January): 5. <a href="https://doi.org/10.1186/1471-2105-11-5">https://doi.org/10.1186/1471-2105-11-5</a>.
</div>
</div>
<!--radix_placeholder_article_footer-->
<div class="article-footer">
  <p class="social_footer">
    <span class="article-sharing">
      Share: &nbsp;
      <a href="https://twitter.com/share?text=Comparing%20HGNC%20Symbol%20Mappings%20by%203%20Different%20Databases&amp;url=https%3A%2F%2Fwww.ffli.dev%2Fposts%2F2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases%2F" aria-label="share on twitter">
        <i class="fab fa-twitter" aria-hidden="true"></i>
      </a>
      <a href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3A%2F%2Fwww.ffli.dev%2Fposts%2F2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases%2F&amp;title=Comparing%20HGNC%20Symbol%20Mappings%20by%203%20Different%20Databases" aria-label="share on linkedin">
        <i class="fab fa-linkedin" aria-hidden="true"></i>
      </a>
    </span>
  </p>
</div>
<!--/radix_placeholder_article_footer-->
</div>

<div class="d-appendix">
</div>


<!--radix_placeholder_site_after_body-->
<!--
Buhr (2021, March 14). jmbuhr.de: Making a Darkmode for your distill site. Retrieved from
https://jmbuhr.de/posts/2021-03-14-making-a-darkmode-for-your-distill-site/
-->
<script>
const darkmodeToggle = document.getElementsByClassName("fas fa-lightbulb")[0];
let theme = localStorage.getItem("theme");

if (!theme) {
  if (window.matchMedia("(prefers-color-scheme: dark)").matches ) {
    theme = "dark";
  } else {
    theme = "light";
  }
}
setTheme();

function switchTheme() {
  if (theme === "light") {
    theme = "dark";
  } else {
    theme = "light";
  }
  localStorage.setItem("theme", theme);
  setTheme();
  console.log(theme);
}

function setTheme() {
  if (theme === "dark") {
    document.body.classList.add('darkmode');
  } else {
    document.body.classList.remove('darkmode');
  }
}

darkmodeToggle.addEventListener('click', switchTheme)
</script>
<!-- Add function to open links to external links in new tab, from: -->
<!-- https://yihui.name/en/2018/09/target-blank/ -->
<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>
<!--/radix_placeholder_site_after_body-->
<!--radix_placeholder_appendices-->
<div class="appendix-bottom">
  <h3 id="references">References</h3>
  <div id="references-listing"></div>
  <h3 id="updates-and-corrections">Corrections</h3>
  <p>If you see mistakes or want to suggest changes, please <a href="https://github.com/ff98li/ffli.dev/issues/new">create an issue</a> on the source repository.</p>
  <h3 id="reuse">Reuse</h3>
  <p>Text and figures are licensed under Creative Commons Attribution <a rel="license" href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>. Source code is available at <a href="https://github.com/ff98li/ffli.dev">https://github.com/ff98li/ffli.dev</a>, unless otherwise noted. The figures that have been reused from other sources don't fall under this license and can be recognized by a note in their caption: "Figure from ...".</p>
  <h3 id="citation">Citation</h3>
  <p>For attribution, please cite this work as</p>
  <pre class="citation-appendix short">Li (2021, July 11). ffli.dev: Comparing HGNC Symbol Mappings by 3 Different Databases. Retrieved from https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/</pre>
  <p>BibTeX citation</p>
  <pre class="citation-appendix long">@misc{li2021comparing,
  author = {Li, Feifei},
  title = {ffli.dev: Comparing HGNC Symbol Mappings by 3 Different Databases},
  url = {https://www.ffli.dev/posts/2021-07-11-comparing-hgnc-symbol-mappings-by-3-different-databases/},
  year = {2021}
}</pre>
</div>
<!--/radix_placeholder_appendices-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<!--radix_placeholder_navigation_after_body--><html><body>
<div class="distill-site-nav distill-site-footer">
<div class="footer">
    <p>
        <svg style="height:14px;fill:currentColor;" viewbox="0 0 496 512"><path d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm0 448c-110.532 0-200-89.451-200-200 0-110.531 89.451-200 200-200 110.532 0 200 89.451 200 200 0 110.532-89.451 200-200 200zm107.351-101.064c-9.614 9.712-45.53 41.396-104.065 41.396-82.43 0-140.484-61.425-140.484-141.567 0-79.152 60.275-139.401 139.762-139.401 55.531 0 88.738 26.62 97.593 34.779a11.965 11.965 0 0 1 1.936 15.322l-18.155 28.113c-3.841 5.95-11.966 7.282-17.499 2.921-8.595-6.776-31.814-22.538-61.708-22.538-48.303 0-77.916 35.33-77.916 80.082 0 41.589 26.888 83.692 78.277 83.692 32.657 0 56.843-19.039 65.726-27.225 5.27-4.857 13.596-4.039 17.82 1.738l19.865 27.17a11.947 11.947 0 0 1-1.152 15.518z"></path></svg><script>
            if (Date.parse(document.lastModified) != 0)
                document.write(new Date().getFullYear())
        </script> Feifei Li
        <a href="https://creativecommons.org/licenses/by-sa/4.0/">
            <svg style="height:14px;fill:currentColor;" viewbox="0 0 496 512"><path d="M245.83 214.87l-33.22 17.28c-9.43-19.58-25.24-19.93-27.46-19.93-22.13 0-33.22 14.61-33.22 43.84 0 23.57 9.21 43.84 33.22 43.84 14.47 0 24.65-7.09 30.57-21.26l30.55 15.5c-6.17 11.51-25.69 38.98-65.1 38.98-22.6 0-73.96-10.32-73.96-77.05 0-58.69 43-77.06 72.63-77.06 30.72-.01 52.7 11.95 65.99 35.86zm143.05 0l-32.78 17.28c-9.5-19.77-25.72-19.93-27.9-19.93-22.14 0-33.22 14.61-33.22 43.84 0 23.55 9.23 43.84 33.22 43.84 14.45 0 24.65-7.09 30.54-21.26l31 15.5c-2.1 3.75-21.39 38.98-65.09 38.98-22.69 0-73.96-9.87-73.96-77.05 0-58.67 42.97-77.06 72.63-77.06 30.71-.01 52.58 11.95 65.56 35.86zM247.56 8.05C104.74 8.05 0 123.11 0 256.05c0 138.49 113.6 248 247.56 248 129.93 0 248.44-100.87 248.44-248 0-137.87-106.62-248-248.44-248zm.87 450.81c-112.54 0-203.7-93.04-203.7-202.81 0-105.42 85.43-203.27 203.72-203.27 112.53 0 202.82 89.46 202.82 203.26-.01 121.69-99.68 202.82-202.84 202.82z"></path></svg><svg style="height:14px;fill:currentColor;" viewbox="0 0 496 512"><path d="M314.9 194.4v101.4h-28.3v120.5h-77.1V295.9h-28.3V194.4c0-4.4 1.6-8.2 4.6-11.3 3.1-3.1 6.9-4.7 11.3-4.7H299c4.1 0 7.8 1.6 11.1 4.7 3.1 3.2 4.8 6.9 4.8 11.3zm-101.5-63.7c0-23.3 11.5-35 34.5-35s34.5 11.7 34.5 35c0 23-11.5 34.5-34.5 34.5s-34.5-11.5-34.5-34.5zM247.6 8C389.4 8 496 118.1 496 256c0 147.1-118.5 248-248.4 248C113.6 504 0 394.5 0 256 0 123.1 104.7 8 247.6 8zm.8 44.7C130.2 52.7 44.7 150.6 44.7 256c0 109.8 91.2 202.8 203.7 202.8 103.2 0 202.8-81.1 202.8-202.8.1-113.8-90.2-203.3-202.8-203.3z"></path></svg><svg style="height:14px;fill:currentColor;" viewbox="0 0 496 512"><path d="M247.6 8C389.4 8 496 118.1 496 256c0 147.1-118.5 248-248.4 248C113.6 504 0 394.5 0 256 0 123.1 104.7 8 247.6 8zm.8 44.7C130.2 52.7 44.7 150.6 44.7 256c0 109.8 91.2 202.8 203.7 202.8 103.2 0 202.8-81.1 202.8-202.8.1-113.8-90.2-203.3-202.8-203.3zM137.7 221c13-83.9 80.5-95.7 108.9-95.7 99.8 0 127.5 82.5 127.5 134.2 0 63.6-41 132.9-128.9 132.9-38.9 0-99.1-20-109.4-97h62.5c1.5 30.1 19.6 45.2 54.5 45.2 23.3 0 58-18.2 58-82.8 0-82.5-49.1-80.6-56.7-80.6-33.1 0-51.7 14.6-55.8 43.8h18.2l-49.2 49.2-49-49.2h19.4z"></path></svg></a>
    </p>
</div>
<p>
    <svg style="height:14px;fill:currentColor;" viewbox="0 0 512 512"><path d="M507.73 109.1c-2.24-9.03-13.54-12.09-20.12-5.51l-74.36 74.36-67.88-11.31-11.31-67.88 74.36-74.36c6.62-6.62 3.43-17.9-5.66-20.16-47.38-11.74-99.55.91-136.58 37.93-39.64 39.64-50.55 97.1-34.05 147.2L18.74 402.76c-24.99 24.99-24.99 65.51 0 90.5 24.99 24.99 65.51 24.99 90.5 0l213.21-213.21c50.12 16.71 107.47 5.68 147.37-34.22 37.07-37.07 49.7-89.32 37.91-136.73zM64 472c-13.25 0-24-10.75-24-24 0-13.26 10.75-24 24-24s24 10.74 24 24c0 13.25-10.75 24-24 24z"></path></svg>
    Made with
    <svg style="height:14px;fill:currentColor;" viewbox="0 0 512 512"><path d="M192 384h192c53 0 96-43 96-96h32c70.6 0 128-57.4 128-128S582.6 32 512 32H120c-13.3 0-24 10.7-24 24v232c0 53 43 96 96 96zM512 96c35.3 0 64 28.7 64 64s-28.7 64-64 64h-32V96h32zm47.7 384H48.3c-47.6 0-61-64-36-64h583.3c25 0 11.8 64-35.9 64z"></path></svg>,
    <a href="https://github.com/FortAwesome/Font-Awesome">
        <svg style="height:14px;fill:currentColor;" viewbox="0 0 512 512"><path d="M397.8 32H50.2C22.7 32 0 54.7 0 82.2v347.6C0 457.3 22.7 480 50.2 480h347.6c27.5 0 50.2-22.7 50.2-50.2V82.2c0-27.5-22.7-50.2-50.2-50.2zm-45.4 284.3c0 4.2-3.6 6-7.8 7.8-16.7 7.2-34.6 13.7-53.8 13.7-26.9 0-39.4-16.7-71.7-16.7-23.3 0-47.8 8.4-67.5 17.3-1.2.6-2.4.6-3.6 1.2V385c0 1.8 0 3.6-.6 4.8v1.2c-2.4 8.4-10.2 14.3-19.1 14.3-11.3 0-20.3-9-20.3-20.3V166.4c-7.8-6-13.1-15.5-13.1-26.3 0-18.5 14.9-33.5 33.5-33.5 18.5 0 33.5 14.9 33.5 33.5 0 10.8-4.8 20.3-13.1 26.3v18.5c1.8-.6 3.6-1.2 5.4-2.4 18.5-7.8 40.6-14.3 61.5-14.3 22.7 0 40.6 6 60.9 13.7 4.2 1.8 8.4 2.4 13.1 2.4 22.7 0 47.8-16.1 53.8-16.1 4.8 0 9 3.6 9 7.8v140.3z"></path></svg>
        Font Awesome 5
    </a>,
    <a href="https://github.com/ff98li/bioinfo-blog">
        <svg style="height:14px;fill:currentColor;" viewbox="0 0 512 512"><path d="M384 144c0-44.2-35.8-80-80-80s-80 35.8-80 80c0 36.4 24.3 67.1 57.5 76.8-.6 16.1-4.2 28.5-11 36.9-15.4 19.2-49.3 22.4-85.2 25.7-28.2 2.6-57.4 5.4-81.3 16.9v-144c32.5-10.2 56-40.5 56-76.3 0-44.2-35.8-80-80-80S0 35.8 0 80c0 35.8 23.5 66.1 56 76.3v199.3C23.5 365.9 0 396.2 0 432c0 44.2 35.8 80 80 80s80-35.8 80-80c0-34-21.2-63.1-51.2-74.6 3.1-5.2 7.8-9.8 14.9-13.4 16.2-8.2 40.4-10.4 66.1-12.8 42.2-3.9 90-8.4 118.2-43.4 14-17.4 21.1-39.8 21.6-67.9 31.6-10.8 54.4-40.7 54.4-75.9zM80 64c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16zm0 384c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zm224-320c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16z"></path></svg>
        GitHub
    </a>,
    <a href="https://pkgs.rstudio.com/distill/index.html">
        <svg style="height:14px;fill:currentColor;" viewbox="0 0 512 512"><path d="M581 226.6C581 119.1 450.9 32 290.5 32S0 119.1 0 226.6C0 322.4 103.3 402 239.4 418.1V480h99.1v-61.5c24.3-2.7 47.6-7.4 69.4-13.9L448 480h112l-67.4-113.7c54.5-35.4 88.4-84.9 88.4-139.7zm-466.8 14.5c0-73.5 98.9-133 220.8-133s211.9 40.7 211.9 133c0 50.1-26.5 85-70.3 106.4-2.4-1.6-4.7-2.9-6.4-3.7-10.2-5.2-27.8-10.5-27.8-10.5s86.6-6.4 86.6-92.7-90.6-87.9-90.6-87.9h-199V361c-74.1-21.5-125.2-67.1-125.2-119.9zm225.1 38.3v-55.6c57.8 0 87.8-6.8 87.8 27.3 0 36.5-38.2 28.3-87.8 28.3zm-.9 72.5H365c10.8 0 18.9 11.7 24 19.2-16.1 1.9-33 2.8-50.6 2.9v-22.1z"></path></svg>
        distill
    </a>
    and
    <a href="https://pkgs.rstudio.com/rmarkdown/">
        <svg style="height:14px;fill:currentColor;" viewbox="0 0 512 512"><path d="M581 226.6C581 119.1 450.9 32 290.5 32S0 119.1 0 226.6C0 322.4 103.3 402 239.4 418.1V480h99.1v-61.5c24.3-2.7 47.6-7.4 69.4-13.9L448 480h112l-67.4-113.7c54.5-35.4 88.4-84.9 88.4-139.7zm-466.8 14.5c0-73.5 98.9-133 220.8-133s211.9 40.7 211.9 133c0 50.1-26.5 85-70.3 106.4-2.4-1.6-4.7-2.9-6.4-3.7-10.2-5.2-27.8-10.5-27.8-10.5s86.6-6.4 86.6-92.7-90.6-87.9-90.6-87.9h-199V361c-74.1-21.5-125.2-67.1-125.2-119.9zm225.1 38.3v-55.6c57.8 0 87.8-6.8 87.8 27.3 0 36.5-38.2 28.3-87.8 28.3zm-.9 72.5H365c10.8 0 18.9 11.7 24 19.2-16.1 1.9-33 2.8-50.6 2.9v-22.1z"></path></svg>
        Rmarkdown
    </a>.
</p>
<p>
    <span style="font-size:0.8rem;">
        <script>
            if (Date.parse(document.lastModified) != 0)
                document.write('Last updated: ' + document.lastModified);
        </script></span>
</p>
</div>
<!--/radix_placeholder_navigation_after_body-->
</body></html>


</body>

</html>
